{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "from Bio import Entrez\n",
    "import os"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'RNA AND disease AND mouse'"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "searchKeywords = [\"RNA\", \"disease\", \"mouse\"]\n",
    "searchTermString = \" AND \".join(searchKeywords)\n",
    "searchTermString"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "Entrez.email = \"diaorch@umich.edu\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['28009297', '28008926', '28007334', '28002605', '28002126', '27998767', '27995765', '27981604', '27981238', '27976428', '27975168', '27966608', '27965467', '27965448', '27955979', '27955842', '27955791', '27939857', '27939723', '27939432', '27933543', '27933528', '27932527', '27931246', '27930406', '27929729', '27927222', '27924068', '27920257', '27920150', '27919828', '27918553', '27918527', '27913791', '27913212', '27911317', '27910163', '27910050', '27909811', '27909059', '27909014', '27906075', '27903277', '27902457', '27901018', '27900797', '27900389', '27900262', '27899663', '27899633']"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "searchHandle = Entrez.esearch(db = \"pubmed\", term = searchTermString, retmax = 50)\n",
    "searchRecord = Entrez.read(searchHandle)\n",
    "searchRecord[\"IdList\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[u'Count',\n",
       " u'RetMax',\n",
       " u'IdList',\n",
       " u'TranslationStack',\n",
       " u'TranslationSet',\n",
       " u'RetStart',\n",
       " u'QueryTranslation']"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "searchRecord.keys()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PMID- 28009297\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "DA  - 20161223\n",
      "LR  - 20161224\n",
      "IS  - 2211-1247 (Electronic)\n",
      "VI  - 17\n",
      "IP  - 12\n",
      "DP  - 2016 Dec 20\n",
      "TI  - Integrative Analysis of PRKAG2 Cardiomyopathy iPS and Microtissue Models\n",
      "      Identifies AMPK as a Regulator of Metabolism, Survival, and Fibrosis.\n",
      "PG  - 3292-3304\n",
      "LID - S2211-1247(16)31640-0 [pii]\n",
      "LID - 10.1016/j.celrep.2016.11.066 [doi]\n",
      "AB  - AMP-activated protein kinase (AMPK) is a metabolic enzyme that can be activated\n",
      "      by nutrient stress or genetic mutations. Missense mutations in the regulatory\n",
      "      subunit, PRKAG2, activate AMPK and cause left ventricular hypertrophy, glycogen\n",
      "      accumulation, and ventricular pre-excitation. Using human iPS cell models\n",
      "      combined with three-dimensional cardiac microtissues, we show that activating\n",
      "      PRKAG2 mutations increase microtissue twitch force by enhancing myocyte survival.\n",
      "      Integrating RNA sequencing with metabolomics, PRKAG2 mutations that activate AMPK\n",
      "      remodeled global metabolism by regulating RNA transcripts to favor glycogen\n",
      "      storage and oxidative metabolism instead of glycolysis. As in patients with\n",
      "      PRKAG2 cardiomyopathy, iPS cell and mouse models are protected from cardiac\n",
      "      fibrosis, and we define a crosstalk between AMPK and post-transcriptional\n",
      "      regulation of TGFbeta isoform signaling that has implications in fibrotic forms\n",
      "      of cardiomyopathy. Our results establish critical connections among metabolic\n",
      "      sensing, myocyte survival, and TGFbeta signaling.\n",
      "CI  - Copyright (c) 2016 The Author(s). Published by Elsevier Inc. All rights reserved.\n",
      "FAU - Hinson, J Travis\n",
      "AU  - Hinson JT\n",
      "AD  - The Jackson Laboratory for Genomic Medicine, Farmington, CT 06032, USA;\n",
      "      Cardiology Center, University of Connecticut Health, Farmington, CT 06030, USA.\n",
      "      Electronic address: travis.hinson@jax.org.\n",
      "FAU - Chopra, Anant\n",
      "AU  - Chopra A\n",
      "AD  - Department of Biomedical Engineering, Boston University, Boston, MA 02215, USA;\n",
      "      The Wyss Institute for Biologically Inspired Engineering at Harvard University,\n",
      "      Boston, MA 02115, USA.\n",
      "FAU - Lowe, Andre\n",
      "AU  - Lowe A\n",
      "AD  - The Jackson Laboratory for Genomic Medicine, Farmington, CT 06032, USA.\n",
      "FAU - Sheng, Calvin C\n",
      "AU  - Sheng CC\n",
      "AD  - Department of Genetics, Harvard Medical School, Boston, MA 02115, USA.\n",
      "FAU - Gupta, Rajat M\n",
      "AU  - Gupta RM\n",
      "AD  - Division of Cardiovascular Medicine, Brigham and Women's Hospital, Boston, MA\n",
      "      02115, USA.\n",
      "FAU - Kuppusamy, Rajarajan\n",
      "AU  - Kuppusamy R\n",
      "AD  - Division of Cardiovascular Medicine, Cardiovascular Institute, Stanford\n",
      "      University School of Medicine, Stanford, CA 94305, USA.\n",
      "FAU - O'Sullivan, John\n",
      "AU  - O'Sullivan J\n",
      "AD  - Division of Cardiovascular Medicine, Massachusetts General Hospital, Boston, MA\n",
      "      02114, USA.\n",
      "FAU - Rowe, Glenn\n",
      "AU  - Rowe G\n",
      "AD  - Division of Cardiovascular Disease, University of Alabama at Birmingham,\n",
      "      Birmingham, AL 35294, USA.\n",
      "FAU - Wakimoto, Hiroko\n",
      "AU  - Wakimoto H\n",
      "AD  - Department of Genetics, Harvard Medical School, Boston, MA 02115, USA.\n",
      "FAU - Gorham, Joshua\n",
      "AU  - Gorham J\n",
      "AD  - Department of Genetics, Harvard Medical School, Boston, MA 02115, USA.\n",
      "FAU - Zhang, Kehan\n",
      "AU  - Zhang K\n",
      "AD  - Department of Biomedical Engineering, Boston University, Boston, MA 02215, USA;\n",
      "      The Wyss Institute for Biologically Inspired Engineering at Harvard University,\n",
      "      Boston, MA 02115, USA.\n",
      "FAU - Musunuru, Kiran\n",
      "AU  - Musunuru K\n",
      "AD  - Penn Cardiovascular Institute, University of Pennsylvania, Philadelphia, PA\n",
      "      19104, USA.\n",
      "FAU - Gerszten, Robert E\n",
      "AU  - Gerszten RE\n",
      "AD  - Division of Cardiovascular Medicine, Massachusetts General Hospital, Boston, MA\n",
      "      02114, USA; Division of Cardiovascular Medicine, Beth Israel Deaconess Hospital, \n",
      "      Boston, MA 02115, USA.\n",
      "FAU - Wu, Sean M\n",
      "AU  - Wu SM\n",
      "AD  - Division of Cardiovascular Medicine, Cardiovascular Institute, Stanford\n",
      "      University School of Medicine, Stanford, CA 94305, USA.\n",
      "FAU - Chen, Christopher S\n",
      "AU  - Chen CS\n",
      "AD  - Department of Biomedical Engineering, Boston University, Boston, MA 02215, USA;\n",
      "      The Wyss Institute for Biologically Inspired Engineering at Harvard University,\n",
      "      Boston, MA 02115, USA.\n",
      "FAU - Seidman, Jonathan G\n",
      "AU  - Seidman JG\n",
      "AD  - Department of Genetics, Harvard Medical School, Boston, MA 02115, USA.\n",
      "FAU - Seidman, Christine E\n",
      "AU  - Seidman CE\n",
      "AD  - Department of Genetics, Harvard Medical School, Boston, MA 02115, USA; Division\n",
      "      of Cardiovascular Medicine, Brigham and Women's Hospital, Boston, MA 02115, USA; \n",
      "      Howard Hughes Medical Institute, Chevy Chase, MD 20815, USA. Electronic address: \n",
      "      cseidman@genetics.med.harvard.edu.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PL  - United States\n",
      "TA  - Cell Rep\n",
      "JT  - Cell reports\n",
      "JID - 101573691\n",
      "OTO - NOTNLM\n",
      "OT  - AMP-activated protein kinase\n",
      "OT  - AMPK\n",
      "OT  - PRKAG2\n",
      "OT  - TGF-beta\n",
      "OT  - Wolff-Parkinson-White syndrome\n",
      "OT  - fibrosis\n",
      "OT  - glycogen storage disease\n",
      "OT  - hypertrophic cardiomyopathy\n",
      "EDAT- 2016/12/24 06:00\n",
      "MHDA- 2016/12/24 06:00\n",
      "CRDT- 2016/12/24 06:00\n",
      "PHST- 2016/07/01 [received]\n",
      "PHST- 2016/09/19 [revised]\n",
      "PHST- 2016/11/21 [accepted]\n",
      "AID - S2211-1247(16)31640-0 [pii]\n",
      "AID - 10.1016/j.celrep.2016.11.066 [doi]\n",
      "PST - ppublish\n",
      "SO  - Cell Rep. 2016 Dec 20;17(12):3292-3304. doi: 10.1016/j.celrep.2016.11.066.\n",
      "\n",
      "PMID- 28008926\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161223\n",
      "LR  - 20161223\n",
      "IS  - 1748-7838 (Electronic)\n",
      "IS  - 1001-0602 (Linking)\n",
      "DP  - 2016 Dec 23\n",
      "TI  - Long-term propagation of tree shrew spermatogonial stem cells in culture and\n",
      "      successful generation of transgenic offspring.\n",
      "LID - 10.1038/cr.2016.156 [doi]\n",
      "AB  - Tree shrews have a close relationship to primates and have many advantages over\n",
      "      rodents in biomedical research. However, the lack of gene manipulation methods\n",
      "      has hindered the wider use of this animal. Spermatogonial stem cells (SSCs) have \n",
      "      been successfully expanded in culture to permit sophisticated gene editing in the\n",
      "      mouse and rat. Here, we describe a culture system for the long-term expansion of \n",
      "      tree shrew SSCs without the loss of stem cell properties. In our study, thymus\n",
      "      cell antigen 1 was used to enrich tree shrew SSCs. RNA-sequencing analysis\n",
      "      revealed that the Wnt/beta-catenin signaling pathway was active in\n",
      "      undifferentiated SSCs, but was downregulated upon the initiation of SSC\n",
      "      differentiation. Exposure of tree shrew primary SSCs to recombinant Wnt3a protein\n",
      "      during the initial passages of culture enhanced the survival of SSCs. Use of tree\n",
      "      shrew Sertoli cells, but not mouse embryonic fibroblasts, as feeder was found to \n",
      "      be necessary for tree shrew SSC proliferation, leading to a robust cell expansion\n",
      "      and long-term culture. The expanded tree shrew SSCs were transfected with\n",
      "      enhanced green fluorescent protein (EGFP)-expressing lentiviral vectors. After\n",
      "      transplantation into sterilized adult male tree shrew's testes, the EGFP-tagged\n",
      "      SSCs were able to restore spermatogenesis and successfully generate transgenic\n",
      "      offspring. Moreover, these SSCs were suitable for the CRISPR/Cas9-mediated gene\n",
      "      modification. The development of a culture system to expand tree shrew SSCs in\n",
      "      combination with a gene editing approach paves the way for precise genome\n",
      "      manipulation using the tree shrew.Cell Research advance online publication 23\n",
      "      December 2016; doi:10.1038/cr.2016.156.\n",
      "FAU - Li, Chao-Hui\n",
      "AU  - Li CH\n",
      "AD  - State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of\n",
      "      Zoology, Chinese Academy of Sciences, Kunming 650223, China.\n",
      "AD  - Yunnan Key Laboratory of Animal Reproduction, Kunming Institute of Zoology,\n",
      "      Chinese Academy of Sciences, Kunming 650223, China.\n",
      "AD  - Kunming College of Life Science, University of Chinese Academy of Sciences,\n",
      "      Kunming 650204, China.\n",
      "FAU - Yan, Lan-Zhen\n",
      "AU  - Yan LZ\n",
      "AD  - Key Laboratory of Animal Models and Human Disease Mechanisms, Kunming Institute\n",
      "      of Zoology, Chinese Academy of Sciences, Kunming 650223, China.\n",
      "AD  - Primate Research Center, Kunming Institute of Zoology, Chinese Academy of\n",
      "      Sciences, Kunming 650223, China.\n",
      "FAU - Ban, Wen-Zan\n",
      "AU  - Ban WZ\n",
      "AD  - State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of\n",
      "      Zoology, Chinese Academy of Sciences, Kunming 650223, China.\n",
      "FAU - Tu, Qiu\n",
      "AU  - Tu Q\n",
      "AD  - Key Laboratory of Animal Models and Human Disease Mechanisms, Kunming Institute\n",
      "      of Zoology, Chinese Academy of Sciences, Kunming 650223, China.\n",
      "AD  - Kunming College of Life Science, University of Chinese Academy of Sciences,\n",
      "      Kunming 650204, China.\n",
      "FAU - Wu, Yong\n",
      "AU  - Wu Y\n",
      "AD  - Key Laboratory of Animal Models and Human Disease Mechanisms, Kunming Institute\n",
      "      of Zoology, Chinese Academy of Sciences, Kunming 650223, China.\n",
      "AD  - Kunming College of Life Science, University of Chinese Academy of Sciences,\n",
      "      Kunming 650204, China.\n",
      "FAU - Wang, Lin\n",
      "AU  - Wang L\n",
      "AD  - State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of\n",
      "      Zoology, Chinese Academy of Sciences, Kunming 650223, China.\n",
      "AD  - Yunnan Key Laboratory of Animal Reproduction, Kunming Institute of Zoology,\n",
      "      Chinese Academy of Sciences, Kunming 650223, China.\n",
      "FAU - Bi, Rui\n",
      "AU  - Bi R\n",
      "AD  - Key Laboratory of Animal Models and Human Disease Mechanisms, Kunming Institute\n",
      "      of Zoology, Chinese Academy of Sciences, Kunming 650223, China.\n",
      "FAU - Ji, Shuang\n",
      "AU  - Ji S\n",
      "AD  - State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of\n",
      "      Zoology, Chinese Academy of Sciences, Kunming 650223, China.\n",
      "FAU - Ma, Yu-Hua\n",
      "AU  - Ma YH\n",
      "AD  - Primate Research Center, Kunming Institute of Zoology, Chinese Academy of\n",
      "      Sciences, Kunming 650223, China.\n",
      "FAU - Nie, Wen-Hui\n",
      "AU  - Nie WH\n",
      "AD  - State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of\n",
      "      Zoology, Chinese Academy of Sciences, Kunming 650223, China.\n",
      "FAU - Lv, Long-Bao\n",
      "AU  - Lv LB\n",
      "AD  - Primate Research Center, Kunming Institute of Zoology, Chinese Academy of\n",
      "      Sciences, Kunming 650223, China.\n",
      "FAU - Yao, Yong-Gang\n",
      "AU  - Yao YG\n",
      "AD  - Kunming College of Life Science, University of Chinese Academy of Sciences,\n",
      "      Kunming 650204, China.\n",
      "AD  - Key Laboratory of Animal Models and Human Disease Mechanisms, Kunming Institute\n",
      "      of Zoology, Chinese Academy of Sciences, Kunming 650223, China.\n",
      "AD  - Primate Research Center, Kunming Institute of Zoology, Chinese Academy of\n",
      "      Sciences, Kunming 650223, China.\n",
      "AD  - CAS Center for Excellence in Brain Science and Intelligence Technology, Chinese\n",
      "      Academy of Sciences, Shanghai 200031, China.\n",
      "FAU - Zhao, Xu-Dong\n",
      "AU  - Zhao XD\n",
      "AD  - Key Laboratory of Animal Models and Human Disease Mechanisms, Kunming Institute\n",
      "      of Zoology, Chinese Academy of Sciences, Kunming 650223, China.\n",
      "AD  - Primate Research Center, Kunming Institute of Zoology, Chinese Academy of\n",
      "      Sciences, Kunming 650223, China.\n",
      "FAU - Zheng, Ping\n",
      "AU  - Zheng P\n",
      "AD  - State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of\n",
      "      Zoology, Chinese Academy of Sciences, Kunming 650223, China.\n",
      "AD  - Yunnan Key Laboratory of Animal Reproduction, Kunming Institute of Zoology,\n",
      "      Chinese Academy of Sciences, Kunming 650223, China.\n",
      "AD  - Primate Research Center, Kunming Institute of Zoology, Chinese Academy of\n",
      "      Sciences, Kunming 650223, China.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161223\n",
      "PL  - England\n",
      "TA  - Cell Res\n",
      "JT  - Cell research\n",
      "JID - 9425763\n",
      "EDAT- 2016/12/23 06:00\n",
      "MHDA- 2016/12/23 06:00\n",
      "CRDT- 2016/12/24 06:00\n",
      "PHST- 2016/07/20 [received]\n",
      "PHST- 2016/10/03 [revised]\n",
      "PHST- 2016/10/11 [accepted]\n",
      "AID - cr2016156 [pii]\n",
      "AID - 10.1038/cr.2016.156 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Cell Res. 2016 Dec 23. doi: 10.1038/cr.2016.156.\n",
      "\n",
      "PMID- 28007334\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161223\n",
      "LR  - 20161223\n",
      "IS  - 1873-460X (Electronic)\n",
      "IS  - 1056-8727 (Linking)\n",
      "DP  - 2016 Dec 08\n",
      "TI  - Microarray analysis of long noncoding RNA expression patterns in diabetic\n",
      "      nephropathy.\n",
      "LID - S1056-8727(16)30976-X [pii]\n",
      "LID - 10.1016/j.jdiacomp.2016.11.017 [doi]\n",
      "AB  - AIMS: Long noncoding RNAs (lncRNAs) are implicated in various biological\n",
      "      processes and human diseases. Diabetic nephropathy (DN) is the leading cause of\n",
      "      end-stage renal disease (ESRD). We explored the potential functions of lncRNAs in\n",
      "      DN. METHODS: We established a mouse model of DN and compared lncRNA expression\n",
      "      patterns between DN model and db/m control mouse kidney tissues using microarray \n",
      "      analysis. lncRNA function was predicted by gene ontology enrichment and KEGG\n",
      "      pathway analyses of lncRNAs-coexpressed mRNAs. Quantitative reverse-transcription\n",
      "      PCR was used for validation. Cis- and trans-regulation analyses were conducted to\n",
      "      reveal potential relationships between lncRNAs and their target genes. RESULTS:\n",
      "      In DN, 311 lncRNAs were dysregulated. LncRNA-coexpressed mRNAs were mainly\n",
      "      targeted to golgi apparatus (ontology: cellular component), catalytic activity\n",
      "      (ontology: molecular function), and mitotic nuclear division (ontology:\n",
      "      biological process), and were mostly enriched in glutathione metabolism\n",
      "      signaling. One hundred forty-seven lncRNAs were regarded as cis-regulatory.\n",
      "      Several groups of lncRNAs may participate in biological pathways related to DN\n",
      "      via trans-regulation of protein-coding genes. CONCLUSION: Hundreds of lncRNAs are\n",
      "      dysregulated in DN. These lncRNAs might be involved in the pathogenesis of DN by \n",
      "      modulating multiple molecular pathways. Our findings provide potential candidate \n",
      "      biomarkers for predicting or diagnosing DN.\n",
      "CI  - Copyright (c) 2016 Elsevier Inc. All rights reserved.\n",
      "FAU - Chen, Sheng\n",
      "AU  - Chen S\n",
      "AD  - Department of Endocrinology, The Second Affiliated Hospital of Nanjing Medical\n",
      "      University, Nanjing, 210000,China; Nanjing Medical University, Nanjing, 210029,\n",
      "      China.\n",
      "FAU - Dong, Chenglong\n",
      "AU  - Dong C\n",
      "AD  - Department of Endocrinology, Yancheng First People's Hospital, Yancheng, 224006, \n",
      "      China.\n",
      "FAU - Qian, Xiaoxiao\n",
      "AU  - Qian X\n",
      "AD  - Department of Endocrinology, The Second Affiliated Hospital of Nanjing Medical\n",
      "      University, Nanjing, 210000,China; Nanjing Medical University, Nanjing, 210029,\n",
      "      China.\n",
      "FAU - Huang, Shanshan\n",
      "AU  - Huang S\n",
      "AD  - Department of Endocrinology, Nanjing General Hospital of Nanjing Military\n",
      "      Command, Nanjing, 210002, China.\n",
      "FAU - Feng, Yamin\n",
      "AU  - Feng Y\n",
      "AD  - Department of Endocrinology, The Second Affiliated Hospital of Nanjing Medical\n",
      "      University, Nanjing, 210000,China.\n",
      "FAU - Ye, Xiaolong\n",
      "AU  - Ye X\n",
      "AD  - Department of Endocrinology, The Second Affiliated Hospital of Nanjing Medical\n",
      "      University, Nanjing, 210000,China.\n",
      "FAU - Miao, Heng\n",
      "AU  - Miao H\n",
      "AD  - Department of Endocrinology, The Second Affiliated Hospital of Nanjing Medical\n",
      "      University, Nanjing, 210000,China.\n",
      "FAU - You, Qiang\n",
      "AU  - You Q\n",
      "AD  - Department of Endocrinology, The Second Affiliated Hospital of Nanjing Medical\n",
      "      University, Nanjing, 210000,China.\n",
      "FAU - Lu, Yibing\n",
      "AU  - Lu Y\n",
      "AD  - Department of Endocrinology, The Second Affiliated Hospital of Nanjing Medical\n",
      "      University, Nanjing, 210000,China. Electronic address: luyibing2004@126.com.\n",
      "FAU - Ding, Dafa\n",
      "AU  - Ding D\n",
      "AD  - Department of Endocrinology, The Second Affiliated Hospital of Nanjing Medical\n",
      "      University, Nanjing, 210000,China. Electronic address: dingdafa2004@aliyun.com.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161208\n",
      "PL  - United States\n",
      "TA  - J Diabetes Complications\n",
      "JT  - Journal of diabetes and its complications\n",
      "JID - 9204583\n",
      "OTO - NOTNLM\n",
      "OT  - Biomarkers\n",
      "OT  - Diabetic nephropathy\n",
      "OT  - Long noncoding RNA\n",
      "OT  - MAPK\n",
      "OT  - Microarray analysis\n",
      "EDAT- 2016/12/23 06:00\n",
      "MHDA- 2016/12/23 06:00\n",
      "CRDT- 2016/12/24 06:00\n",
      "PHST- 2016/07/07 [received]\n",
      "PHST- 2016/11/17 [revised]\n",
      "PHST- 2016/11/25 [accepted]\n",
      "AID - S1056-8727(16)30976-X [pii]\n",
      "AID - 10.1016/j.jdiacomp.2016.11.017 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - J Diabetes Complications. 2016 Dec 8. pii: S1056-8727(16)30976-X. doi:\n",
      "      10.1016/j.jdiacomp.2016.11.017.\n",
      "\n",
      "PMID- 28002605\n",
      "OWN - NLM\n",
      "STAT- In-Process\n",
      "DA  - 20161221\n",
      "LR  - 20161222\n",
      "IS  - 1676-5680 (Electronic)\n",
      "IS  - 1676-5680 (Linking)\n",
      "VI  - 15\n",
      "IP  - 4\n",
      "DP  - 2016 Dec 19\n",
      "TI  - Knockdown of the putative Lifeguard homologue CG3814 in neurons of Drosophila\n",
      "      melanogaster.\n",
      "LID - 10.4238/gmr15049290 [doi]\n",
      "AB  - Lifeguard is an integral transmembrane protein that modulates FasL-mediated\n",
      "      apoptosis by interfering with the activation of caspase 8. It is evolutionarily\n",
      "      conserved, with homologues present in plants, nematodes, zebra fish, frog,\n",
      "      chicken, mouse, monkey, and human. The Lifeguard homologue in Drosophila, CG3814,\n",
      "      contains the Bax inhibitor-1 family motif of unknown function. Downregulation of \n",
      "      Lifeguard disrupts cellular homeostasis and disease by sensitizing neurons to\n",
      "      FasL-mediated apoptosis. We used bioinformatic analyses to identify CG3814, a\n",
      "      putative homologue of Lifeguard, and knocked down CG3814/LFG expression under the\n",
      "      control of the Dopa decarboxylase (Ddc-Gal4) transgene in Drosophila melanogaster\n",
      "      neurons to investigate whether it possesses neuroprotective activity. Knockdown\n",
      "      of CG3814/LFG in Ddc-Gal4-expressing neurons resulted in a shortened lifespan and\n",
      "      impaired locomotor ability, phenotypes that are strongly associated with the\n",
      "      degeneration and loss of dopaminergic neurons. Lifeguard interacts with\n",
      "      anti-apoptotic Bcl-2 proteins and possibly pro-apoptotic proteins to exert its\n",
      "      neuroprotective function. The co-expression of Buffy, the sole anti-apoptotic\n",
      "      Bcl-2 gene family member in Drosophila, and CG3814/LFG by stable inducible RNA\n",
      "      interference, suppresses the shortened lifespan and the premature age-dependent\n",
      "      loss in climbing ability. Suppression of CG3814/LFG in the Drosophila eye reduces\n",
      "      the number of ommatidia and increases disruption of the ommatidial array.\n",
      "      Overexpression of Buffy, along with the knockdown of CG3814/LFG, counteracts the \n",
      "      eye phenotypes. Knockdown of CG3814/LFG in Ddc-Gal4-expressing neurons in\n",
      "      Drosophila diminishes its neuroprotective ability and results in a shortened\n",
      "      lifespan and loss of climbing ability, phenotypes that are improved upon\n",
      "      overexpression of the pro-survival Buffy.\n",
      "FAU - M'Angale, P G\n",
      "AU  - M'Angale PG\n",
      "AD  - Department of Biology, Memorial University of Newfoundland, St. John's,\n",
      "      Newfoundland & Labrador, Canada.\n",
      "FAU - Staveley, B E\n",
      "AU  - Staveley BE\n",
      "AD  - Department of Biology, Memorial University of Newfoundland, St. John's,\n",
      "      Newfoundland & Labrador, Canada bestave@mun.ca.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161219\n",
      "PL  - Brazil\n",
      "TA  - Genet Mol Res\n",
      "JT  - Genetics and molecular research : GMR\n",
      "JID - 101169387\n",
      "EDAT- 2016/12/22 06:00\n",
      "MHDA- 2016/12/22 06:00\n",
      "CRDT- 2016/12/22 06:00\n",
      "AID - gmr-15-04-gmr.15049290 [pii]\n",
      "AID - 10.4238/gmr15049290 [doi]\n",
      "PST - epublish\n",
      "SO  - Genet Mol Res. 2016 Dec 19;15(4). doi: 10.4238/gmr15049290.\n",
      "\n",
      "PMID- 28002126\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "DA  - 20161221\n",
      "LR  - 20161222\n",
      "IS  - 1536-4844 (Electronic)\n",
      "IS  - 1078-0998 (Linking)\n",
      "VI  - 23\n",
      "IP  - 1\n",
      "DP  - 2017 Jan\n",
      "TI  - Stress Response Protein RBM3 Promotes the Development of Colitis-associated\n",
      "      Cancer.\n",
      "PG  - 66-74\n",
      "LID - 10.1097/MIB.0000000000000987 [doi]\n",
      "AB  - BACKGROUND: Colitis-associated cancer (CAC) is caused by chronic intestinal\n",
      "      inflammation and often results from refractory inflammatory bowel disease (IBD). \n",
      "      Stress response proteins Cirp and heat shock protein A4 are involved in the\n",
      "      refractory clinical course and development of CAC. RNA-binding motif protein 3\n",
      "      (RBM3) is induced in response to various stresses and is upregulated in several\n",
      "      cancers. However, the role of RBM3 in CAC is unclear. METHODS: We assessed RBM3\n",
      "      expression and function in 263 human intestinal mucosa samples from patients with\n",
      "      IBD and in Rbm3-deficient (Rbm3) mice. RESULTS: Expression of RBM3 was correlated\n",
      "      with the expression of stress response proteins Cirp, heat shock protein A4, and \n",
      "      HSP27 in the colonic mucosa of patients with IBD. Significant correlation was\n",
      "      observed between the expression of RBM3 and that of Bcl-xL or stem cell markers. \n",
      "      RBM3 expression increased and significantly correlated with R-spondin expression \n",
      "      in the colonic mucosa of patients with refractory IBD, a condition associated\n",
      "      with increased cancer risk, and RBM3 was overexpressed in human CACs. In the\n",
      "      murine CAC model, Rbm3 deficiency decreased R-spondin and Bcl-xL expression and\n",
      "      increased apoptotic cell number in the colonic mucosa, leading to reduced tumor\n",
      "      multiplicity. Transplantation of wild-type and Rbm3 bone marrow did not alter\n",
      "      tumor burden, indicating the importance of RBM3 in epithelial cells. CONCLUSIONS:\n",
      "      Our findings indicated that RBM3 was required for efficient inflammatory\n",
      "      carcinogenesis in the murine CAC model and suggested that RBM3 could be a\n",
      "      predictive biomarker of CAC risk and a new therapeutic target for cancer\n",
      "      prevention in patients with IBD.\n",
      "FAU - Sakurai, Toshiharu\n",
      "AU  - Sakurai T\n",
      "AD  - *Department of Gastroenterology and Hepatology, Faculty of Medicine, Kindai\n",
      "      University, Osaka-Sayama, Japan; daggerDepartment of Clinical Molecular Biology, \n",
      "      Graduate School of Medicine, Kyoto University, Kyoto, Japan; and double\n",
      "      daggerKaifukuki Rehabilitation Ward, Biwako-Chuo Hospital, Otsu, Japan.\n",
      "FAU - Kashida, Hiroshi\n",
      "AU  - Kashida H\n",
      "FAU - Komeda, Yoriaki\n",
      "AU  - Komeda Y\n",
      "FAU - Nagai, Tomoyuki\n",
      "AU  - Nagai T\n",
      "FAU - Hagiwara, Satoru\n",
      "AU  - Hagiwara S\n",
      "FAU - Watanabe, Tomohiro\n",
      "AU  - Watanabe T\n",
      "FAU - Kitano, Masayuki\n",
      "AU  - Kitano M\n",
      "FAU - Nishida, Naoshi\n",
      "AU  - Nishida N\n",
      "FAU - Fujita, Jun\n",
      "AU  - Fujita J\n",
      "FAU - Kudo, Masatoshi\n",
      "AU  - Kudo M\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PL  - United States\n",
      "TA  - Inflamm Bowel Dis\n",
      "JT  - Inflammatory bowel diseases\n",
      "JID - 9508162\n",
      "EDAT- 2016/12/22 06:00\n",
      "MHDA- 2016/12/22 06:00\n",
      "CRDT- 2016/12/22 06:00\n",
      "AID - 10.1097/MIB.0000000000000987 [doi]\n",
      "AID - 00054725-201701000-00008 [pii]\n",
      "PST - ppublish\n",
      "SO  - Inflamm Bowel Dis. 2017 Jan;23(1):66-74. doi: 10.1097/MIB.0000000000000987.\n",
      "\n",
      "PMID- 27998767\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161221\n",
      "LR  - 20161224\n",
      "IS  - 1090-2104 (Electronic)\n",
      "IS  - 0006-291X (Linking)\n",
      "DP  - 2016 Dec 18\n",
      "TI  - Maternal exosomes in diabetes contribute to the cardiac development deficiency.\n",
      "LID - S0006-291X(16)32142-8 [pii]\n",
      "LID - 10.1016/j.bbrc.2016.12.097 [doi]\n",
      "AB  - Maternal diabetes mellitus induces an increased risk of congenital heart defects \n",
      "      (CHD), however, the exact mechanisms are still not fully illustrated. In this\n",
      "      study, diabetic pregnant C57BL/6 mice were induced by injection of streptozotocin\n",
      "      before mating. Compared with the control normal mice, diabetic pregnant mice\n",
      "      displayed significant changes of the exosomal miRNA contents in the blood, as\n",
      "      revealed by RNA-seq analysis. Multiple of these miRNAs were found involved in\n",
      "      cardiac development regulation. Moreover, fluorescence labeled exosomes and gold \n",
      "      nanoparticles could cross the placenta barrier and infiltrated into the embryonic\n",
      "      organs/tissues, including the heart, during embryonic development. Injection of\n",
      "      diabetic maternal exosomes strikingly increased the risk of CHD in the normal\n",
      "      recipient pregnant mice. Taken together, we could draw the conclusion that\n",
      "      maternal exosomes in diabetes could cross the maternal-fetal barrier and\n",
      "      contribute to the cardiac development deficiency possibly via miRNAs, providing\n",
      "      new insights in CHD prevention and treatment.\n",
      "CI  - Copyright (c) 2016 Elsevier Inc. All rights reserved.\n",
      "FAU - Shi, Ruijing\n",
      "AU  - Shi R\n",
      "AD  - Department of Ultrasound Diagnostics, Tangdu Hospital, Fourth Military Medical\n",
      "      University, Xi'an, China; Department of Biochemistry and Molecular Biology,\n",
      "      Fourth Military Medical University, Xi'an, China.\n",
      "FAU - Zhao, Lianbi\n",
      "AU  - Zhao L\n",
      "AD  - Department of Ultrasound Diagnostics, Tangdu Hospital, Fourth Military Medical\n",
      "      University, Xi'an, China; Department of Biochemistry and Molecular Biology,\n",
      "      Fourth Military Medical University, Xi'an, China.\n",
      "FAU - Cai, Wenbin\n",
      "AU  - Cai W\n",
      "AD  - Department of Ultrasound Diagnostics, Tangdu Hospital, Fourth Military Medical\n",
      "      University, Xi'an, China; Department of Biochemistry and Molecular Biology,\n",
      "      Fourth Military Medical University, Xi'an, China.\n",
      "FAU - Wei, Mengying\n",
      "AU  - Wei M\n",
      "AD  - Department of Biochemistry and Molecular Biology, Fourth Military Medical\n",
      "      University, Xi'an, China.\n",
      "FAU - Zhou, Xueying\n",
      "AU  - Zhou X\n",
      "AD  - Department of Ultrasound Diagnostics, Tangdu Hospital, Fourth Military Medical\n",
      "      University, Xi'an, China; Department of Biochemistry and Molecular Biology,\n",
      "      Fourth Military Medical University, Xi'an, China.\n",
      "FAU - Yang, Guodong\n",
      "AU  - Yang G\n",
      "AD  - Department of Biochemistry and Molecular Biology, Fourth Military Medical\n",
      "      University, Xi'an, China. Electronic address: yanggd@fmmu.edu.cn.\n",
      "FAU - Yuan, Lijun\n",
      "AU  - Yuan L\n",
      "AD  - Department of Ultrasound Diagnostics, Tangdu Hospital, Fourth Military Medical\n",
      "      University, Xi'an, China. Electronic address: yuanlj@fmmu.edu.cn.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161218\n",
      "PL  - United States\n",
      "TA  - Biochem Biophys Res Commun\n",
      "JT  - Biochemical and biophysical research communications\n",
      "JID - 0372516\n",
      "OTO - NOTNLM\n",
      "OT  - Congenital heart disease\n",
      "OT  - Exosomes\n",
      "OT  - Maternal diabetes mellitus\n",
      "OT  - miRNAs\n",
      "EDAT- 2016/12/22 06:00\n",
      "MHDA- 2016/12/22 06:00\n",
      "CRDT- 2016/12/22 06:00\n",
      "PHST- 2016/12/02 [received]\n",
      "PHST- 2016/12/14 [accepted]\n",
      "AID - S0006-291X(16)32142-8 [pii]\n",
      "AID - 10.1016/j.bbrc.2016.12.097 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Biochem Biophys Res Commun. 2016 Dec 18. pii: S0006-291X(16)32142-8. doi:\n",
      "      10.1016/j.bbrc.2016.12.097.\n",
      "\n",
      "PMID- 27995765\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161220\n",
      "LR  - 20161221\n",
      "IS  - 1582-4934 (Electronic)\n",
      "IS  - 1582-1838 (Linking)\n",
      "DP  - 2016 Dec 20\n",
      "TI  - Shockwaves prevent from heart failure after acute myocardial ischaemia via\n",
      "      RNA/protein complexes.\n",
      "LID - 10.1111/jcmm.13021 [doi]\n",
      "AB  - Shock wave treatment (SWT) was shown to induce regeneration of ischaemic\n",
      "      myocardium via Toll-like receptor 3 (TLR3). The antimicrobial peptide LL37 gets\n",
      "      released by mechanical stress and is known to form complexes with nucleic acids\n",
      "      thus activating Toll-like receptors. We suggested that SWT in the acute setting\n",
      "      prevents from the development of heart failure via RNA/protein release.\n",
      "      Myocardial infarction in mice was induced followed by subsequent SWT. Heart\n",
      "      function was assessed 4 weeks later via transthoracic echocardiography and\n",
      "      pressure-volume measurements. Human umbilical vein endothelial cells (HUVECs)\n",
      "      were treated with SWT in the presence of RNase and proteinase and analysed for\n",
      "      proliferation, tube formation and LL37 expression. RNA release and uptake after\n",
      "      SWT was evaluated. We found significantly improved cardiac function after SWT.\n",
      "      SWT resulted in significantly higher numbers of capillaries and arterioles and\n",
      "      less left ventricular fibrosis. Supernatants of treated cells activated TLR3\n",
      "      reporter cells. Analysis of the supernatant revealed increased RNA levels. The\n",
      "      effect could not be abolished by pre-treatment of the supernatant with RNase, but\n",
      "      only by a sequential digestion with proteinase and RNase hinting strongly towards\n",
      "      the involvement of RNA/protein complexes. Indeed, LL37 expression as well as\n",
      "      cellular RNA uptake were significantly increased after SWT. We show for the first\n",
      "      time that SWT prevents from left ventricular remodelling and cardiac dysfunction \n",
      "      via RNA/protein complex release and subsequent induction of angiogenesis. It\n",
      "      might therefore develop a potent regenerative treatment alternative for ischaemic\n",
      "      heart disease.\n",
      "CI  - (c) 2016 The Authors. Journal of Cellular and Molecular Medicine published by\n",
      "      John Wiley & Sons Ltd and Foundation for Cellular and Molecular Medicine.\n",
      "FAU - Tepekoylu, Can\n",
      "AU  - Tepekoylu C\n",
      "AD  - Department for Cardiac Surgery, Medical University of Innsbruck, Innsbruck,\n",
      "      Austria.\n",
      "FAU - Primessnig, Uwe\n",
      "AU  - Primessnig U\n",
      "AD  - Department of Cardiology, Charite University Berlin, Berlin, Germany.\n",
      "FAU - Polzl, Leo\n",
      "AU  - Polzl L\n",
      "AD  - Department for Cardiac Surgery, Medical University of Innsbruck, Innsbruck,\n",
      "      Austria.\n",
      "FAU - Graber, Michael\n",
      "AU  - Graber M\n",
      "AD  - Department for Cardiac Surgery, Medical University of Innsbruck, Innsbruck,\n",
      "      Austria.\n",
      "FAU - Lobenwein, Daniela\n",
      "AU  - Lobenwein D\n",
      "AD  - Department for Cardiac Surgery, Medical University of Innsbruck, Innsbruck,\n",
      "      Austria.\n",
      "FAU - Nagele, Felix\n",
      "AU  - Nagele F\n",
      "AD  - Department for Cardiac Surgery, Medical University of Innsbruck, Innsbruck,\n",
      "      Austria.\n",
      "FAU - Kirchmair, Elke\n",
      "AU  - Kirchmair E\n",
      "AD  - Department for Cardiac Surgery, Medical University of Innsbruck, Innsbruck,\n",
      "      Austria.\n",
      "FAU - Pechriggl, Elisabeth\n",
      "AU  - Pechriggl E\n",
      "AD  - Division of Clinical and Functional Anatomy, Department of Anatomy, Histology and\n",
      "      Embryology, Innsbruck Medical University, Innsbruck, Austria.\n",
      "FAU - Grimm, Michael\n",
      "AU  - Grimm M\n",
      "AD  - Department for Cardiac Surgery, Medical University of Innsbruck, Innsbruck,\n",
      "      Austria.\n",
      "FAU - Holfeld, Johannes\n",
      "AU  - Holfeld J\n",
      "AD  - Department for Cardiac Surgery, Medical University of Innsbruck, Innsbruck,\n",
      "      Austria.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161220\n",
      "PL  - England\n",
      "TA  - J Cell Mol Med\n",
      "JT  - Journal of cellular and molecular medicine\n",
      "JID - 101083777\n",
      "OTO - NOTNLM\n",
      "OT  - inflammation\n",
      "OT  - myocardial infarction\n",
      "OT  - myocardial regeneration\n",
      "OT  - shock wave therapy\n",
      "EDAT- 2016/12/21 06:00\n",
      "MHDA- 2016/12/21 06:00\n",
      "CRDT- 2016/12/21 06:00\n",
      "PHST- 2016/06/14 [received]\n",
      "PHST- 2016/09/27 [accepted]\n",
      "AID - 10.1111/jcmm.13021 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - J Cell Mol Med. 2016 Dec 20. doi: 10.1111/jcmm.13021.\n",
      "\n",
      "PMID- 27981604\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161216\n",
      "LR  - 20161217\n",
      "IS  - 1527-3350 (Electronic)\n",
      "IS  - 0270-9139 (Linking)\n",
      "DP  - 2016 Dec 16\n",
      "TI  - Malondialdehyde epitopes are sterile mediators of hepatic inflammation in\n",
      "      hypercholesterolemic mice.\n",
      "LID - 10.1002/hep.28970 [doi]\n",
      "AB  - BACKGROUND AND RATIONALE: Diet-related health issues such as non-alcoholic fatty \n",
      "      liver disease and cardiovascular disorders are known to have a major inflammatory\n",
      "      component. However, the exact pathways linking diet-induced changes e.g.\n",
      "      hyperlipidemia, and the ensuing inflammation have remained elusive so far. MAIN\n",
      "      RESULTS: We identified biological processes related to innate immunity and\n",
      "      oxidative stress as prime response pathways in livers of Ldlr-/- mice on\n",
      "      Western-type diet using RNA sequencing and in silico functional analyses of\n",
      "      transcriptome data. The observed changes were independent of the presence of\n",
      "      microbiota and thus indicative of a role for sterile triggers. We further show\n",
      "      that malondialdehyde (MDA) epitopes, products of lipid peroxidation and markers\n",
      "      for enhanced oxidative stress, are detectable in hepatic inflammation\n",
      "      predominantly on dying cells and stimulate cytokine secretion as well as\n",
      "      leukocyte recruitment in vitro and in vivo. MDA-induced cytokine secretion in\n",
      "      vitro was dependent on the presence of scavenger receptors CD36 and MSR1.\n",
      "      Moreover, in vivo neutralization of endogenously generated MDA epitopes by\n",
      "      intravenous injection of a specific MDA antibody results in decreased hepatic\n",
      "      inflammation in Ldlr-/- mice on Western-type diet. CONCLUSIONS: Accumulation of\n",
      "      MDA epitopes plays a major role during diet-induced hepatic inflammation and can \n",
      "      be ameliorated by administration of an anti-MDA antibody. This article is\n",
      "      protected by copyright. All rights reserved.\n",
      "CI  - (c) 2016 by the American Association for the Study of Liver Diseases.\n",
      "FAU - Busch, Clara Jana-Lui\n",
      "AU  - Busch CJ\n",
      "AD  - Department of Laboratory Medicine, Medical University of Vienna, Vienna, Austria.\n",
      "AD  - CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, \n",
      "      Vienna, Austria.\n",
      "FAU - Hendrikx, Tim\n",
      "AU  - Hendrikx T\n",
      "AD  - Department of Laboratory Medicine, Medical University of Vienna, Vienna, Austria.\n",
      "AD  - CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, \n",
      "      Vienna, Austria.\n",
      "AD  - Department of Molecular Genetics, School of Nutrition and Translational Research \n",
      "      in Metabolism (NUTRIM), Maastricht University, Maastricht, The Netherlands.\n",
      "FAU - Weismann, David\n",
      "AU  - Weismann D\n",
      "AD  - Department of Laboratory Medicine, Medical University of Vienna, Vienna, Austria.\n",
      "AD  - CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, \n",
      "      Vienna, Austria.\n",
      "FAU - Jackel, Sven\n",
      "AU  - Jackel S\n",
      "AD  - Center for Thrombosis and Haemostasis (CTH), University Medical Center Mainz,\n",
      "      Mainz, Germany.\n",
      "AD  - German Center for Cardiovascular Research (DZHK), Partner Site Rhein/Main, Mainz,\n",
      "      Germany.\n",
      "FAU - Walenbergh, Sofie M A\n",
      "AU  - Walenbergh SM\n",
      "AD  - Department of Molecular Genetics, School of Nutrition and Translational Research \n",
      "      in Metabolism (NUTRIM), Maastricht University, Maastricht, The Netherlands.\n",
      "FAU - Rendeiro, Andre F\n",
      "AU  - Rendeiro AF\n",
      "AD  - CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, \n",
      "      Vienna, Austria.\n",
      "FAU - Weisser, Juliane\n",
      "AU  - Weisser J\n",
      "AD  - CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, \n",
      "      Vienna, Austria.\n",
      "FAU - Puhm, Florian\n",
      "AU  - Puhm F\n",
      "AD  - Department of Laboratory Medicine, Medical University of Vienna, Vienna, Austria.\n",
      "AD  - CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, \n",
      "      Vienna, Austria.\n",
      "FAU - Hladik, Anastasiya\n",
      "AU  - Hladik A\n",
      "AD  - CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, \n",
      "      Vienna, Austria.\n",
      "AD  - Laboratory of Infection Biology, Department of Medicine 1, Medical University of \n",
      "      Vienna, Vienna, Austria.\n",
      "FAU - Goderle, Laura\n",
      "AU  - Goderle L\n",
      "AD  - Department of Laboratory Medicine, Medical University of Vienna, Vienna, Austria.\n",
      "AD  - CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, \n",
      "      Vienna, Austria.\n",
      "FAU - Papac-Milicevic, Nikolina\n",
      "AU  - Papac-Milicevic N\n",
      "AD  - Department of Laboratory Medicine, Medical University of Vienna, Vienna, Austria.\n",
      "AD  - CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, \n",
      "      Vienna, Austria.\n",
      "FAU - Haas, Gerald\n",
      "AU  - Haas G\n",
      "AD  - Department of Laboratory Medicine, Medical University of Vienna, Vienna, Austria.\n",
      "AD  - CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, \n",
      "      Vienna, Austria.\n",
      "FAU - Millischer, Vincent\n",
      "AU  - Millischer V\n",
      "AD  - Department of Laboratory Medicine, Medical University of Vienna, Vienna, Austria.\n",
      "AD  - CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, \n",
      "      Vienna, Austria.\n",
      "FAU - Subramaniam, Saravanan\n",
      "AU  - Subramaniam S\n",
      "AD  - Center for Thrombosis and Haemostasis (CTH), University Medical Center Mainz,\n",
      "      Mainz, Germany.\n",
      "FAU - Knapp, Sylvia\n",
      "AU  - Knapp S\n",
      "AD  - CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, \n",
      "      Vienna, Austria.\n",
      "AD  - Laboratory of Infection Biology, Department of Medicine 1, Medical University of \n",
      "      Vienna, Vienna, Austria.\n",
      "FAU - Bennett, Keiryn L\n",
      "AU  - Bennett KL\n",
      "AD  - CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, \n",
      "      Vienna, Austria.\n",
      "FAU - Bock, Christoph\n",
      "AU  - Bock C\n",
      "AD  - Department of Laboratory Medicine, Medical University of Vienna, Vienna, Austria.\n",
      "AD  - CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, \n",
      "      Vienna, Austria.\n",
      "AD  - Max Planck Institute for Informatics, 66123, Saarbrucken, Germany.\n",
      "FAU - Reinhardt, Christoph\n",
      "AU  - Reinhardt C\n",
      "AD  - Center for Thrombosis and Haemostasis (CTH), University Medical Center Mainz,\n",
      "      Mainz, Germany.\n",
      "AD  - German Center for Cardiovascular Research (DZHK), Partner Site Rhein/Main, Mainz,\n",
      "      Germany.\n",
      "FAU - Shiri-Sverdlov, Ronit\n",
      "AU  - Shiri-Sverdlov R\n",
      "AD  - Department of Molecular Genetics, School of Nutrition and Translational Research \n",
      "      in Metabolism (NUTRIM), Maastricht University, Maastricht, The Netherlands.\n",
      "FAU - Binder, Christoph J\n",
      "AU  - Binder CJ\n",
      "AD  - Department of Laboratory Medicine, Medical University of Vienna, Vienna, Austria.\n",
      "AD  - CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, \n",
      "      Vienna, Austria.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161216\n",
      "PL  - United States\n",
      "TA  - Hepatology\n",
      "JT  - Hepatology (Baltimore, Md.)\n",
      "JID - 8302946\n",
      "OTO - NOTNLM\n",
      "OT  - dyslipidemia\n",
      "OT  - lipid peroxidation\n",
      "OT  - natural IgM antibodies\n",
      "OT  - oxidative stress\n",
      "EDAT- 2016/12/17 06:00\n",
      "MHDA- 2016/12/17 06:00\n",
      "CRDT- 2016/12/17 06:00\n",
      "PHST- 2016/06/24 [received]\n",
      "PHST- 2016/10/31 [revised]\n",
      "PHST- 2016/11/21 [accepted]\n",
      "AID - 10.1002/hep.28970 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Hepatology. 2016 Dec 16. doi: 10.1002/hep.28970.\n",
      "\n",
      "PMID- 27981238\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "DA  - 20161216\n",
      "LR  - 20161219\n",
      "IS  - 2379-5042 (Linking)\n",
      "VI  - 1\n",
      "IP  - 6\n",
      "DP  - 2016 Nov-Dec\n",
      "TI  - New Insights into Autoinducer-2 Signaling as a Virulence Regulator in a Mouse\n",
      "      Model of Pneumonic Plague.\n",
      "LID - e00342-16 [pii]\n",
      "AB  - The Enterobacteriaceae family members, including the infamous Yersinia pestis,\n",
      "      the causative agent of plague, have a highly conserved interbacterial signaling\n",
      "      system that is mediated by the autoinducer-2 (AI-2) quorum-sensing molecule. The \n",
      "      AI-2 system is implicated in regulating various bacterial virulence genes in\n",
      "      diverse environmental niches. Deletion of the gene encoding the synthetic enzyme \n",
      "      for the AI-2 substrate, luxS, leads to either no significant change or,\n",
      "      paradoxically, an increase in in vivo bacterial virulence. We showed that\n",
      "      deletion of the rbsA and lsrA genes, components of ABC transport systems that\n",
      "      interact with AI-2, synergistically disrupted AI-2 signaling patterns and\n",
      "      resulted in a more-than-50-fold decrease in Y. pestis strain CO92 virulence in a \n",
      "      stringent pneumonic plague mouse model. Deletion of luxS or lsrK (encoding AI-2\n",
      "      kinase) from the DeltarbsA DeltalsrA background strain or complementation of the \n",
      "      DeltarbsA DeltalsrA mutant with the corresponding gene(s) reverted the virulence \n",
      "      phenotype to that of the wild-type Y. pestis CO92. Furthermore, the\n",
      "      administration of synthetic AI-2 in mice infected with the DeltarbsA DeltalsrA\n",
      "      DeltaluxS mutant strain attenuated this triple mutant to a virulence phenotype\n",
      "      similar to that of the DeltarbsA DeltalsrA strain in a pneumonic plague model.\n",
      "      Conversely, the administration of AI-2 to mice infected with the DeltarbsA\n",
      "      DeltalsrA DeltaluxS DeltalsrK mutant did not rescue animals from lethality,\n",
      "      indicating the importance of the AI-2-LsrK axis in regulating bacterial\n",
      "      virulence. By performing high-throughput RNA sequencing, the potential role of\n",
      "      some AI-2-signaling-regulated genes that modulated bacterial virulence was\n",
      "      determined. We anticipate that the characterization of AI-2 signaling in Y.\n",
      "      pestis will lead to reexamination of AI-2 systems in other pathogens and that\n",
      "      AI-2 signaling may represent a broad-spectrum therapeutic target to combat\n",
      "      antibiotic-resistant bacteria, which represent a global crisis of the 21st\n",
      "      century. IMPORTANCEYersinia pestis is the bacterial agent that causes the highly \n",
      "      fatal disease plague. The organism represents a significant concern because of\n",
      "      its potential use as a bioterror agent, beyond the several thousand naturally\n",
      "      occurring human infection cases occurring globally each year. While there has\n",
      "      been development of effective antibiotics, the narrow therapeutic window and\n",
      "      challenges posed by the existence of antibiotic-resistant strains represent\n",
      "      serious concerns. We sought to identify novel virulence factors that could\n",
      "      potentially be incorporated into an attenuated vaccine platform or be targeted by\n",
      "      novel therapeutics. We show here that a highly conserved quorum-sensing system,\n",
      "      autoinducer-2, significantly affected the virulence of Y. pestis in a mouse model\n",
      "      of pneumonic plague. We also identified steps in autoinducer-2 signaling which\n",
      "      had confounded previous studies and demonstrated the potential for intervention\n",
      "      in the virulence mechanism(s) of autoinducer-2. Our findings may have an impact\n",
      "      on bacterial pathogenesis research in many other organisms and could result in\n",
      "      identifying potential broad-spectrum therapeutic targets to combat\n",
      "      antibiotic-resistant bacteria, which represent a global crisis of the 21st\n",
      "      century.\n",
      "FAU - Fitts, Eric C\n",
      "AU  - Fitts EC\n",
      "AD  - Department of Microbiology and Immunology, University of Texas Medical Branch at \n",
      "      Galveston, Galveston, Texas, USA.\n",
      "FAU - Andersson, Jourdan A\n",
      "AU  - Andersson JA\n",
      "AD  - Department of Microbiology and Immunology, University of Texas Medical Branch at \n",
      "      Galveston, Galveston, Texas, USA.\n",
      "FAU - Kirtley, Michelle L\n",
      "AU  - Kirtley ML\n",
      "AD  - Department of Microbiology and Immunology, University of Texas Medical Branch at \n",
      "      Galveston, Galveston, Texas, USA.\n",
      "FAU - Sha, Jian\n",
      "AU  - Sha J\n",
      "AD  - Department of Microbiology and Immunology, University of Texas Medical Branch at \n",
      "      Galveston, Galveston, Texas, USA.\n",
      "FAU - Erova, Tatiana E\n",
      "AU  - Erova TE\n",
      "AD  - Department of Microbiology and Immunology, University of Texas Medical Branch at \n",
      "      Galveston, Galveston, Texas, USA.\n",
      "FAU - Chauhan, Sadhana\n",
      "AU  - Chauhan S\n",
      "AD  - Department of Microbiology and Immunology, University of Texas Medical Branch at \n",
      "      Galveston, Galveston, Texas, USA.\n",
      "FAU - Motin, Vladimir L\n",
      "AU  - Motin VL\n",
      "AD  - Department of Microbiology and Immunology, University of Texas Medical Branch at \n",
      "      Galveston, Galveston, Texas, USA; Department of Pathology, University of Texas\n",
      "      Medical Branch at Galveston, Galveston, Texas, USA.\n",
      "FAU - Chopra, Ashok K\n",
      "AU  - Chopra AK\n",
      "AD  - Department of Microbiology and Immunology, University of Texas Medical Branch at \n",
      "      Galveston, Galveston, Texas, USA.\n",
      "LA  - eng\n",
      "GR  - R01 AI064389/AI/NIAID NIH HHS/United States\n",
      "GR  - T32 AI060549/AI/NIAID NIH HHS/United States\n",
      "GR  - UC7 AI070083/AI/NIAID NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "DEP - 20161214\n",
      "PL  - United States\n",
      "TA  - mSphere\n",
      "JT  - mSphere\n",
      "JID - 101674533\n",
      "PMC - PMC5156673\n",
      "OTO - NOTNLM\n",
      "OT  - Yersinia pestis\n",
      "OT  - animal models\n",
      "OT  - autoinducer-2\n",
      "OT  - pneumonic plague\n",
      "OT  - quorum sensing\n",
      "OT  - transcriptomics\n",
      "OT  - virulence\n",
      "EDAT- 2016/12/17 06:00\n",
      "MHDA- 2016/12/17 06:00\n",
      "CRDT- 2016/12/17 06:00\n",
      "PHST- 2016/11/18 [received]\n",
      "PHST- 2016/11/19 [accepted]\n",
      "AID - 10.1128/mSphere.00342-16 [doi]\n",
      "AID - mSphere00342-16 [pii]\n",
      "PST - epublish\n",
      "SO  - mSphere. 2016 Dec 14;1(6). pii: e00342-16. eCollection 2016 Nov-Dec.\n",
      "\n",
      "PMID- 27976428\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161215\n",
      "LR  - 20161216\n",
      "IS  - 1365-2184 (Electronic)\n",
      "IS  - 0960-7722 (Linking)\n",
      "DP  - 2016 Dec 14\n",
      "TI  - Long intragenic non-coding RNA lincRNA-p21 suppresses development of human\n",
      "      prostate cancer.\n",
      "LID - 10.1111/cpr.12318 [doi]\n",
      "AB  - OBJECTIVES: Prostate cancer is one of the most frequent malignancies in men,\n",
      "      worldwide, although its underlying mechanisms are not fully understood. Long\n",
      "      non-coding RNAs participate in development of human cancers. In this\n",
      "      invetsigation, we aimed to study the roles of lincRNA-p21 in development of human\n",
      "      prostate cancer. MATERIALS AND METHODS: Expression of lincRNA-p21 was assessed by\n",
      "      real-time PCR in cell lines and in human tissues. Lentivirus carrying\n",
      "      sh-lincRNA-p21, lincRNA-p21 or control constructs were used to determine their\n",
      "      effects on cell proliferation and apoptosis. A mouse xenograft model was employed\n",
      "      to explore the functions of lincRNA-p21 on cancer cell population growth in vivo.\n",
      "      Relationships between p53 downstream genes and lincRNA-p21 levels were explored\n",
      "      by real-time PCR, western blotting and chromatin immunoprecipitation. RESULTS:\n",
      "      LincRNA-p21 was found to be down-regulated in human prostate cancer, and low\n",
      "      levels of lincRNA-p21 correlated with high disease stage and prediction of poor\n",
      "      survival. We further showed that lincRNA-p21 inhibited prostate cancer cell\n",
      "      proliferation and colony formation in vitro and reduced rate of prostate cancer\n",
      "      cell population growth in vivo. Study of mechanisms involved revealed that\n",
      "      lincRNA-p21 promoted apoptosis and induced expression of p53 downstream genes by \n",
      "      regulating p53 binding to their promoters. Finally, we showed that expression of \n",
      "      p53 downstream genes was reduced in the malignant prostate tissues, which\n",
      "      correlated with lincRNA-p21 level. CONCLUSIONS: Our findings indicated that\n",
      "      lincRNA-p21 inhibited development of human prostate cancer partly by regulating\n",
      "      p53 downstream gene expression and partly by apoptotic activation.\n",
      "CI  - (c) 2016 John Wiley & Sons Ltd.\n",
      "FAU - Wang, Xiaohai\n",
      "AU  - Wang X\n",
      "AD  - Department of Urology, Shanghai General Hospital, School of Medicine, Shanghai\n",
      "      Jiao Tong University, Shanghai, China.\n",
      "FAU - Ruan, Yuan\n",
      "AU  - Ruan Y\n",
      "AD  - Department of Urology, Shanghai General Hospital, School of Medicine, Shanghai\n",
      "      Jiao Tong University, Shanghai, China.\n",
      "FAU - Wang, Xingjie\n",
      "AU  - Wang X\n",
      "AD  - Department of Urology, Shanghai General Hospital, School of Medicine, Shanghai\n",
      "      Jiao Tong University, Shanghai, China.\n",
      "FAU - Zhao, Wei\n",
      "AU  - Zhao W\n",
      "AD  - Department of Urology, Shanghai General Hospital, School of Medicine, Shanghai\n",
      "      Jiao Tong University, Shanghai, China.\n",
      "FAU - Jiang, Qi\n",
      "AU  - Jiang Q\n",
      "AD  - Department of Urology, Shanghai General Hospital, School of Medicine, Shanghai\n",
      "      Jiao Tong University, Shanghai, China.\n",
      "FAU - Jiang, Chenyi\n",
      "AU  - Jiang C\n",
      "AD  - Department of Urology, Shanghai General Hospital, School of Medicine, Shanghai\n",
      "      Jiao Tong University, Shanghai, China.\n",
      "FAU - Zhao, Yuyang\n",
      "AU  - Zhao Y\n",
      "AD  - Department of Urology, Shanghai General Hospital, School of Medicine, Shanghai\n",
      "      Jiao Tong University, Shanghai, China.\n",
      "FAU - Xu, Yongzhi\n",
      "AU  - Xu Y\n",
      "AD  - Department of Urology, Shanghai General Hospital, School of Medicine, Shanghai\n",
      "      Jiao Tong University, Shanghai, China.\n",
      "FAU - Sun, Feng\n",
      "AU  - Sun F\n",
      "AD  - Department of Urology, Shanghai General Hospital, School of Medicine, Shanghai\n",
      "      Jiao Tong University, Shanghai, China.\n",
      "FAU - Zhu, Yiping\n",
      "AU  - Zhu Y\n",
      "AD  - Department of Urology, Shanghai General Hospital, School of Medicine, Shanghai\n",
      "      Jiao Tong University, Shanghai, China.\n",
      "FAU - Xia, Shujie\n",
      "AU  - Xia S\n",
      "AD  - Department of Urology, Shanghai General Hospital, School of Medicine, Shanghai\n",
      "      Jiao Tong University, Shanghai, China.\n",
      "FAU - Xu, Dongliang\n",
      "AU  - Xu D\n",
      "AD  - Department of Urology, Shanghai General Hospital, School of Medicine, Shanghai\n",
      "      Jiao Tong University, Shanghai, China.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161214\n",
      "PL  - England\n",
      "TA  - Cell Prolif\n",
      "JT  - Cell proliferation\n",
      "JID - 9105195\n",
      "OTO - NOTNLM\n",
      "OT  - apoptosis\n",
      "OT  - lincRNA-p21\n",
      "OT  - p53\n",
      "OT  - prostate cancer\n",
      "EDAT- 2016/12/16 06:00\n",
      "MHDA- 2016/12/16 06:00\n",
      "CRDT- 2016/12/16 06:00\n",
      "PHST- 2016/08/08 [received]\n",
      "PHST- 2016/10/19 [accepted]\n",
      "AID - 10.1111/cpr.12318 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Cell Prolif. 2016 Dec 14. doi: 10.1111/cpr.12318.\n",
      "\n",
      "PMID- 27975168\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161215\n",
      "LR  - 20161216\n",
      "IS  - 1559-1182 (Electronic)\n",
      "IS  - 0893-7648 (Linking)\n",
      "DP  - 2016 Dec 14\n",
      "TI  - Molecular Alterations in the Cerebellum of Sporadic Creutzfeldt-Jakob Disease\n",
      "      Subtypes with DJ-1 as a Key Regulator of Oxidative Stress.\n",
      "AB  - Cerebellar damage and granular and Purkinje cell loss in sporadic\n",
      "      Creutzfeldt-Jakob disease (sCJD) highlight a critical involvement of the\n",
      "      cerebellum during symptomatic progression of the disease. In this project, global\n",
      "      proteomic alterations in the cerebellum of brain from the two most prevalent\n",
      "      subtypes (MM1 and VV2) of sCJD were studied. Two-dimensional gel electrophoresis \n",
      "      (2DE) coupled mass spectrometric identification revealed 40 proteins in MM1 and\n",
      "      43 proteins in VV2 subtype to be differentially expressed. Of those, 12 proteins \n",
      "      showed common differential expression in their expression between two subtypes.\n",
      "      Differentially expressed proteins mainly belonged to (i) cell cycle, gene\n",
      "      expression and cell death; (ii) cellular stress response/oxidative stress (OS)\n",
      "      and (iii) signal transduction and synaptic functions, related molecular\n",
      "      functions. We verified 10 differentially expressed proteins at transcriptional\n",
      "      and translational level as well. Interestingly, protein deglycase DJ-1 (an\n",
      "      antioxidative protein) showed an increase in its messenger RNA (mRNA) expression \n",
      "      in both MM1 and VV2 subtypes but protein expression only in VV2 subtype in\n",
      "      cerebellum of sCJD patients. Nuclear translocalization of DJ-1 confirmed its\n",
      "      expressional alteration due to OS in sCJD. Downstream experiments showed the\n",
      "      activation of nuclear factor erythroid-2 related factor 2 (Nrf2)/antioxidative\n",
      "      response element (ARE) pathway. DJ-1 protein concentration was significantly\n",
      "      increased during the clinical phase in cerebrospinal fluid of sCJD patients and\n",
      "      also at presymptomatic and symptomatic stages in cerebellum of humanized PrP\n",
      "      transgenic mice inoculated with sCJD (MM1 and VV2) brain. These results suggest\n",
      "      the implication of oxidative stress during the pathophysiology of sCJD.\n",
      "FAU - Tahir, Waqas\n",
      "AU  - Tahir W\n",
      "AD  - Department of Neurology, University Medical Center Goettingen (UMG) and German\n",
      "      Center for Neurodegenerative Diseases (DZNE) Goettingen, Robert-Koch-Str., 40,\n",
      "      37075, Goettingen, Germany.\n",
      "FAU - Zafar, Saima\n",
      "AU  - Zafar S\n",
      "AD  - Department of Neurology, University Medical Center Goettingen (UMG) and German\n",
      "      Center for Neurodegenerative Diseases (DZNE) Goettingen, Robert-Koch-Str., 40,\n",
      "      37075, Goettingen, Germany. sz_awaan@yahoo.com.\n",
      "FAU - Llorens, Franc\n",
      "AU  - Llorens F\n",
      "AD  - Department of Neurology, University Medical Center Goettingen (UMG) and German\n",
      "      Center for Neurodegenerative Diseases (DZNE) Goettingen, Robert-Koch-Str., 40,\n",
      "      37075, Goettingen, Germany.\n",
      "FAU - Arora, Amandeep Singh\n",
      "AU  - Arora AS\n",
      "AD  - Department of Neurology, University Medical Center Goettingen (UMG) and German\n",
      "      Center for Neurodegenerative Diseases (DZNE) Goettingen, Robert-Koch-Str., 40,\n",
      "      37075, Goettingen, Germany.\n",
      "FAU - Thune, Katrin\n",
      "AU  - Thune K\n",
      "AD  - Department of Neurology, University Medical Center Goettingen (UMG) and German\n",
      "      Center for Neurodegenerative Diseases (DZNE) Goettingen, Robert-Koch-Str., 40,\n",
      "      37075, Goettingen, Germany.\n",
      "FAU - Schmitz, Matthias\n",
      "AU  - Schmitz M\n",
      "AD  - Department of Neurology, University Medical Center Goettingen (UMG) and German\n",
      "      Center for Neurodegenerative Diseases (DZNE) Goettingen, Robert-Koch-Str., 40,\n",
      "      37075, Goettingen, Germany.\n",
      "FAU - Gotzmann, Nadine\n",
      "AU  - Gotzmann N\n",
      "AD  - Department of Neurology, University Medical Center Goettingen (UMG) and German\n",
      "      Center for Neurodegenerative Diseases (DZNE) Goettingen, Robert-Koch-Str., 40,\n",
      "      37075, Goettingen, Germany.\n",
      "FAU - Kruse, Niels\n",
      "AU  - Kruse N\n",
      "AD  - Institute of Neuropathology, University Medical Center Goettingen (UMG),\n",
      "      Robert-Koch-Str. 40, 37075, Goettingen, Germany.\n",
      "FAU - Mollenhauer, Brit\n",
      "AU  - Mollenhauer B\n",
      "AD  - Institute of Neuropathology, University Medical Center Goettingen (UMG),\n",
      "      Robert-Koch-Str. 40, 37075, Goettingen, Germany.\n",
      "FAU - Torres, Juan Maria\n",
      "AU  - Torres JM\n",
      "AD  - Instituto Nacional de Investigacion y Tecnologia Agraria y Alimentaria, Carretera\n",
      "      de Algete a El Casar Km. 8,1 S/N, 28130, Valdeolmos, Madrid, Spain.\n",
      "FAU - Andreoletti, Olivier\n",
      "AU  - Andreoletti O\n",
      "AD  - Institut National de la Recherche Agronomique/Ecole Nationale Veterinaire,\n",
      "      Toulouse, France.\n",
      "FAU - Ferrer, Isidre\n",
      "AU  - Ferrer I\n",
      "AD  - Institute of Neuropathology, Hospitalet de Llobregat, IDIBELL-University Hospital\n",
      "      Bellvitge, University of Barcelona, Barcelona, Spain.\n",
      "AD  - Network Center for Biomedical Research of Neurodegenerative Diseases (CIBERNED), \n",
      "      Ministry of Health, Institute Carlos III, Madrid, Spain.\n",
      "FAU - Zerr, Inga\n",
      "AU  - Zerr I\n",
      "AD  - Department of Neurology, University Medical Center Goettingen (UMG) and German\n",
      "      Center for Neurodegenerative Diseases (DZNE) Goettingen, Robert-Koch-Str., 40,\n",
      "      37075, Goettingen, Germany.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161214\n",
      "PL  - United States\n",
      "TA  - Mol Neurobiol\n",
      "JT  - Molecular neurobiology\n",
      "JID - 8900963\n",
      "OTO - NOTNLM\n",
      "OT  - Antioxidative response elements\n",
      "OT  - Mass spectrometry\n",
      "OT  - Parkinson disease protein 7/protein deglycase DJ-1\n",
      "OT  - Sporadic Creutzfeldt-Jakob disease\n",
      "OT  - Two-dimensional gel electrophoresis\n",
      "EDAT- 2016/12/16 06:00\n",
      "MHDA- 2016/12/16 06:00\n",
      "CRDT- 2016/12/16 06:00\n",
      "PHST- 2016/09/02 [received]\n",
      "PHST- 2016/11/08 [accepted]\n",
      "AID - 10.1007/s12035-016-0294-4 [doi]\n",
      "AID - 10.1007/s12035-016-0294-4 [pii]\n",
      "PST - aheadofprint\n",
      "SO  - Mol Neurobiol. 2016 Dec 14.\n",
      "\n",
      "PMID- 27966608\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "DA  - 20161214\n",
      "LR  - 20161215\n",
      "IS  - 2045-2322 (Electronic)\n",
      "IS  - 2045-2322 (Linking)\n",
      "VI  - 6\n",
      "DP  - 2016 Dec 14\n",
      "TI  - Slit-Robo GTPase-Activating Protein 2 as a metastasis suppressor in osteosarcoma.\n",
      "PG  - 39059\n",
      "LID - 10.1038/srep39059 [doi]\n",
      "AB  - Osteosarcoma is the most common primary bone tumor, with metastatic disease\n",
      "      responsible for most treatment failure and patient death. A forward genetic\n",
      "      screen utilizing Sleeping Beauty mutagenesis in mice previously identified\n",
      "      potential genetic drivers of osteosarcoma metastasis, including Slit-Robo\n",
      "      GTPase-Activating Protein 2 (Srgap2). This study evaluates the potential role of \n",
      "      SRGAP2 in metastases-associated properties of osteosarcoma cell lines through\n",
      "      Srgap2 knockout via the CRISPR/Cas9 nuclease system and conditional\n",
      "      overexpression in the murine osteosarcoma cell lines K12 and K7M2. Proliferation,\n",
      "      migration, and anchorage independent growth were evaluated. RNA sequencing and\n",
      "      immunohistochemistry of human osteosarcoma tissue samples were used to further\n",
      "      evaluate the potential role of the Slit-Robo pathway in osteosarcoma. The effects\n",
      "      of Srgap2 expression modulation in the murine OS cell lines support the\n",
      "      hypothesis that SRGAP2 may have a role as a suppressor of metastases in\n",
      "      osteosarcoma. Additionally, SRGAP2 and other genes in the Slit-Robo pathway have \n",
      "      altered transcript levels in a subset of mouse and human osteosarcoma, and SRGAP2\n",
      "      protein expression is reduced or absent in a subset of primary tumor samples.\n",
      "      SRGAP2 and other axon guidance proteins likely play a role in osteosarcoma\n",
      "      metastasis, with loss of SRGAP2 potentially contributing to a more aggressive\n",
      "      phenotype.\n",
      "FAU - Marko, Tracy A\n",
      "AU  - Marko TA\n",
      "AD  - University of Minnesota, Masonic Cancer Center Minneapolis, MN, USA.\n",
      "FAU - Shamsan, Ghaidan A\n",
      "AU  - Shamsan GA\n",
      "AD  - Department of Biomedical Engineering University of Minnesota, Minneapolis, MN,\n",
      "      USA.\n",
      "FAU - Edwards, Elizabeth N\n",
      "AU  - Edwards EN\n",
      "AD  - University of Minnesota, Masonic Cancer Center Minneapolis, MN, USA.\n",
      "FAU - Hazelton, Paige E\n",
      "AU  - Hazelton PE\n",
      "AD  - University of Minnesota, Masonic Cancer Center Minneapolis, MN, USA.\n",
      "FAU - Rathe, Susan K\n",
      "AU  - Rathe SK\n",
      "AD  - University of Minnesota, Masonic Cancer Center Minneapolis, MN, USA.\n",
      "FAU - Cornax, Ingrid\n",
      "AU  - Cornax I\n",
      "AD  - University of Minnesota, Masonic Cancer Center Minneapolis, MN, USA.\n",
      "AD  - Comparative Pathology Shared Resource, University of Minnesota, Minneapolis, MN, \n",
      "      USA.\n",
      "FAU - Overn, Paula R\n",
      "AU  - Overn PR\n",
      "AD  - University of Minnesota, Masonic Cancer Center Minneapolis, MN, USA.\n",
      "AD  - Comparative Pathology Shared Resource, University of Minnesota, Minneapolis, MN, \n",
      "      USA.\n",
      "FAU - Varshney, Jyotika\n",
      "AU  - Varshney J\n",
      "AD  - University of Minnesota, Masonic Cancer Center Minneapolis, MN, USA.\n",
      "FAU - Diessner, Brandon J\n",
      "AU  - Diessner BJ\n",
      "AD  - Department of Pediatrics, University of Minnesota, Minneapolis, MN, USA.\n",
      "FAU - Moriarity, Branden S\n",
      "AU  - Moriarity BS\n",
      "AD  - University of Minnesota, Masonic Cancer Center Minneapolis, MN, USA.\n",
      "AD  - Department of Pediatrics, University of Minnesota, Minneapolis, MN, USA.\n",
      "AD  - Center for Genome Engineering, University of Minnesota, Minneapolis, MN, USA.\n",
      "FAU - O'Sullivan, M Gerard\n",
      "AU  - O'Sullivan MG\n",
      "AD  - University of Minnesota, Masonic Cancer Center Minneapolis, MN, USA.\n",
      "AD  - Comparative Pathology Shared Resource, University of Minnesota, Minneapolis, MN, \n",
      "      USA.\n",
      "AD  - College of Veterinary Medicine, Department of Veterinary Population Medicine,\n",
      "      University of Minnesota, Minneapolis, MN, USA.\n",
      "FAU - Odde, David J\n",
      "AU  - Odde DJ\n",
      "AD  - Department of Biomedical Engineering University of Minnesota, Minneapolis, MN,\n",
      "      USA.\n",
      "FAU - Largaespada, David A\n",
      "AU  - Largaespada DA\n",
      "AD  - University of Minnesota, Masonic Cancer Center Minneapolis, MN, USA.\n",
      "AD  - Department of Pediatrics, University of Minnesota, Minneapolis, MN, USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161214\n",
      "PL  - England\n",
      "TA  - Sci Rep\n",
      "JT  - Scientific reports\n",
      "JID - 101563288\n",
      "EDAT- 2016/12/15 06:00\n",
      "MHDA- 2016/12/15 06:00\n",
      "CRDT- 2016/12/15 06:00\n",
      "PHST- 2016/08/23 [received]\n",
      "PHST- 2016/11/16 [accepted]\n",
      "AID - srep39059 [pii]\n",
      "AID - 10.1038/srep39059 [doi]\n",
      "PST - epublish\n",
      "SO  - Sci Rep. 2016 Dec 14;6:39059. doi: 10.1038/srep39059.\n",
      "\n",
      "PMID- 27965467\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161214\n",
      "LR  - 20161215\n",
      "IS  - 1949-2553 (Electronic)\n",
      "IS  - 1949-2553 (Linking)\n",
      "DP  - 2016 Dec 11\n",
      "TI  - MicroRNA expressing profiles in A53T mutant alpha-synuclein transgenic mice and\n",
      "      Parkinsonian.\n",
      "LID - 10.18632/oncotarget.13905 [doi]\n",
      "AB  - alpha-synuclein gene mutations can cause alpha-synuclein protein aggregation in\n",
      "      the midbrain of Parkinson's disease (PD) patients. MicroRNAs (miRNAs) play a key \n",
      "      role in the metabolism of alpha-synuclein but the mechanism involved in\n",
      "      synucleinopathy remains unclear. In this study, we investigated the miRNA\n",
      "      profiles in A53T-alpha-synuclein transgenic mice and analyzed the candidate\n",
      "      miRNAs in the cerebrospinal fluid (CSF) of PD patients. The 12-month\n",
      "      A53T-transgenic mouse displayed hyperactive movement and anxiolytic-like\n",
      "      behaviors with alpha-synuclein aggregation in midbrain. A total of 317,759 total \n",
      "      and 289,207 unique small RNA sequences in the midbrain of mice were identified by\n",
      "      high-throughput deep sequencing. We found 644 miRNAs were significantly changed\n",
      "      in the transgenic mice. Based on the conserved characteristic of miRNAs, we\n",
      "      selected 11 candidates from the 40 remarkably expressed miRNAs and explored their\n",
      "      expression in 44 CSF samples collected from PD patients. The results revealed\n",
      "      that 11 microRNAs were differently expressed in CSF, emphatically as miR-144-5p, \n",
      "      miR-200a-3p and miR-542-3p, which were dramatically up-regulated in both\n",
      "      A53T-transgenic mice and PD patients, and had a helpful accuracy for the PD\n",
      "      prediction. The ordered logistic regression analysis showed that the severity of \n",
      "      PD has strong correlation with an up-expression of miR-144-5p, miR-200a-3p and\n",
      "      miR-542-3p in CSF. Taken together, our data suggested that miRNAs in CSF, such as\n",
      "      miR-144-5p, miR-200a-3p and miR-542-3p, may be useful to the PD diagnosis as\n",
      "      potential biomarkers.\n",
      "FAU - Mo, Mingshu\n",
      "AU  - Mo M\n",
      "AD  - Department of Neurology, The First Affiliated Hospital of Guangzhou Medical\n",
      "      University, Guangdong, China.\n",
      "FAU - Xiao, Yousheng\n",
      "AU  - Xiao Y\n",
      "AD  - Department of Neurology, The First Affiliated Hospital of Sun Yat-sen University,\n",
      "      Guangdong, China.\n",
      "FAU - Huang, Shuxuan\n",
      "AU  - Huang S\n",
      "AD  - Department of Neurology, The First Affiliated Hospital of Guangzhou Medical\n",
      "      University, Guangdong, China.\n",
      "FAU - Cen, Luan\n",
      "AU  - Cen L\n",
      "AD  - Department of Neurology, The First Affiliated Hospital of Sun Yat-sen University,\n",
      "      Guangdong, China.\n",
      "FAU - Chen, Xiang\n",
      "AU  - Chen X\n",
      "AD  - Department of Neurology, The First Affiliated Hospital of Sun Yat-sen University,\n",
      "      Guangdong, China.\n",
      "FAU - Zhang, Limin\n",
      "AU  - Zhang L\n",
      "AD  - Department of Neurology, The First Affiliated Hospital of Sun Yat-sen University,\n",
      "      Guangdong, China.\n",
      "FAU - Luo, Qin\n",
      "AU  - Luo Q\n",
      "AD  - Department of Neurology, The Third Affiliated Hospital of Xinjiang Medical\n",
      "      University, Urumqi, China.\n",
      "FAU - Li, Shaomin\n",
      "AU  - Li S\n",
      "AD  - Ann Romney Center for Neurologic Disease, Brigham and Women's Hospital, Harvard\n",
      "      Medical School, Boston, MA, USA.\n",
      "FAU - Yang, Xinling\n",
      "AU  - Yang X\n",
      "AD  - Department of Neurology, The Third Affiliated Hospital of Xinjiang Medical\n",
      "      University, Urumqi, China.\n",
      "FAU - Lin, Xian\n",
      "AU  - Lin X\n",
      "AD  - Department of Anatomy & Guangdong Province Key Laboratory of Brain Function and\n",
      "      Disease, Zhongshan School of Medicine, Sun Yat-sen University, Guangdong, China.\n",
      "FAU - Xu, Pingyi\n",
      "AU  - Xu P\n",
      "AD  - Department of Neurology, The First Affiliated Hospital of Guangzhou Medical\n",
      "      University, Guangdong, China.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161211\n",
      "PL  - United States\n",
      "TA  - Oncotarget\n",
      "JT  - Oncotarget\n",
      "JID - 101532965\n",
      "OTO - NOTNLM\n",
      "OT  - A53T mutation\n",
      "OT  - Gerotarget\n",
      "OT  - Parkinson's disease\n",
      "OT  - deep sequencing\n",
      "OT  - microRNAs\n",
      "EDAT- 2016/12/15 06:00\n",
      "MHDA- 2016/12/15 06:00\n",
      "CRDT- 2016/12/15 06:00\n",
      "PHST- 2016/05/25 [received]\n",
      "PHST- 2016/12/01 [accepted]\n",
      "AID - 13905 [pii]\n",
      "AID - 10.18632/oncotarget.13905 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Oncotarget. 2016 Dec 11. doi: 10.18632/oncotarget.13905.\n",
      "\n",
      "PMID- 27965448\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "DA  - 20161214\n",
      "LR  - 20161215\n",
      "IS  - 2150-7511 (Electronic)\n",
      "VI  - 7\n",
      "IP  - 6\n",
      "DP  - 2016 Dec 13\n",
      "TI  - The Conserved Coronavirus Macrodomain Promotes Virulence and Suppresses the\n",
      "      Innate Immune Response during Severe Acute Respiratory Syndrome Coronavirus\n",
      "      Infection.\n",
      "LID - e01721-16 [pii]\n",
      "LID - 10.1128/mBio.01721-16 [doi]\n",
      "AB  - ADP-ribosylation is a common posttranslational modification that may have\n",
      "      antiviral properties and impact innate immunity. To regulate this activity,\n",
      "      macrodomain proteins enzymatically remove covalently attached ADP-ribose from\n",
      "      protein targets. All members of the Coronavirinae, a subfamily of positive-sense \n",
      "      RNA viruses, contain a highly conserved macrodomain within nonstructural protein \n",
      "      3 (nsp3). However, its function or targets during infection remain unknown. We\n",
      "      identified several macrodomain mutations that greatly reduced nsp3's\n",
      "      de-ADP-ribosylation activity in vitro Next, we created recombinant severe acute\n",
      "      respiratory syndrome coronavirus (SARS-CoV) strains with these mutations. These\n",
      "      mutations led to virus attenuation and a modest reduction of viral loads in\n",
      "      infected mice, despite normal replication in cell culture. Further, macrodomain\n",
      "      mutant virus elicited an early, enhanced interferon (IFN), interferon-stimulated \n",
      "      gene (ISG), and proinflammatory cytokine response in mice and in a human\n",
      "      bronchial epithelial cell line. Using a coinfection assay, we found that\n",
      "      inclusion of mutant virus in the inoculum protected mice from an otherwise lethal\n",
      "      SARS-CoV infection without reducing virus loads, indicating that the changes in\n",
      "      innate immune response were physiologically significant. In conclusion, we have\n",
      "      established a novel function for the SARS-CoV macrodomain that implicates\n",
      "      ADP-ribose in the regulation of the innate immune response and helps to\n",
      "      demonstrate why this domain is conserved in CoVs. IMPORTANCE: The macrodomain is \n",
      "      a ubiquitous structural domain that removes ADP-ribose from proteins, reversing\n",
      "      the activity of ADP-ribosyltransferases. All coronaviruses contain a macrodomain,\n",
      "      suggesting that ADP-ribosylation impacts coronavirus infection. However, its\n",
      "      function during infection remains unknown. Here, we found that the macrodomain is\n",
      "      an important virulence factor for a highly pathogenic human CoV, SARS-CoV.\n",
      "      Viruses with macrodomain mutations that abrogate its ability to remove ADP-ribose\n",
      "      from protein were unable to cause lethal disease in mice. Importantly, the\n",
      "      SARS-CoV macrodomain suppressed the innate immune response during infection. Our \n",
      "      data suggest that an early innate immune response can protect mice from lethal\n",
      "      disease. Understanding the mechanism used by this enzyme to promote disease will \n",
      "      open up novel avenues for coronavirus therapies and give further insight into the\n",
      "      role of macrodomains in viral pathogenesis.\n",
      "CI  - Copyright (c) 2016 Fehr et al.\n",
      "FAU - Fehr, Anthony R\n",
      "AU  - Fehr AR\n",
      "AD  - Department of Microbiology, University of Iowa, Iowa City, Iowa, USA.\n",
      "FAU - Channappanavar, Rudragouda\n",
      "AU  - Channappanavar R\n",
      "AD  - Department of Microbiology, University of Iowa, Iowa City, Iowa, USA.\n",
      "FAU - Jankevicius, Gytis\n",
      "AU  - Jankevicius G\n",
      "AD  - Sir William Dunn School of Pathology, University of Oxford, Oxford, United\n",
      "      Kingdom.\n",
      "FAU - Fett, Craig\n",
      "AU  - Fett C\n",
      "AD  - Department of Microbiology, University of Iowa, Iowa City, Iowa, USA.\n",
      "FAU - Zhao, Jincun\n",
      "AU  - Zhao J\n",
      "AD  - Department of Microbiology, University of Iowa, Iowa City, Iowa, USA.\n",
      "AD  - State Key Laboratory of Respiratory Diseases, Guangzhou Institute of Respiratory \n",
      "      Disease, The First Affiliated Hospital of Guangzhou Medical University, Yuexiu\n",
      "      District, Guangzhou, Guangdong, China.\n",
      "FAU - Athmer, Jeremiah\n",
      "AU  - Athmer J\n",
      "AD  - Department of Microbiology, University of Iowa, Iowa City, Iowa, USA.\n",
      "FAU - Meyerholz, David K\n",
      "AU  - Meyerholz DK\n",
      "AD  - Department of Pathology, University of Iowa, Iowa City, Iowa, USA.\n",
      "FAU - Ahel, Ivan\n",
      "AU  - Ahel I\n",
      "AD  - Sir William Dunn School of Pathology, University of Oxford, Oxford, United\n",
      "      Kingdom.\n",
      "FAU - Perlman, Stanley\n",
      "AU  - Perlman S\n",
      "AD  - Department of Microbiology, University of Iowa, Iowa City, Iowa, USA\n",
      "      Stanley-Perlman@uiowa.edu.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161213\n",
      "PL  - United States\n",
      "TA  - MBio\n",
      "JT  - mBio\n",
      "JID - 101519231\n",
      "EDAT- 2016/12/15 06:00\n",
      "MHDA- 2016/12/15 06:00\n",
      "CRDT- 2016/12/15 06:00\n",
      "AID - mBio.01721-16 [pii]\n",
      "AID - 10.1128/mBio.01721-16 [doi]\n",
      "PST - epublish\n",
      "SO  - MBio. 2016 Dec 13;7(6). pii: e01721-16. doi: 10.1128/mBio.01721-16.\n",
      "\n",
      "PMID- 27955979\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161213\n",
      "LR  - 20161224\n",
      "IS  - 1095-8584 (Electronic)\n",
      "IS  - 0022-2828 (Linking)\n",
      "VI  - 102\n",
      "DP  - 2016 Dec 10\n",
      "TI  - Cardiac inflammation in genetic dilated cardiomyopathy caused by MYBPC3 mutation.\n",
      "PG  - 83-93\n",
      "LID - S0022-2828(16)30444-8 [pii]\n",
      "LID - 10.1016/j.yjmcc.2016.12.002 [doi]\n",
      "AB  - Cardiomyopathies are a leading cause of heart failure and are often caused by\n",
      "      mutations in sarcomeric genes, resulting in contractile dysfunction and cellular \n",
      "      damage. This may stimulate the production of a robust proinflammatory response.\n",
      "      To determine whether myocardial inflammation is associated with cardiac\n",
      "      dysfunction in dilated cardiomyopathy (DCM) caused by MYBPC3 mutation, we used\n",
      "      the well-characterized cMyBP-C(t/t) mouse model of DCM at 3months of age.\n",
      "      Compared to wild type (WT) mice, DCM mice exhibited significantly decreased\n",
      "      fractional shortening (36.4+/-2% vs. 15.5+/-1.0%, p<0.0001) and significantly\n",
      "      increased spleen weight (5.3+/-0.3 vs. 7.2+/-0.4mg/mm, p=0.002). Intriguingly,\n",
      "      flow cytometry analysis revealed a significant increase in total\n",
      "      (CD45+CD11b+Ly6C-MHCII+F480+) macrophages (6.5+/-1.4% vs. 14.8+/-1.4%, p=0.002)\n",
      "      and classically activated (CD45+CD11b+Ly6C-MHCII+F480+CD206-) proinflammatory\n",
      "      (M1) macrophages (3.4+/-0.8% vs. 10.3+/-1.2%, p=0.0009) in DCM hearts as compared\n",
      "      with WT hearts. These results were further confirmed by immunofluorescence\n",
      "      analysis of heart tissue sections. Splenic red pulp (CD11b+Ly6C+MHCIIlowF480hi)\n",
      "      macrophages were significantly elevated (1.3+/-0.1% vs. 2.4+/-0.1%, p=0.0001) in \n",
      "      DCM compared to WT animals. Serum cytokine analysis in DCM animals exhibited a\n",
      "      significant increase (0.65+/-0.2 vs. 2.175+/-0.5pg/mL, p=0.02) in interleukin\n",
      "      (IL)-6 compared to WT animals. Furthermore, RNA-seq analysis revealed the\n",
      "      upregulation of inflammatory pathways in the DCM hearts. Together, these data\n",
      "      indicate a robust proinflammatory response in DCM hearts, likely in response to\n",
      "      cellular damage triggered by MYBPC3 mutation and resultant contractile\n",
      "      dysfunction.\n",
      "CI  - Copyright (c) 2016 The Authors. Published by Elsevier Ltd.. All rights reserved.\n",
      "FAU - Lynch, Thomas L 4th\n",
      "AU  - Lynch TL 4th\n",
      "AD  - Department of Cell and Molecular Physiology, Health Sciences Division, Loyola\n",
      "      University Chicago, Maywood, IL 60153, USA; Department of Cell and Regenerative\n",
      "      Biology, University of Wisconsin-Madison, Madison, WI 53715, USA.\n",
      "FAU - Ismahil, Mohamed Ameen\n",
      "AU  - Ismahil MA\n",
      "AD  - Division of Cardiovascular Disease, University of Alabama at Birmingham,\n",
      "      Birmingham, AL 35294, USA.\n",
      "FAU - Jegga, Anil G\n",
      "AU  - Jegga AG\n",
      "AD  - Division of Biomedical Informatics, Cincinnati Children's Hospital Medical\n",
      "      Center, Cincinnati, OH 45229, USA.\n",
      "FAU - Zilliox, Michael J\n",
      "AU  - Zilliox MJ\n",
      "AD  - Department of Public Health Sciences, Health Sciences Division, Loyola University\n",
      "      Chicago, Maywood, IL 60153, USA.\n",
      "FAU - Troidl, Christian\n",
      "AU  - Troidl C\n",
      "AD  - Kerckhoff Heart and Thorax Center, Department of Cardiology, Bad Nauheim,\n",
      "      Germany.\n",
      "FAU - Prabhu, Sumanth D\n",
      "AU  - Prabhu SD\n",
      "AD  - Division of Cardiovascular Disease, University of Alabama at Birmingham,\n",
      "      Birmingham, AL 35294, USA.\n",
      "FAU - Sadayappan, Sakthivel\n",
      "AU  - Sadayappan S\n",
      "AD  - Department of Cell and Molecular Physiology, Health Sciences Division, Loyola\n",
      "      University Chicago, Maywood, IL 60153, USA. Electronic address:\n",
      "      sadayasl@ucmail.uc.edu.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161210\n",
      "PL  - England\n",
      "TA  - J Mol Cell Cardiol\n",
      "JT  - Journal of molecular and cellular cardiology\n",
      "JID - 0262322\n",
      "OTO - NOTNLM\n",
      "OT  - Dilated cardiomyopathy\n",
      "OT  - Inflammation\n",
      "OT  - MYBPC3\n",
      "OT  - Mouse models\n",
      "OT  - Sarcomere biology\n",
      "EDAT- 2016/12/14 06:00\n",
      "MHDA- 2016/12/14 06:00\n",
      "CRDT- 2016/12/14 06:00\n",
      "PHST- 2016/10/01 [received]\n",
      "PHST- 2016/12/06 [revised]\n",
      "PHST- 2016/12/08 [accepted]\n",
      "AID - S0022-2828(16)30444-8 [pii]\n",
      "AID - 10.1016/j.yjmcc.2016.12.002 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - J Mol Cell Cardiol. 2016 Dec 10;102:83-93. doi: 10.1016/j.yjmcc.2016.12.002.\n",
      "\n",
      "PMID- 27955842\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161213\n",
      "LR  - 20161214\n",
      "IS  - 1873-569X (Electronic)\n",
      "IS  - 0923-1811 (Linking)\n",
      "DP  - 2016 Dec 05\n",
      "TI  - Sox10 regulates skin melanocyte proliferation by activating the DNA replication\n",
      "      licensing factor MCM5.\n",
      "LID - S0923-1811(16)31014-3 [pii]\n",
      "LID - 10.1016/j.jdermsci.2016.12.002 [doi]\n",
      "AB  - BACKGROUND: The control of cell proliferation is a fundamental aspect of tissue\n",
      "      formation in development and regeneration. A cell type that illustrates this\n",
      "      point particularly well is the neural crest-derived melanocyte, the pigment cell \n",
      "      of vertebrates, as melanocytes can be followed easily during development and\n",
      "      their pigment is directly visible in the integument of the adult. In mammals,\n",
      "      melanocytes undergo physiological cycles of loss and proliferative regeneration\n",
      "      during the hair cycle, and their proliferation is also critical during wound\n",
      "      healing, repigmentation of depigmented lesions, and in melanoma formation and\n",
      "      progression. Hence, a thorough analysis of the molecular parameters controlling\n",
      "      melanocyte proliferation is crucial for our understanding of the physiology of\n",
      "      this cell type both in health and disease. OBJECTIVE: SOX10 is a critical\n",
      "      regulator in melanocytes and melanoma cells, but its specific role in their\n",
      "      proliferation is far from clear. In this study we analyze the role of SOX10 in\n",
      "      regulating mammalian melanocyte proliferation in a mouse model. METHODS: The role\n",
      "      of SOX10 in melanoblast proliferation was analyzed in Sox10/+ mice by co-staining\n",
      "      for melanocyte-specific markers and cell proliferation. In vitro, the role of\n",
      "      SOX10 was studied by manipulating its levels using RNAi and analyzing the effects\n",
      "      on DNA synthesis and cell growth and on gene expression at the RNA and protein\n",
      "      levels. RESULTS: Reduction of Sox10 gene dose led to a reduction in the number of\n",
      "      melanoblasts. Knockdown of Sox10 in melanocytes led to inhibition of cell\n",
      "      proliferation and a decrease in the expression of the minichromosome maintenance \n",
      "      complex component 5 (MCM5). In fact, SOX10 directly activated MCM5 transcription \n",
      "      by binding to conserved SOX10 consensus DNA sequences in the MCM5 promoter.\n",
      "      Furthermore, the defect in cell proliferation could be rescued partially by\n",
      "      overexpression of MCM5 in Sox10 knockdown melanocytes. CONCLUSION: The results\n",
      "      suggest that the SOX10-MCM5 axis plays an important role in controlling\n",
      "      melanocyte proliferation. Our findings provide novel insights into the regulatory\n",
      "      mechanisms of melanocyte proliferation and may have implications for our\n",
      "      understanding of the roles of SOX10 and MCM5 in abnormal melanocyte proliferation\n",
      "      disorders such as cutaneous melanoma.\n",
      "CI  - Copyright (c) 2016 Japanese Society for Investigative Dermatology. Published by\n",
      "      Elsevier Ireland Ltd. All rights reserved.\n",
      "FAU - Su, Zhongyuan\n",
      "AU  - Su Z\n",
      "AD  - Labratory of Developmental Cell Biology and Disease, School of Ophthalmology and \n",
      "      Optometry and Eye Hospital, Wenzhou Medical University, Wenzhou, 325003, China;\n",
      "      State Key Laboratory Cultivation Base and Key Laboratory of Vision Science of\n",
      "      Ministry of Health and Zhejiang Provincial Key Laboratory of Ophthalmology,\n",
      "      Wenzhou, 325003, China.\n",
      "FAU - Zheng, Xiaozi\n",
      "AU  - Zheng X\n",
      "AD  - Labratory of Developmental Cell Biology and Disease, School of Ophthalmology and \n",
      "      Optometry and Eye Hospital, Wenzhou Medical University, Wenzhou, 325003, China.\n",
      "FAU - Zhang, Xiaobo\n",
      "AU  - Zhang X\n",
      "AD  - Labratory of Developmental Cell Biology and Disease, School of Ophthalmology and \n",
      "      Optometry and Eye Hospital, Wenzhou Medical University, Wenzhou, 325003, China.\n",
      "FAU - Wang, Yipin\n",
      "AU  - Wang Y\n",
      "AD  - Labratory of Developmental Cell Biology and Disease, School of Ophthalmology and \n",
      "      Optometry and Eye Hospital, Wenzhou Medical University, Wenzhou, 325003, China.\n",
      "FAU - Zhu, Shanpu\n",
      "AU  - Zhu S\n",
      "AD  - Labratory of Developmental Cell Biology and Disease, School of Ophthalmology and \n",
      "      Optometry and Eye Hospital, Wenzhou Medical University, Wenzhou, 325003, China.\n",
      "FAU - Lu, Fan\n",
      "AU  - Lu F\n",
      "AD  - State Key Laboratory Cultivation Base and Key Laboratory of Vision Science of\n",
      "      Ministry of Health and Zhejiang Provincial Key Laboratory of Ophthalmology,\n",
      "      Wenzhou, 325003, China.\n",
      "FAU - Qu, Jia\n",
      "AU  - Qu J\n",
      "AD  - State Key Laboratory Cultivation Base and Key Laboratory of Vision Science of\n",
      "      Ministry of Health and Zhejiang Provincial Key Laboratory of Ophthalmology,\n",
      "      Wenzhou, 325003, China.\n",
      "FAU - Hou, Ling\n",
      "AU  - Hou L\n",
      "AD  - Labratory of Developmental Cell Biology and Disease, School of Ophthalmology and \n",
      "      Optometry and Eye Hospital, Wenzhou Medical University, Wenzhou, 325003, China;\n",
      "      State Key Laboratory Cultivation Base and Key Laboratory of Vision Science of\n",
      "      Ministry of Health and Zhejiang Provincial Key Laboratory of Ophthalmology,\n",
      "      Wenzhou, 325003, China. Electronic address: lhou@mail.eye.ac.cn.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161205\n",
      "PL  - Netherlands\n",
      "TA  - J Dermatol Sci\n",
      "JT  - Journal of dermatological science\n",
      "JID - 9011485\n",
      "OTO - NOTNLM\n",
      "OT  - Cell proliferation\n",
      "OT  - Gene regulation\n",
      "OT  - Neural crest\n",
      "OT  - Pigment cell\n",
      "OT  - Skin melanocytes\n",
      "OT  - Transcription factor\n",
      "EDAT- 2016/12/14 06:00\n",
      "MHDA- 2016/12/14 06:00\n",
      "CRDT- 2016/12/14 06:00\n",
      "PHST- 2016/07/22 [received]\n",
      "PHST- 2016/10/21 [revised]\n",
      "PHST- 2016/12/02 [accepted]\n",
      "AID - S0923-1811(16)31014-3 [pii]\n",
      "AID - 10.1016/j.jdermsci.2016.12.002 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - J Dermatol Sci. 2016 Dec 5. pii: S0923-1811(16)31014-3. doi:\n",
      "      10.1016/j.jdermsci.2016.12.002.\n",
      "\n",
      "PMID- 27955791\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "DA  - 20161213\n",
      "LR  - 20161214\n",
      "IS  - 2214-109X (Electronic)\n",
      "IS  - 2214-109X (Linking)\n",
      "VI  - 5\n",
      "IP  - 1\n",
      "DP  - 2017 Jan\n",
      "TI  - Persistence and clearance of Ebola virus RNA from seminal fluid of Ebola virus\n",
      "      disease survivors: a longitudinal analysis and modelling study.\n",
      "PG  - e80-e88\n",
      "LID - S2214-109X(16)30243-1 [pii]\n",
      "LID - 10.1016/S2214-109X(16)30243-1 [doi]\n",
      "AB  - BACKGROUND: By January, 2016, all known transmission chains of the Ebola virus\n",
      "      disease (EVD) outbreak in west Africa had been stopped. However, there is concern\n",
      "      about persistence of Ebola virus in the reproductive tract of men who have\n",
      "      survived EVD. We aimed to use biostatistical modelling to describe the dynamics\n",
      "      of Ebola virus RNA load in seminal fluid, including clearance parameters.\n",
      "      METHODS: In this longitudinal study, we recruited men who had been discharged\n",
      "      from three Ebola treatment units in Guinea between January and July, 2015.\n",
      "      Participants provided samples of seminal fluid at follow-up every 3-6 weeks,\n",
      "      which we tested for Ebola virus RNA using quantitative real-time RT-PCR.\n",
      "      Representative specimens from eight participants were then inoculated into\n",
      "      immunodeficient mice to test for infectivity. We used a linear mixed-effect model\n",
      "      to analyse the dynamics of virus persistence in seminal fluid over time.\n",
      "      FINDINGS: We enrolled 26 participants and tested 130 seminal fluid specimens;\n",
      "      median follow up was 197 days (IQR 187-209 days) after enrolment, which\n",
      "      corresponded to 255 days (228-287) after disease onset. Ebola virus RNA was\n",
      "      detected in 86 semen specimens from 19 (73%) participants. Median duration of\n",
      "      Ebola virus RNA detection was 158 days after onset (73-181; maximum 407 days at\n",
      "      end of follow-up). Mathematical modelling of the quantitative time-series data\n",
      "      showed a mean clearance rate of Ebola virus RNA from seminal fluid of -0.58 log\n",
      "      units per month, although the clearance kinetic varied greatly between\n",
      "      participants. Using our biostatistical model, we predict that 50% and 90% of male\n",
      "      survivors clear Ebola virus RNA from seminal fluid at 115 days (90% prediction\n",
      "      interval 72-160) and 294 days (212-399) after disease onset, respectively. We\n",
      "      also predicted that the number of men positive for Ebola virus RNA in affected\n",
      "      countries would decrease from about 50 in January 2016, to fewer than 1 person by\n",
      "      July, 2016. Infectious virus was detected in 15 of 26 (58%) specimens tested in\n",
      "      mice. INTERPRETATION: Time to clearance of Ebola virus RNA from seminal fluid\n",
      "      varies greatly between individuals and could be more than 13 months. Our\n",
      "      predictions will assist in decision-making about surveillance and preventive\n",
      "      measures in EVD outbreaks. FUNDING: This study was funded by European Union's\n",
      "      Horizon 2020 research and innovation programme, Directorate-General for\n",
      "      International Cooperation and Development of the European Commission, Institut\n",
      "      national de la sante et de la recherche medicale (INSERM), German Research\n",
      "      Foundation (DFG), and Innovative Medicines Initiative 2 Joint Undertaking.\n",
      "CI  - Copyright (c) 2017 The Author(s). Published by Elsevier Ltd. This is an Open\n",
      "      Access article under the CC BY license. Published by Elsevier Ltd.. All rights\n",
      "      reserved.\n",
      "FAU - Sissoko, Daouda\n",
      "AU  - Sissoko D\n",
      "AD  - INSERM U1219, Bordeaux University, Bordeaux, France; Bordeaux University\n",
      "      Hospital, Bordeaux, France.\n",
      "FAU - Duraffour, Sophie\n",
      "AU  - Duraffour S\n",
      "AD  - The European Mobile Laboratory Consortium, Hamburg, Germany;\n",
      "      Bernhard-Nocht-Institute for Tropical Medicine, Hamburg, Germany.\n",
      "FAU - Kerber, Romy\n",
      "AU  - Kerber R\n",
      "AD  - Bernhard-Nocht-Institute for Tropical Medicine, Hamburg, Germany; German Centre\n",
      "      for Infection Research (DZIF), Partner Site Hamburg, Conakry, Guinea.\n",
      "FAU - Kolie, Jacques Seraphin\n",
      "AU  - Kolie JS\n",
      "AD  - Centre de Formation et de Recherche en Sante Rurale de Maferinyah, Conakry,\n",
      "      Guinea.\n",
      "FAU - Beavogui, Abdoul Habib\n",
      "AU  - Beavogui AH\n",
      "AD  - Centre de Formation et de Recherche en Sante Rurale de Maferinyah, Conakry,\n",
      "      Guinea.\n",
      "FAU - Camara, Alseny-Modet\n",
      "AU  - Camara AM\n",
      "AD  - Department of Infectious Diseases, Donka Hospital, Conakry, Guinea.\n",
      "FAU - Colin, Geraldine\n",
      "AU  - Colin G\n",
      "AD  - INSERM U1219, Bordeaux University, Bordeaux, France; PAC-CI, ANRS Research Site, \n",
      "      Treichville University Hospital, Abidjan, Cote d'Ivoire.\n",
      "FAU - Rieger, Toni\n",
      "AU  - Rieger T\n",
      "AD  - Bernhard-Nocht-Institute for Tropical Medicine, Hamburg, Germany; German Centre\n",
      "      for Infection Research (DZIF), Partner Site Hamburg, Conakry, Guinea.\n",
      "FAU - Oestereich, Lisa\n",
      "AU  - Oestereich L\n",
      "AD  - Bernhard-Nocht-Institute for Tropical Medicine, Hamburg, Germany; German Centre\n",
      "      for Infection Research (DZIF), Partner Site Hamburg, Conakry, Guinea.\n",
      "FAU - Palyi, Bernadett\n",
      "AU  - Palyi B\n",
      "AD  - National Centre for Epidemiology, Hungarian National Biosafety Laboratory,\n",
      "      Budapest, Hungary.\n",
      "FAU - Wurr, Stephanie\n",
      "AU  - Wurr S\n",
      "AD  - Bernhard-Nocht-Institute for Tropical Medicine, Hamburg, Germany; German Centre\n",
      "      for Infection Research (DZIF), Partner Site Hamburg, Conakry, Guinea.\n",
      "FAU - Guedj, Jeremie\n",
      "AU  - Guedj J\n",
      "AD  - INSERM, IAME, UMR 1137 Paris, and Universite Paris Diderot, Sorbonne Paris Cite, \n",
      "      France.\n",
      "FAU - Nguyen, Thi Huyen Tram\n",
      "AU  - Nguyen TH\n",
      "AD  - INSERM, IAME, UMR 1137 Paris, and Universite Paris Diderot, Sorbonne Paris Cite, \n",
      "      France.\n",
      "FAU - Eggo, Rosalind M\n",
      "AU  - Eggo RM\n",
      "AD  - Department of Infectious Disease Epidemiology, London School of Hygiene &\n",
      "      Tropical Medicine, London, UK.\n",
      "FAU - Watson, Conall H\n",
      "AU  - Watson CH\n",
      "AD  - Department of Infectious Disease Epidemiology, London School of Hygiene &\n",
      "      Tropical Medicine, London, UK.\n",
      "FAU - Edmunds, W John\n",
      "AU  - Edmunds WJ\n",
      "AD  - Department of Infectious Disease Epidemiology, London School of Hygiene &\n",
      "      Tropical Medicine, London, UK.\n",
      "FAU - Bore, Joseph Akoi\n",
      "AU  - Bore JA\n",
      "AD  - The European Mobile Laboratory Consortium, Hamburg, Germany; Ministry of Health\n",
      "      Guinea, Conakry, Guinea.\n",
      "FAU - Koundouno, Fara Raymond\n",
      "AU  - Koundouno FR\n",
      "AD  - The European Mobile Laboratory Consortium, Hamburg, Germany; Ministry of Health\n",
      "      Guinea, Conakry, Guinea.\n",
      "FAU - Cabeza-Cabrerizo, Mar\n",
      "AU  - Cabeza-Cabrerizo M\n",
      "AD  - The European Mobile Laboratory Consortium, Hamburg, Germany;\n",
      "      Bernhard-Nocht-Institute for Tropical Medicine, Hamburg, Germany.\n",
      "FAU - Carter, Lisa L\n",
      "AU  - Carter LL\n",
      "AD  - The European Mobile Laboratory Consortium, Hamburg, Germany; University College\n",
      "      London, London, UK.\n",
      "FAU - Kafetzopoulou, Liana Eleni\n",
      "AU  - Kafetzopoulou LE\n",
      "AD  - The European Mobile Laboratory Consortium, Hamburg, Germany; Laboratory for\n",
      "      Clinical and Epidemiological Virology, Department of Microbiology and Immunology,\n",
      "      KU Leuven, Leuven, Belgium.\n",
      "FAU - Kuisma, Eeva\n",
      "AU  - Kuisma E\n",
      "AD  - The European Mobile Laboratory Consortium, Hamburg, Germany; Public Health\n",
      "      England, Porton Down, Salisbury, UK.\n",
      "FAU - Michel, Janine\n",
      "AU  - Michel J\n",
      "AD  - The European Mobile Laboratory Consortium, Hamburg, Germany; Robert Koch\n",
      "      Institute, Berlin, Germany.\n",
      "FAU - Patrono, Livia Victoria\n",
      "AU  - Patrono LV\n",
      "AD  - The European Mobile Laboratory Consortium, Hamburg, Germany;\n",
      "      Bernhard-Nocht-Institute for Tropical Medicine, Hamburg, Germany.\n",
      "FAU - Rickett, Natasha Y\n",
      "AU  - Rickett NY\n",
      "AD  - The European Mobile Laboratory Consortium, Hamburg, Germany; Institute of\n",
      "      Infection and Global Health, University of Liverpool, Liverpool, UK.\n",
      "FAU - Singethan, Katrin\n",
      "AU  - Singethan K\n",
      "AD  - The European Mobile Laboratory Consortium, Hamburg, Germany; Institute of\n",
      "      Virology, Technische Universitat Munchen, Munich, Germany.\n",
      "FAU - Rudolf, Martin\n",
      "AU  - Rudolf M\n",
      "AD  - Bernhard-Nocht-Institute for Tropical Medicine, Hamburg, Germany; German Centre\n",
      "      for Infection Research (DZIF), Partner Site Hamburg, Conakry, Guinea.\n",
      "FAU - Lander, Angelika\n",
      "AU  - Lander A\n",
      "AD  - Robert Koch Institute, Berlin, Germany.\n",
      "FAU - Pallasch, Elisa\n",
      "AU  - Pallasch E\n",
      "AD  - Bernhard-Nocht-Institute for Tropical Medicine, Hamburg, Germany; German Centre\n",
      "      for Infection Research (DZIF), Partner Site Hamburg, Conakry, Guinea.\n",
      "FAU - Bockholt, Sabrina\n",
      "AU  - Bockholt S\n",
      "AD  - Bernhard-Nocht-Institute for Tropical Medicine, Hamburg, Germany; German Centre\n",
      "      for Infection Research (DZIF), Partner Site Hamburg, Conakry, Guinea.\n",
      "FAU - Rodriguez, Estefania\n",
      "AU  - Rodriguez E\n",
      "AD  - German Centre for Infection Research (DZIF), Partner Site Hamburg, Conakry,\n",
      "      Guinea; Heinrich Pette Institute, Leibniz Institute for Experimental Virology,\n",
      "      Hamburg.\n",
      "FAU - Di Caro, Antonino\n",
      "AU  - Di Caro A\n",
      "AD  - The European Mobile Laboratory Consortium, Hamburg, Germany; National Institute\n",
      "      for Infectious Diseases Lazzaro Spallanzani, Rome, Italy.\n",
      "FAU - Wolfel, Roman\n",
      "AU  - Wolfel R\n",
      "AD  - The European Mobile Laboratory Consortium, Hamburg, Germany; Bundeswehr Institute\n",
      "      of Microbiology, Munich, Switzerland.\n",
      "FAU - Gabriel, Martin\n",
      "AU  - Gabriel M\n",
      "AD  - The European Mobile Laboratory Consortium, Hamburg, Germany;\n",
      "      Bernhard-Nocht-Institute for Tropical Medicine, Hamburg, Germany.\n",
      "FAU - Gurry, Celine\n",
      "AU  - Gurry C\n",
      "AD  - WHO, Geneva, Switzerland.\n",
      "FAU - Formenty, Pierre\n",
      "AU  - Formenty P\n",
      "AD  - WHO, Geneva, Switzerland.\n",
      "FAU - Keita, Sakoba\n",
      "AU  - Keita S\n",
      "AD  - Ministry of Health Guinea, Conakry, Guinea.\n",
      "FAU - Malvy, Denis\n",
      "AU  - Malvy D\n",
      "AD  - INSERM U1219, Bordeaux University, Bordeaux, France; Bordeaux University\n",
      "      Hospital, Bordeaux, France.\n",
      "FAU - Carroll, Miles W\n",
      "AU  - Carroll MW\n",
      "AD  - The European Mobile Laboratory Consortium, Hamburg, Germany; Public Health\n",
      "      England, Porton Down, Salisbury, UK; University of Southampton, South General\n",
      "      Hospital, Southampton, UK.\n",
      "FAU - Anglaret, Xavier\n",
      "AU  - Anglaret X\n",
      "AD  - INSERM U1219, Bordeaux University, Bordeaux, France; PAC-CI, ANRS Research Site, \n",
      "      Treichville University Hospital, Abidjan, Cote d'Ivoire.\n",
      "FAU - Gunther, Stephan\n",
      "AU  - Gunther S\n",
      "AD  - The European Mobile Laboratory Consortium, Hamburg, Germany;\n",
      "      Bernhard-Nocht-Institute for Tropical Medicine, Hamburg, Germany; German Centre\n",
      "      for Infection Research (DZIF), Partner Site Hamburg, Conakry, Guinea. Electronic \n",
      "      address: guenther@bni.uni-hamburg.de.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PL  - England\n",
      "TA  - Lancet Glob Health\n",
      "JT  - The Lancet. Global health\n",
      "JID - 101613665\n",
      "EDAT- 2016/12/14 06:00\n",
      "MHDA- 2016/12/14 06:00\n",
      "CRDT- 2016/12/14 06:00\n",
      "PHST- 2016/03/10 [received]\n",
      "PHST- 2016/09/04 [revised]\n",
      "PHST- 2016/09/22 [accepted]\n",
      "AID - S2214-109X(16)30243-1 [pii]\n",
      "AID - 10.1016/S2214-109X(16)30243-1 [doi]\n",
      "PST - ppublish\n",
      "SO  - Lancet Glob Health. 2017 Jan;5(1):e80-e88. doi: 10.1016/S2214-109X(16)30243-1.\n",
      "\n",
      "PMID- 27939857\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161212\n",
      "LR  - 20161224\n",
      "IS  - 1095-953X (Electronic)\n",
      "IS  - 0969-9961 (Linking)\n",
      "VI  - 99\n",
      "DP  - 2016 Dec 09\n",
      "TI  - Blockade of the IL-1R1/TLR4 pathway mediates disease-modification therapeutic\n",
      "      effects in a model of acquired epilepsy.\n",
      "PG  - 12-23\n",
      "LID - S0969-9961(16)30288-1 [pii]\n",
      "LID - 10.1016/j.nbd.2016.12.007 [doi]\n",
      "AB  - We recently discovered that forebrain activation of the IL-1 receptor/Toll-like\n",
      "      receptor (IL-1R1/TLR4) innate immunity signal plays a pivotal role in neuronal\n",
      "      hyperexcitability underlying seizures in rodents. Since this pathway is activated\n",
      "      in neurons and glia in human epileptogenic foci, it represents a potential target\n",
      "      for developing drugs interfering with the mechanisms of epileptogenesis that lead\n",
      "      to spontaneous seizures. The lack of such drugs represents a major unmet clinical\n",
      "      need. We tested therefore novel therapies inhibiting the IL-1R1/TLR4 signaling in\n",
      "      an established murine model of acquired epilepsy. We used an epigenetic approach \n",
      "      by injecting a synthetic mimic of micro(mi)RNA-146a that impairs IL1R1/TLR4\n",
      "      signal transduction, or we blocked receptor activation with antiinflammatory\n",
      "      drugs. Both interventions when transiently applied to mice after epilepsy onset, \n",
      "      prevented disease progression and dramatically reduced chronic seizure\n",
      "      recurrence, while the anticonvulsant drug carbamazepine was ineffective. We\n",
      "      conclude that IL-1R1/TLR4 is a novel potential therapeutic target for attaining\n",
      "      disease-modifications in patients with diagnosed epilepsy.\n",
      "CI  - Copyright (c) 2016 Elsevier Inc. All rights reserved.\n",
      "FAU - Iori, Valentina\n",
      "AU  - Iori V\n",
      "AD  - Department of Neuroscience, IRCCS-Istituto di Ricerche Farmacologiche \"Mario\n",
      "      Negri\", Milano, Italy; Department of (Neuro)Pathology, Academic Medical Center,\n",
      "      Amsterdam, The Netherlands.\n",
      "FAU - Iyer, Anand M\n",
      "AU  - Iyer AM\n",
      "AD  - Department of (Neuro)Pathology, Academic Medical Center, Amsterdam, The\n",
      "      Netherlands.\n",
      "FAU - Ravizza, Teresa\n",
      "AU  - Ravizza T\n",
      "AD  - Department of Neuroscience, IRCCS-Istituto di Ricerche Farmacologiche \"Mario\n",
      "      Negri\", Milano, Italy.\n",
      "FAU - Beltrame, Luca\n",
      "AU  - Beltrame L\n",
      "AD  - Department of Oncology, IRCCS-Istituto di Ricerche Farmacologiche \"Mario Negri\", \n",
      "      Milano, Italy.\n",
      "FAU - Paracchini, Lara\n",
      "AU  - Paracchini L\n",
      "AD  - Department of Oncology, IRCCS-Istituto di Ricerche Farmacologiche \"Mario Negri\", \n",
      "      Milano, Italy.\n",
      "FAU - Marchini, Sergio\n",
      "AU  - Marchini S\n",
      "AD  - Department of Oncology, IRCCS-Istituto di Ricerche Farmacologiche \"Mario Negri\", \n",
      "      Milano, Italy.\n",
      "FAU - Cerovic, Milica\n",
      "AU  - Cerovic M\n",
      "AD  - Department of Neuroscience, IRCCS-Istituto di Ricerche Farmacologiche \"Mario\n",
      "      Negri\", Milano, Italy.\n",
      "FAU - Hill, Cameron\n",
      "AU  - Hill C\n",
      "AD  - Department of Pharmacy, University of Washington, Seattle, WA, USA.\n",
      "FAU - Ferrari, Mariella\n",
      "AU  - Ferrari M\n",
      "AD  - Department of Oncology, IRCCS-Istituto di Ricerche Farmacologiche \"Mario Negri\", \n",
      "      Milano, Italy.\n",
      "FAU - Zucchetti, Massimo\n",
      "AU  - Zucchetti M\n",
      "AD  - Department of Oncology, IRCCS-Istituto di Ricerche Farmacologiche \"Mario Negri\", \n",
      "      Milano, Italy.\n",
      "FAU - Molteni, Monica\n",
      "AU  - Molteni M\n",
      "AD  - Department of Biotechnologies and Life Sciences, Insubria University, Varese,\n",
      "      Italy.\n",
      "FAU - Rossetti, Carlo\n",
      "AU  - Rossetti C\n",
      "AD  - Department of Biotechnologies and Life Sciences, Insubria University, Varese,\n",
      "      Italy.\n",
      "FAU - Brambilla, Riccardo\n",
      "AU  - Brambilla R\n",
      "AD  - Department of Neuroscience, IRCCS-Istituto di Ricerche Farmacologiche \"Mario\n",
      "      Negri\", Milano, Italy; Neuroscience and Mental Health Research Institute,\n",
      "      Division of Neuroscience, School of Biosciences, Cardiff University, United\n",
      "      Kingdom.\n",
      "FAU - Steve White, H\n",
      "AU  - Steve White H\n",
      "AD  - Department of Pharmacy, University of Washington, Seattle, WA, USA.\n",
      "FAU - D'Incalci, Maurizio\n",
      "AU  - D'Incalci M\n",
      "AD  - Department of Oncology, IRCCS-Istituto di Ricerche Farmacologiche \"Mario Negri\", \n",
      "      Milano, Italy.\n",
      "FAU - Aronica, Eleonora\n",
      "AU  - Aronica E\n",
      "AD  - Department of (Neuro)Pathology, Academic Medical Center, Amsterdam, The\n",
      "      Netherlands; Swammerdam Institute for Life Sciences, Center for Neuroscience,\n",
      "      University of Amsterdam, The Netherlands; Stichting Epilepsie Instellingen (SEIN)\n",
      "      Nederland and Epilepsy Institute in The Netherlands Foundation, The Netherlands. \n",
      "      Electronic address: e.aronica@amc.uva.nl.\n",
      "FAU - Vezzani, Annamaria\n",
      "AU  - Vezzani A\n",
      "AD  - Department of Neuroscience, IRCCS-Istituto di Ricerche Farmacologiche \"Mario\n",
      "      Negri\", Milano, Italy. Electronic address: annamaria.vezzani@marionegri.it.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161209\n",
      "PL  - United States\n",
      "TA  - Neurobiol Dis\n",
      "JT  - Neurobiology of disease\n",
      "JID - 9500169\n",
      "OTO - NOTNLM\n",
      "OT  - Disease-modification\n",
      "OT  - Epilepsy\n",
      "OT  - Hyperexcitability\n",
      "OT  - Neuroinflammation\n",
      "OT  - Seizures\n",
      "EDAT- 2016/12/13 06:00\n",
      "MHDA- 2016/12/13 06:00\n",
      "CRDT- 2016/12/13 06:00\n",
      "PHST- 2016/10/27 [received]\n",
      "PHST- 2016/11/29 [revised]\n",
      "PHST- 2016/12/07 [accepted]\n",
      "AID - S0969-9961(16)30288-1 [pii]\n",
      "AID - 10.1016/j.nbd.2016.12.007 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Neurobiol Dis. 2016 Dec 9;99:12-23. doi: 10.1016/j.nbd.2016.12.007.\n",
      "\n",
      "PMID- 27939723\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161212\n",
      "LR  - 20161218\n",
      "IS  - 1532-0456 (Print)\n",
      "IS  - 1532-0456 (Linking)\n",
      "VI  - 192\n",
      "DP  - 2016 Dec 07\n",
      "TI  - Transcriptomic characterization of zebrafish larvae in response to mercury\n",
      "      exposure.\n",
      "PG  - 40-49\n",
      "LID - S1532-0456(16)30168-5 [pii]\n",
      "LID - 10.1016/j.cbpc.2016.12.006 [doi]\n",
      "AB  - Mercury is a widespread toxicant in aquatic environment that can cause\n",
      "      deleterious effects on fish. Although a number of mercury-regulated genes have\n",
      "      been investigated in adult fish, the transcriptional responses of fish larvae to \n",
      "      acute mercury exposure are not well understood. In this study, RNA sequencing was\n",
      "      used to examine the transcriptional changes in developing zebrafish larvae under \n",
      "      a low concentration of mercuric chloride exposure from 24 to 120hpf. Our initial \n",
      "      results showed that a total of 142.59 million raw reads were obtained from\n",
      "      sequencing libraries and about 86% of the processed reads were mapped to the\n",
      "      reference genome of zebrafish. Differential expression analysis identified 391\n",
      "      up- and 87 down-regulated genes. Gene ontology enrichment analysis revealed that \n",
      "      most of the differential expressed genes are closely related to the regulation of\n",
      "      cellular process, metabolic process, multicellular organismal process, biological\n",
      "      regulation, pigmentation, and response to stimulus. Kyoto Encyclopedia of Genes\n",
      "      and Genomes pathway enrichment analysis demonstrated that antigen processing and \n",
      "      presentation was the most significantly enriched pathway. Moreover, we\n",
      "      characterized a novel and sensitive mercury-induced ABCB (ATP- binding cassette B\n",
      "      subfamily) transporter gene - abcb5. This gene is localized on zebrafish\n",
      "      chromosome 16 and contains a 4014bp open-reading frame. The deduced polypeptide\n",
      "      is composed of 1337 amino acids and possesses most of functional domains and\n",
      "      critical residues defined in human and mouse ABCB5/Abcb5. Functional analysis in \n",
      "      vitro demonstrated that overexpression of zebrafish abcb5 gene can significantly \n",
      "      decrease the cytotoxicity of mercury in LLC-PK1 cells, implying it is a potential\n",
      "      efflux transporter of mercury. Thus, these findings provide useful insights to\n",
      "      help further understand the transcriptional response and detoxification ability\n",
      "      of zebrafish larvae following acute exposure to mercury.\n",
      "CI  - Copyright (c) 2016 Elsevier Inc. All rights reserved.\n",
      "FAU - Lu, Xing\n",
      "AU  - Lu X\n",
      "AD  - Key Laboratory of Freshwater Biodiversity Conservation and Utilization of\n",
      "      Ministry of Agriculture, Yangtze River Fisheries Research Institute, Chinese\n",
      "      Academy of Fishery Sciences, Wuhan 430223, Hubei, China. Electronic address:\n",
      "      luxing@yfi.ac.cn.\n",
      "FAU - Xiang, Ying\n",
      "AU  - Xiang Y\n",
      "AD  - School of Basic Medical Science, Wuhan University, Wuhan 430071, Hubei, China.\n",
      "      Electronic address: xiangy12580@whu.edu.cn.\n",
      "FAU - Yang, Guohua\n",
      "AU  - Yang G\n",
      "AD  - School of Basic Medical Science, Wuhan University, Wuhan 430071, Hubei, China.\n",
      "      Electronic address: ghyang@whu.edu.cn.\n",
      "FAU - Zhang, Lang\n",
      "AU  - Zhang L\n",
      "AD  - School of Basic Medical Science, Wuhan University, Wuhan 430071, Hubei, China.\n",
      "      Electronic address: zhanglang211@163.com.\n",
      "FAU - Wang, Hui\n",
      "AU  - Wang H\n",
      "AD  - School of Basic Medical Science, Wuhan University, Wuhan 430071, Hubei, China;\n",
      "      Hubei Provincial Key Laboratory of Developmentally Originated Disease, Wuhan\n",
      "      430071, Hubei, China. Electronic address: wanghui19@whu.edu.cn.\n",
      "FAU - Zhong, Shan\n",
      "AU  - Zhong S\n",
      "AD  - School of Basic Medical Science, Wuhan University, Wuhan 430071, Hubei, China;\n",
      "      Hubei Provincial Key Laboratory of Developmentally Originated Disease, Wuhan\n",
      "      430071, Hubei, China; Hubei Province Key Laboratory of Allergy and Immunology,\n",
      "      Wuhan 430071, Hubei, China. Electronic address: zhongshan@whu.edu.cn.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161207\n",
      "PL  - United States\n",
      "TA  - Comp Biochem Physiol C Toxicol Pharmacol\n",
      "JT  - Comparative biochemistry and physiology. Toxicology & pharmacology : CBP\n",
      "JID - 100959500\n",
      "OTO - NOTNLM\n",
      "OT  - Abcb5\n",
      "OT  - Gene expression\n",
      "OT  - Mercury\n",
      "OT  - RNA-seq\n",
      "OT  - Zebrafish\n",
      "EDAT- 2016/12/13 06:00\n",
      "MHDA- 2016/12/13 06:00\n",
      "CRDT- 2016/12/13 06:00\n",
      "PHST- 2016/09/14 [received]\n",
      "PHST- 2016/11/21 [revised]\n",
      "PHST- 2016/12/02 [accepted]\n",
      "AID - S1532-0456(16)30168-5 [pii]\n",
      "AID - 10.1016/j.cbpc.2016.12.006 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Comp Biochem Physiol C Toxicol Pharmacol. 2016 Dec 7;192:40-49. doi:\n",
      "      10.1016/j.cbpc.2016.12.006.\n",
      "\n",
      "PMID- 27939432\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161212\n",
      "LR  - 20161225\n",
      "IS  - 0006-3002 (Print)\n",
      "IS  - 0006-3002 (Linking)\n",
      "DP  - 2016 Dec 08\n",
      "TI  - Regulation of microRNA expression in vascular smooth muscle by MRTF-A and actin\n",
      "      polymerization.\n",
      "LID - S0167-4889(16)30330-5 [pii]\n",
      "LID - 10.1016/j.bbamcr.2016.12.005 [doi]\n",
      "AB  - The dynamic properties of the actin cytoskeleton in smooth muscle cells play an\n",
      "      important role in a number of cardiovascular disease states. The state of actin\n",
      "      does not only mediate mechanical stability and contractile function but can also \n",
      "      regulate gene expression via myocardin related transcription factors (MRTFs).\n",
      "      These transcriptional co-activators regulate genes encoding contractile and\n",
      "      cytoskeletal proteins in smooth muscle. Regulation of small non-coding microRNAs \n",
      "      (miRNAs) by actin polymerization may mediate some of these effects. MiRNAs are\n",
      "      short non-coding RNAs that modulate gene expression by post-transcriptional\n",
      "      regulation of target messenger RNA. In this study we aimed to determine a profile\n",
      "      of miRNAs that were 1) regulated by actin/MRTF-A, 2) associated with the\n",
      "      contractile smooth muscle phenotype and 3) enriched in muscle cells. This\n",
      "      analysis was performed using cardiovascular disease-focused miRNA arrays in both \n",
      "      mouse and human cells. The potential clinical importance of actin polymerization \n",
      "      in aortic aneurysm was evaluated using biopsies from mildly dilated human\n",
      "      thoracic aorta in patients with stenotic tricuspid or bicuspid aortic valve. By\n",
      "      integrating information from multiple qPCR based miRNA arrays we identified a\n",
      "      group of five miRNAs (miR-1, miR-22, miR-143, miR-145 and miR-378a) that were\n",
      "      sensitive to actin polymerization and MRTF-A overexpression in both mouse and\n",
      "      human vascular smooth muscle. With the exception of miR-22, these miRNAs were\n",
      "      also relatively enriched in striated and/or smooth muscle containing tissues.\n",
      "      Actin polymerization was found to be dramatically reduced in the aorta from\n",
      "      patients with mild aortic dilations. This was associated with a decrease in\n",
      "      actin/MRTF-regulated miRNAs. In conclusion, the transcriptional co-activator\n",
      "      MRTF-A and actin polymerization regulated a subset of miRNAs in vascular smooth\n",
      "      muscle. Identification of novel miRNAs regulated by actin/MRTF-A may provide\n",
      "      further insight into the mechanisms underlying vascular disease states, such as\n",
      "      aortic aneurysm, as well as novel ideas regarding therapeutic strategies. This\n",
      "      article is part of a Special Issue entitled: ECS Meeting edited by Claus\n",
      "      Heizmann, Joachim Krebs and Jacques Haiech.\n",
      "CI  - Copyright (c) 2016 Elsevier B.V. All rights reserved.\n",
      "FAU - Alajbegovic, Azra\n",
      "AU  - Alajbegovic A\n",
      "AD  - Department of Experimental Medical Science, Lund University, Lund, Sweden.\n",
      "FAU - Turczynska, Karolina M\n",
      "AU  - Turczynska KM\n",
      "AD  - Department of Experimental Medical Science, Lund University, Lund, Sweden.\n",
      "FAU - Hien, Tran Thi\n",
      "AU  - Hien TT\n",
      "AD  - Department of Experimental Medical Science, Lund University, Lund, Sweden.\n",
      "FAU - Cidad, Pilar\n",
      "AU  - Cidad P\n",
      "AD  - Departamento de Bioquimica y Biologia Molecular y Fisiologia and Instituto de\n",
      "      Biologia y Genetica Molecular (IBGM), Universidad de Valladolid and Consejo\n",
      "      Superior de Investigaciones Cientificas (CSIC), Valladolid, Spain.\n",
      "FAU - Sward, Karl\n",
      "AU  - Sward K\n",
      "AD  - Department of Experimental Medical Science, Lund University, Lund, Sweden.\n",
      "FAU - Hellstrand, Per\n",
      "AU  - Hellstrand P\n",
      "AD  - Department of Experimental Medical Science, Lund University, Lund, Sweden.\n",
      "FAU - Corte, Alessandro Della\n",
      "AU  - Corte AD\n",
      "AD  - Department of Cardiothoracic Sciences, Second University of Naples, Naples,\n",
      "      Italy.\n",
      "FAU - Forte, Amalia\n",
      "AU  - Forte A\n",
      "AD  - Department of Experimental Medicine, Second University of Naples, Naples, Italy.\n",
      "FAU - Albinsson, Sebastian\n",
      "AU  - Albinsson S\n",
      "AD  - Department of Experimental Medical Science, Lund University, Lund, Sweden.\n",
      "      Electronic address: sebastian.albinsson@med.lu.se.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161208\n",
      "PL  - Netherlands\n",
      "TA  - Biochim Biophys Acta\n",
      "JT  - Biochimica et biophysica acta\n",
      "JID - 0217513\n",
      "OTO - NOTNLM\n",
      "OT  - Actin polymerization\n",
      "OT  - MRTF-A\n",
      "OT  - MicroRNA\n",
      "OT  - Phenotype\n",
      "OT  - Smooth muscle\n",
      "OT  - Vascular disease\n",
      "EDAT- 2016/12/13 06:00\n",
      "MHDA- 2016/12/13 06:00\n",
      "CRDT- 2016/12/13 06:00\n",
      "PHST- 2016/09/30 [received]\n",
      "PHST- 2016/12/05 [revised]\n",
      "PHST- 2016/12/06 [accepted]\n",
      "AID - S0167-4889(16)30330-5 [pii]\n",
      "AID - 10.1016/j.bbamcr.2016.12.005 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Biochim Biophys Acta. 2016 Dec 8. pii: S0167-4889(16)30330-5. doi:\n",
      "      10.1016/j.bbamcr.2016.12.005.\n",
      "\n",
      "PMID- 27933543\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "DA  - 20161209\n",
      "LR  - 20161215\n",
      "IS  - 1940-6029 (Electronic)\n",
      "IS  - 1064-3745 (Linking)\n",
      "VI  - 1488\n",
      "DP  - 2017\n",
      "TI  - Integrative Analysis of Genetic, Genomic, and Phenotypic Data for Ethanol\n",
      "      Behaviors: A Network-Based Pipeline for Identifying Mechanisms and Potential Drug\n",
      "      Targets.\n",
      "PG  - 531-549\n",
      "AB  - Complex behavioral traits, such as alcohol abuse, are caused by an interplay of\n",
      "      genetic and environmental factors, producing deleterious functional adaptations\n",
      "      in the central nervous system. The long-term behavioral consequences of such\n",
      "      changes are of substantial cost to both the individual and society. Substantial\n",
      "      progress has been made in the last two decades in understanding elements of brain\n",
      "      mechanisms underlying responses to ethanol in animal models and risk factors for \n",
      "      alcohol use disorder (AUD) in humans. However, treatments for AUD remain largely \n",
      "      ineffective and few medications for this disease state have been licensed.\n",
      "      Genome-wide genetic polymorphism analysis (GWAS) in humans, behavioral genetic\n",
      "      studies in animal models and brain gene expression studies produced by\n",
      "      microarrays or RNA-seq have the potential to produce nonbiased and novel insight \n",
      "      into the underlying neurobiology of AUD. However, the complexity of such\n",
      "      information, both statistical and informational, has slowed progress toward\n",
      "      identifying new targets for intervention in AUD. This chapter describes one\n",
      "      approach for integrating behavioral, genetic, and genomic information across\n",
      "      animal model and human studies. The goal of this approach is to identify networks\n",
      "      of genes functioning in the brain that are most relevant to the underlying\n",
      "      mechanisms of a complex disease such as AUD. We illustrate an example of how\n",
      "      genomic studies in animal models can be used to produce robust gene networks that\n",
      "      have functional implications, and to integrate such animal model genomic data\n",
      "      with human genetic studies such as GWAS for AUD. We describe several useful\n",
      "      analysis tools for such studies: ComBAT, WGCNA, and EW_dmGWAS. The end result of \n",
      "      this analysis is a ranking of gene networks and identification of their cognate\n",
      "      hub genes, which might provide eventual targets for future therapeutic\n",
      "      development. Furthermore, this combined approach may also improve our\n",
      "      understanding of basic mechanisms underlying gene x environmental interactions\n",
      "      affecting brain functioning in health and disease.\n",
      "FAU - Bogenpohl, James W\n",
      "AU  - Bogenpohl JW\n",
      "AD  - Department of Pharmacology and Toxicology, VCU Alcohol Research Center, Virginia \n",
      "      Commonwealth University, 980613, Richmond, VA, 23298, USA.\n",
      "FAU - Mignogna, Kristin M\n",
      "AU  - Mignogna KM\n",
      "AD  - Department of Psychiatry, VCU Alcohol Research Center, Virginia Commonwealth\n",
      "      University, Richmond, VA, 23298, USA.\n",
      "FAU - Smith, Maren L\n",
      "AU  - Smith ML\n",
      "AD  - Department of Human and Molecular Genetics, VCU Alcohol Research Center, Virginia\n",
      "      Commonwealth University, Richmond, VA, 23298, USA.\n",
      "FAU - Miles, Michael F\n",
      "AU  - Miles MF\n",
      "AD  - Department of Pharmacology and Toxicology, VCU Alcohol Research Center, Virginia \n",
      "      Commonwealth University, 980613, Richmond, VA, 23298, USA.\n",
      "      Michael.Miles@vcuhealth.org.\n",
      "LA  - eng\n",
      "GR  - U01 AA016667/AA/NIAAA NIH HHS/United States\n",
      "GR  - UL1 TR000058/TR/NCATS NIH HHS/United States\n",
      "GR  - T32 DK007150/DK/NIDDK NIH HHS/United States\n",
      "GR  - F31 AA023134/AA/NIAAA NIH HHS/United States\n",
      "GR  - P50 AA022537/AA/NIAAA NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PL  - United States\n",
      "TA  - Methods Mol Biol\n",
      "JT  - Methods in molecular biology (Clifton, N.J.)\n",
      "JID - 9214969\n",
      "PMC - PMC5152688\n",
      "MID - NIHMS783422\n",
      "OTO - NOTNLM\n",
      "OT  - Alcoholism\n",
      "OT  - Bioinformatics\n",
      "OT  - Brain\n",
      "OT  - Ethanol\n",
      "OT  - Gene networks\n",
      "OT  - Genetics\n",
      "OT  - Genomics\n",
      "OT  - Mouse\n",
      "OT  - Use case\n",
      "EDAT- 2016/12/10 06:00\n",
      "MHDA- 2016/12/10 06:00\n",
      "CRDT- 2016/12/10 06:00\n",
      "PMCR- 2018/01/01\n",
      "AID - 10.1007/978-1-4939-6427-7_26 [doi]\n",
      "PST - ppublish\n",
      "SO  - Methods Mol Biol. 2017;1488:531-549.\n",
      "\n",
      "PMID- 27933528\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "DA  - 20161209\n",
      "LR  - 20161210\n",
      "IS  - 1940-6029 (Electronic)\n",
      "IS  - 1064-3745 (Linking)\n",
      "VI  - 1488\n",
      "DP  - 2017\n",
      "TI  - RNA-Seq in the Collaborative Cross.\n",
      "PG  - 251-263\n",
      "AB  - The Collaborative Cross (CC) is a large panel of inbred mouse strains currently\n",
      "      being developed for multiple areas of research. Scientists are taking integrated \n",
      "      omics-style approaches to collecting data in order to obtain a deeper\n",
      "      understanding of the biological mechanisms underlying a number of diverse disease\n",
      "      phenotypes. As the cost of the next generation sequencing (NGS) decreases,\n",
      "      RNA-sequencing (RNA-seq) has become the favored approach to transcriptomic\n",
      "      analyses versus microarrays due to increases in sensitivity and resolution. This \n",
      "      is particularly the case with newly defined genomes, where experimental\n",
      "      annotation has not caught up to the new microarray platforms. Traditional RNA-seq\n",
      "      approaches are not ideal when working with results from collaborative cross\n",
      "      studies, as the genomes across individual strains differ considerably. In this\n",
      "      chapter we will provide an overview of how to effectively perform RNA-seq\n",
      "      analysis from data obtained from the CC mice.\n",
      "FAU - Green, Richard\n",
      "AU  - Green R\n",
      "AD  - Department of Immunology, Center for Innate Immunity and Immune Disease,\n",
      "      University of Washington School of Medicine, 750 Republican St., Seattle, WA,\n",
      "      98109, USA. greener@uw.edu.\n",
      "FAU - Wilkins, Courtney\n",
      "AU  - Wilkins C\n",
      "AD  - Department of Immunology, Center for Innate Immunity and Immune Disease,\n",
      "      University of Washington School of Medicine, 750 Republican St., Seattle, WA,\n",
      "      98109, USA.\n",
      "FAU - Ferris, Martin T\n",
      "AU  - Ferris MT\n",
      "AD  - Department of Genetics, University of North Carolina at Chapel Hill, 5000 D,\n",
      "      Genetic Medicine Bldg., CB#7264, 120 mason Farm Road, Chapel Hill, NC, 27599,\n",
      "      USA.\n",
      "FAU - Gale, Michael Jr\n",
      "AU  - Gale M Jr\n",
      "AD  - Department of Immunology, Center for Innate Immunity and Immune Disease,\n",
      "      University of Washington School of Medicine, 750 Republican St., Seattle, WA,\n",
      "      98109, USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PL  - United States\n",
      "TA  - Methods Mol Biol\n",
      "JT  - Methods in molecular biology (Clifton, N.J.)\n",
      "JID - 9214969\n",
      "OTO - NOTNLM\n",
      "OT  - Analysis tools\n",
      "OT  - Collaborative Cross\n",
      "OT  - RNAseq\n",
      "EDAT- 2016/12/10 06:00\n",
      "MHDA- 2016/12/10 06:00\n",
      "CRDT- 2016/12/10 06:00\n",
      "AID - 10.1007/978-1-4939-6427-7_11 [doi]\n",
      "PST - ppublish\n",
      "SO  - Methods Mol Biol. 2017;1488:251-263.\n",
      "\n",
      "PMID- 27932527\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161209\n",
      "LR  - 20161210\n",
      "IS  - 1535-9484 (Electronic)\n",
      "IS  - 1535-9476 (Linking)\n",
      "DP  - 2016 Dec 08\n",
      "TI  - Mechanisms of in vivo ribosome maintenance change in response to nutrient\n",
      "      signals.\n",
      "LID - mcp.M116.063255 [pii]\n",
      "AB  - Control of protein homeostasis is fundamental to the health and longevity of all \n",
      "      organisms. Because the rate of protein synthesis by ribosomes is a central\n",
      "      control point in this process, regulation and maintenance of ribosome function\n",
      "      could have amplified importance in the overall regulatory circuit. Indeed,\n",
      "      ribosomal defects are commonly associated with loss of protein homeostasis, aging\n",
      "      and disease, whereas improved protein homeostasis, implying optimal ribosomal\n",
      "      function, is associated with disease resistance and increased lifespan. To\n",
      "      maintain a high quality ribosome population within the cell, dysfunctional\n",
      "      ribosomes are targeted for autophagic degradation. It is not known if complete\n",
      "      degradation is the only mechanism for eukaryotic ribosome maintenance or if they \n",
      "      might also be repaired by replacement of defective components. We used\n",
      "      stable-isotope feeding and protein mass-spectrometry to measure the kinetics of\n",
      "      turnover of ribosomal RNA (rRNA) and 71 ribosomal proteins (r-proteins) in mice. \n",
      "      The results indicate that exchange of individual proteins and whole ribosome\n",
      "      degradation both contribute to ribosome maintenance in vivo. In general,\n",
      "      peripheral r-proteins and those with more direct roles in peptide-bond formation \n",
      "      are replaced multiple times during the lifespan of the assembled structure,\n",
      "      presumably by exchange with a free cytoplasmic pool, whereas the majority of\n",
      "      r-proteins are stably incorporated for the lifetime of the ribosome. Dietary\n",
      "      signals impact the rates of both new ribosome assembly and component exchange.\n",
      "      Signal-specific modulation of ribosomal repair and degradation could provide a\n",
      "      mechanistic link in the frequently observed associations among diminished rates\n",
      "      of protein synthesis, increased autophagy, and greater longevity.\n",
      "CI  - Copyright (c) 2016, The American Society for Biochemistry and Molecular Biology.\n",
      "FAU - Mathis, Andrew D\n",
      "AU  - Mathis AD\n",
      "AD  - Brigham Young University, United States.\n",
      "FAU - Naylor, Bradley C\n",
      "AU  - Naylor BC\n",
      "AD  - Brigham Young University, United States.\n",
      "FAU - Carson, Richard H\n",
      "AU  - Carson RH\n",
      "AD  - Brigham Young University, United States.\n",
      "FAU - Evans, Eric\n",
      "AU  - Evans E\n",
      "AD  - Brigham Young University, United States.\n",
      "FAU - Harwell, Justin\n",
      "AU  - Harwell J\n",
      "AD  - Brigham Young University, United States.\n",
      "FAU - Knect, Jared\n",
      "AU  - Knect J\n",
      "AD  - Brigham Young University, United States.\n",
      "FAU - Hexem, Eric\n",
      "AU  - Hexem E\n",
      "AD  - Brigham Young University, United States.\n",
      "FAU - Peelor, Fredrick 3rd\n",
      "AU  - Peelor F 3rd\n",
      "AD  - CSU, United States.\n",
      "FAU - Miller, Benjamin F\n",
      "AU  - Miller BF\n",
      "AD  - CSU, United States.\n",
      "FAU - Hamilton, Karyn L\n",
      "AU  - Hamilton KL\n",
      "AD  - CSU, United States.\n",
      "FAU - Transtrum, Mark K\n",
      "AU  - Transtrum MK\n",
      "AD  - Brigham Young University, United States.\n",
      "FAU - Bikman, Benjamin T\n",
      "AU  - Bikman BT\n",
      "AD  - Brigham Young University, United States.\n",
      "FAU - Price, John C\n",
      "AU  - Price JC\n",
      "AD  - Brigham Young University, United States; jcprice@chem.byu.edu.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161208\n",
      "PL  - United States\n",
      "TA  - Mol Cell Proteomics\n",
      "JT  - Molecular & cellular proteomics : MCP\n",
      "JID - 101125647\n",
      "OTO - NOTNLM\n",
      "OT  - Aging\n",
      "OT  - Enzyme Structure*\n",
      "OT  - Protein Turnover*\n",
      "OT  - Ribonucleoproteins\n",
      "OT  - Ribosomes*\n",
      "OT  - Translation*\n",
      "EDAT- 2016/12/10 06:00\n",
      "MHDA- 2016/12/10 06:00\n",
      "CRDT- 2016/12/10 06:00\n",
      "PHST- 2016/08/30 [received]\n",
      "PHST- 2016/11/10 [revised]\n",
      "PHST- 2016/12/08 [accepted]\n",
      "AID - M116.063255 [pii]\n",
      "AID - 10.1074/mcp.M116.063255 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Mol Cell Proteomics. 2016 Dec 8. pii: mcp.M116.063255.\n",
      "\n",
      "PMID- 27931246\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "DA  - 20161209\n",
      "LR  - 20161216\n",
      "IS  - 1756-6606 (Electronic)\n",
      "IS  - 1756-6606 (Linking)\n",
      "VI  - 9\n",
      "IP  - 1\n",
      "DP  - 2016 Dec 08\n",
      "TI  - Increased alternate splicing of Htr2c in a mouse model for Prader-Willi syndrome \n",
      "      leads disruption of 5HT2C receptor mediated appetite.\n",
      "PG  - 95\n",
      "AB  - Alternate splicing of serotonin (5-hydroxytryptamine; 5-HT) 2C receptor (5-HT2CR)\n",
      "      pre-RNA is negatively regulated by the small nucleolar RNA, Snord115, loss of\n",
      "      which is observed in nearly all individuals with Prader-Willi Syndrome (PWS), a\n",
      "      multigenic disorder characterised by hyperphagia and obesity. Given the role of\n",
      "      the 5-HT2CR in the regulation of ingestive behaviour we investigated the\n",
      "      pathophysiological implications of Snord115 deficiency on 5-HT2CR regulated\n",
      "      appetite in a genotypically relevant PWS mouse model (PWS-IC). Specifically, we\n",
      "      demonstrate that loss of Snord115 expression is associated with increased levels \n",
      "      of hypothalamic truncated 5-HT2CR pre-mRNA. The 5-HT2CR promotes appetite\n",
      "      suppression via engagement of the central melanocortin system.\n",
      "      Pro-opiomelancortin (Pomc) mRNA levels within the arcuate nucleus of the\n",
      "      hypothalamus (ARC) were reduced in PWS-IC mice. We then went on to assess the\n",
      "      functional consequences of these molecular changes, demonstrating that PWS-IC\n",
      "      mice are unresponsive to an anorectic doses of a 5-HT2CR agonist and that this is\n",
      "      associated with attenuated activation of POMC neurons within the ARC. These data \n",
      "      provide new insight into the significance of Htr2c pre-mRNA processing to the\n",
      "      physiological regulation of appetite and potentially the pathological\n",
      "      manifestation of hyperphagia in PWS. Furthermore, these findings have\n",
      "      translational relevance for individuals with PWS who may seek to control appetite\n",
      "      with another 5-HT2CR agonist, the new obesity treatment lorcaserin.\n",
      "FAU - Garfield, Alastair S\n",
      "AU  - Garfield AS\n",
      "AD  - Centre for Integrative Physiology, University of Edinburgh, Edinburgh, UK.\n",
      "AD  - Present address: Cardiovascular and Metabolic Disease, Pfizer, Cambridge, MA,\n",
      "      02139, USA.\n",
      "FAU - Davies, Jennifer R\n",
      "AU  - Davies JR\n",
      "AD  - Behavioural Genetics Group, MRC Centre for Neuropsychiatric Genetics and\n",
      "      Genomics, Neuroscience and Mental Health Research Institute, Schools of Medicine \n",
      "      and Pscyhology, Cardiff University, Cardiff, UK.\n",
      "FAU - Burke, Luke K\n",
      "AU  - Burke LK\n",
      "AD  - Department of Pharmacology, University of Cambridge, Cambridge, UK.\n",
      "FAU - Furby, Hannah V\n",
      "AU  - Furby HV\n",
      "AD  - Behavioural Genetics Group, MRC Centre for Neuropsychiatric Genetics and\n",
      "      Genomics, Neuroscience and Mental Health Research Institute, Schools of Medicine \n",
      "      and Pscyhology, Cardiff University, Cardiff, UK.\n",
      "FAU - Wilkinson, Lawrence S\n",
      "AU  - Wilkinson LS\n",
      "AD  - Behavioural Genetics Group, MRC Centre for Neuropsychiatric Genetics and\n",
      "      Genomics, Neuroscience and Mental Health Research Institute, Schools of Medicine \n",
      "      and Pscyhology, Cardiff University, Cardiff, UK.\n",
      "FAU - Heisler, Lora K\n",
      "AU  - Heisler LK\n",
      "AD  - Rowett Institute of Nutrition and Health, University of Aberdeen, Aberdeen, UK.\n",
      "FAU - Isles, Anthony R\n",
      "AU  - Isles AR\n",
      "AD  - Behavioural Genetics Group, MRC Centre for Neuropsychiatric Genetics and\n",
      "      Genomics, Neuroscience and Mental Health Research Institute, Schools of Medicine \n",
      "      and Pscyhology, Cardiff University, Cardiff, UK. IslesAR1@cardiff.ac.uk.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161208\n",
      "PL  - England\n",
      "TA  - Mol Brain\n",
      "JT  - Molecular brain\n",
      "JID - 101468876\n",
      "PMC - PMC5144496\n",
      "OTO - NOTNLM\n",
      "OT  - Alternate splicing\n",
      "OT  - Feeding\n",
      "OT  - Prader-Willi syndrome\n",
      "OT  - Serotonin 2C recptor\n",
      "OT  - Snord115\n",
      "EDAT- 2016/12/10 06:00\n",
      "MHDA- 2016/12/10 06:00\n",
      "CRDT- 2016/12/10 06:00\n",
      "PHST- 2016/05/25 [received]\n",
      "PHST- 2016/11/30 [accepted]\n",
      "AID - 10.1186/s13041-016-0277-4 [doi]\n",
      "AID - 10.1186/s13041-016-0277-4 [pii]\n",
      "PST - epublish\n",
      "SO  - Mol Brain. 2016 Dec 8;9(1):95.\n",
      "\n",
      "PMID- 27930406\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "DA  - 20161208\n",
      "LR  - 20161221\n",
      "IS  - 1536-4844 (Electronic)\n",
      "IS  - 1078-0998 (Linking)\n",
      "VI  - 23\n",
      "IP  - 1\n",
      "DP  - 2017 Jan\n",
      "TI  - Stress Response Protein RBM3 Promotes the Development of Colitis-associated\n",
      "      Cancer.\n",
      "PG  - 57-65\n",
      "LID - 10.1097/MIB.0000000000000968 [doi]\n",
      "AB  - BACKGROUND: Colitis-associated cancer (CAC) is caused by chronic intestinal\n",
      "      inflammation and often results from refractory inflammatory bowel disease (IBD). \n",
      "      Stress response proteins Cirp and HSPA4 are involved in the refractory clinical\n",
      "      course and development of CAC. RNA-binding motif protein 3 (RBM3) is induced in\n",
      "      response to various stresses and is upregulated in several cancers. However, the \n",
      "      role of RBM3 in CAC is unclear. METHODS: We assessed RBM3 expression and function\n",
      "      in 263 human intestinal mucosa samples from patients with IBD and in\n",
      "      Rbm3-deficient (Rbm3) mice. RESULTS: Expression of RBM3 was correlated with the\n",
      "      expression of stress response proteins Cirp, HSPA4, and HSP27 in the colonic\n",
      "      mucosa of patients with IBD. Significant correlation was observed between the\n",
      "      expression of RBM3 and that of Bcl-xL or stem cell markers. RBM3 expression\n",
      "      increased and significantly correlated with R-spondin expression in the colonic\n",
      "      mucosa of patients with refractory IBD, a condition associated with increased\n",
      "      cancer risk, and RBM3 was overexpressed in human CACs. In the murine CAC model,\n",
      "      Rbm3 deficiency decreased R-spondin and Bcl-xL expression and increased apoptotic\n",
      "      cell number in the colonic mucosa, leading to reduced tumor multiplicity.\n",
      "      Transplantation of wild-type and Rbm3 bone marrow did not alter tumor burden,\n",
      "      indicating the importance of RBM3 in epithelial cells. CONCLUSIONS: Our findings \n",
      "      indicated that RBM3 was required for efficient inflammatory carcinogenesis in the\n",
      "      murine CAC model and suggested that RBM3 could be a predictive biomarker of CAC\n",
      "      risk and a new therapeutic target for cancer prevention in patients with IBD.\n",
      "FAU - Sakurai, Toshiharu\n",
      "AU  - Sakurai T\n",
      "AD  - *Department of Gastroenterology and Hepatology, Faculty of Medicine, Kindai\n",
      "      University, Osaka-Sayama, Japan; daggerDepartment of Clinical Molecular Biology, \n",
      "      Graduate School of Medicine, Kyoto University, Kyoto, Japan; and double\n",
      "      daggerKaifukuki Rehabilitation Ward, Biwako-Chuo Hospital, Otsu, Japan.\n",
      "FAU - Kashida, Hiroshi\n",
      "AU  - Kashida H\n",
      "FAU - Komeda, Yoriaki\n",
      "AU  - Komeda Y\n",
      "FAU - Nagai, Tomoyuki\n",
      "AU  - Nagai T\n",
      "FAU - Hagiwara, Satoru\n",
      "AU  - Hagiwara S\n",
      "FAU - Watanabe, Tomohiro\n",
      "AU  - Watanabe T\n",
      "FAU - Kitano, Masayuki\n",
      "AU  - Kitano M\n",
      "FAU - Nishida, Naoshi\n",
      "AU  - Nishida N\n",
      "FAU - Fujita, Jun\n",
      "AU  - Fujita J\n",
      "FAU - Kudo, Masatoshi\n",
      "AU  - Kudo M\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PL  - United States\n",
      "TA  - Inflamm Bowel Dis\n",
      "JT  - Inflammatory bowel diseases\n",
      "JID - 9508162\n",
      "EDAT- 2016/12/09 06:00\n",
      "MHDA- 2016/12/09 06:00\n",
      "CRDT- 2016/12/09 06:00\n",
      "AID - 10.1097/MIB.0000000000000968 [doi]\n",
      "PST - ppublish\n",
      "SO  - Inflamm Bowel Dis. 2017 Jan;23(1):57-65. doi: 10.1097/MIB.0000000000000968.\n",
      "\n",
      "PMID- 27929729\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161208\n",
      "LR  - 20161209\n",
      "IS  - 1554-8635 (Electronic)\n",
      "IS  - 1554-8627 (Linking)\n",
      "DP  - 2016 Dec 08\n",
      "TI  - FYCO1 and autophagy control the integrity of the haploid male germ cell-specific \n",
      "      RNP granules.\n",
      "PG  - 0\n",
      "AB  - Ribonucleoprotein (RNP) granules play a major role in compartmentalizing\n",
      "      cytoplasmic RNA regulation. Haploid round spermatids that have exceptionally\n",
      "      diverse transcriptomes are characterized by a unique germ cell-specific RNP\n",
      "      granule, the chromatoid body (CB). The CB shares many characteristics with\n",
      "      somatic RNP granules but also has germline-specific features. The CB appears to\n",
      "      be a central structure in PIWI-interacting RNA (piRNA)-targeted RNA regulation.\n",
      "      Here, we identified a novel CB component, FYCO1, which is involved in the\n",
      "      intracellular transport of autophagic vesicles in somatic cells. We demonstrated \n",
      "      that the CB is associated with autophagic activity. Induction of autophagy leads \n",
      "      to the recruitment of lysosomal vesicles onto the CB in a FYCO1-dependent manner \n",
      "      as demonstrated by the analysis of a germ cell-specific Fyco1 conditional\n",
      "      knockout mouse model. Furthermore, in the absence of FYCO1, the integrity of the \n",
      "      CB was affected and the CB was fragmented. Our results suggest that RNP granule\n",
      "      homeostasis is regulated by FYCO1-mediated autophagy.\n",
      "FAU - Da Ros, Matteo\n",
      "AU  - Da Ros M\n",
      "AD  - a Institute of Biomedicine, Department of Physiology , University of Turku ,\n",
      "      Turku , Finland.\n",
      "AD  - b Department of Cellular and Molecular Biology , Faculty of Medicine, University \n",
      "      of Ottawa , Ottawa , ON , Canada K1H8M8.\n",
      "FAU - Lehtiniemi, Tiina\n",
      "AU  - Lehtiniemi T\n",
      "AD  - a Institute of Biomedicine, Department of Physiology , University of Turku ,\n",
      "      Turku , Finland.\n",
      "FAU - Olotu, Opeyemi\n",
      "AU  - Olotu O\n",
      "AD  - a Institute of Biomedicine, Department of Physiology , University of Turku ,\n",
      "      Turku , Finland.\n",
      "FAU - Fischer, Daniel\n",
      "AU  - Fischer D\n",
      "AD  - c Natural Resources Institute Finland (Luke), Green Technology , FI-31600\n",
      "      Jokioinen , Finland.\n",
      "FAU - Zhang, Fu-Ping\n",
      "AU  - Zhang FP\n",
      "AD  - a Institute of Biomedicine, Department of Physiology , University of Turku ,\n",
      "      Turku , Finland.\n",
      "AD  - d Turku Center for Disease Modeling, University of Turku , Turku , Finland.\n",
      "FAU - Vihinen, Helena\n",
      "AU  - Vihinen H\n",
      "AD  - e Electron Microscopy Unit, Institute of Biotechnology, University of Helsinki , \n",
      "      Helsinki , Finland.\n",
      "FAU - Jokitalo, Eija\n",
      "AU  - Jokitalo E\n",
      "AD  - e Electron Microscopy Unit, Institute of Biotechnology, University of Helsinki , \n",
      "      Helsinki , Finland.\n",
      "FAU - Sironen, Anu\n",
      "AU  - Sironen A\n",
      "AD  - c Natural Resources Institute Finland (Luke), Green Technology , FI-31600\n",
      "      Jokioinen , Finland.\n",
      "FAU - Toppari, Jorma\n",
      "AU  - Toppari J\n",
      "AD  - a Institute of Biomedicine, Department of Physiology , University of Turku ,\n",
      "      Turku , Finland.\n",
      "AD  - f Department of Pediatrics , University of Turku and Turku University Hospital , \n",
      "      Turku , Finland.\n",
      "FAU - Kotaja, Noora\n",
      "AU  - Kotaja N\n",
      "AD  - b Department of Cellular and Molecular Biology , Faculty of Medicine, University \n",
      "      of Ottawa , Ottawa , ON , Canada K1H8M8.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161208\n",
      "PL  - United States\n",
      "TA  - Autophagy\n",
      "JT  - Autophagy\n",
      "JID - 101265188\n",
      "OTO - NOTNLM\n",
      "OT  - FYCO1\n",
      "OT  - RNP granule\n",
      "OT  - autophagy\n",
      "OT  - chromatoid body\n",
      "OT  - germ cell\n",
      "OT  - knockout\n",
      "OT  - lysosome\n",
      "OT  - posttranscriptional\n",
      "OT  - spermatogenesis\n",
      "OT  - vesicle\n",
      "EDAT- 2016/12/09 06:00\n",
      "MHDA- 2016/12/09 06:00\n",
      "CRDT- 2016/12/09 06:00\n",
      "AID - 10.1080/15548627.2016.1261319 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Autophagy. 2016 Dec 8:0.\n",
      "\n",
      "PMID- 27927222\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "DA  - 20161208\n",
      "LR  - 20161216\n",
      "IS  - 1742-2094 (Electronic)\n",
      "IS  - 1742-2094 (Linking)\n",
      "VI  - 13\n",
      "IP  - 1\n",
      "DP  - 2016 Dec 07\n",
      "TI  - Tollip, an early regulator of the acute inflammatory response in the substantia\n",
      "      nigra.\n",
      "PG  - 303\n",
      "AB  - BACKGROUND: Tollip is a ubiquitously expressed protein, originally described as a\n",
      "      modulator of the IL-1R/TLR-NF-kappaB signaling pathways. Although this property\n",
      "      has been well characterized in peripheral cells, and despite some evidence of its\n",
      "      expression in the central nervous system, the role of Tollip in neuroinflammation\n",
      "      remains poorly understood. The present study sought to explore the implication of\n",
      "      Tollip in inflammation in the substantia nigra pars compacta, the structure\n",
      "      affected in Parkinson's disease. METHODS: We first investigated Tollip\n",
      "      distribution in the midbrain by immunohistochemistry. Then, we addressed\n",
      "      TLR4-mediated response by intra-nigral injections of lipopolysaccharide (LPS), a \n",
      "      TLR4 agonist, on inflammatory markers in Tollip knockout (KO) and wild-type (WT) \n",
      "      mice. RESULTS: We report an unexpectedly high Tollip immunostaining in\n",
      "      dopaminergic neurons of the mice brain. Second, intra-nigral injection of LPS led\n",
      "      to increased susceptibility to neuroinflammation in Tollip KO compared to Tollip \n",
      "      WT mice. This was demonstrated by a significant increase of tumor necrosis factor\n",
      "      alpha (TNF-alpha), interleukin 1 beta (IL-1beta), interleukin 6 (IL-6), and\n",
      "      interferon gamma (IFN-gamma) messenger RNA (mRNA) in the midbrain of Tollip KO\n",
      "      mice upon LPS injection. Consistently, brain rAAV viral vector transduction with \n",
      "      a nuclear factor kappa B (NF-kappaB)-inducible reporter gene confirmed increased \n",
      "      NF-kappaB activation in Tollip KO mice. Lastly, Tollip KO mice displayed higher\n",
      "      inducible NO synthase (iNOS) production, both at the messenger and protein level \n",
      "      when compared to LPS-injected WT mice. Tollip deletion also aggravated\n",
      "      LPS-induced oxidative and nitrosative damages, as indicated by an increase of\n",
      "      8-oxo-2'-deoxyguanosine and nitrotyrosine immunostaining, respectively.\n",
      "      CONCLUSIONS: Altogether, these findings highlight a critical role of Tollip in\n",
      "      the early phase of TLR4-mediated neuroinflammation. As brain inflammation is\n",
      "      known to contribute to Parkinson's disease, Tollip may be a potential target for \n",
      "      neuroprotection.\n",
      "FAU - Humbert-Claude, Marie\n",
      "AU  - Humbert-Claude M\n",
      "AUID- ORCID: http://orcid.org/0000-0001-6610-1662\n",
      "AD  - Laboratory of Cellular and Molecular Neurotherapies, Center for Neuroscience\n",
      "      Research, Department of Clinical Neuroscience, Lausanne University Hospital,\n",
      "      Lausanne, Switzerland. marie.humbert-claude@chuv.ch.\n",
      "FAU - Duc, D\n",
      "AU  - Duc D\n",
      "AD  - Laboratory of Cellular and Molecular Neurotherapies, Center for Neuroscience\n",
      "      Research, Department of Clinical Neuroscience, Lausanne University Hospital,\n",
      "      Lausanne, Switzerland.\n",
      "FAU - Dwir, D\n",
      "AU  - Dwir D\n",
      "AD  - Department of Psychiatry, Center for Psychiatric Neuroscience, Centre Hospitalier\n",
      "      Universitaire Vaudois, University of Lausanne, Prilly, Lausanne, Switzerland.\n",
      "FAU - Thieren, L\n",
      "AU  - Thieren L\n",
      "AD  - Laboratory of Cellular and Molecular Neurotherapies, Center for Neuroscience\n",
      "      Research, Department of Clinical Neuroscience, Lausanne University Hospital,\n",
      "      Lausanne, Switzerland.\n",
      "FAU - Sandstrom von Tobel, J\n",
      "AU  - Sandstrom von Tobel J\n",
      "AD  - Department of Physiology, University of Lausanne, Lausanne, Switzerland.\n",
      "FAU - Begka, C\n",
      "AU  - Begka C\n",
      "AD  - Service of Gastroenterology and Hepatology, Department of Medicine, Lausanne\n",
      "      University Hospital, Lausanne, Switzerland.\n",
      "FAU - Legueux, F\n",
      "AU  - Legueux F\n",
      "AD  - FIRALIS SA, Huningue, France.\n",
      "FAU - Velin, D\n",
      "AU  - Velin D\n",
      "AD  - Service of Gastroenterology and Hepatology, Department of Medicine, Lausanne\n",
      "      University Hospital, Lausanne, Switzerland.\n",
      "FAU - Maillard, M H\n",
      "AU  - Maillard MH\n",
      "AD  - Service of Gastroenterology and Hepatology, Department of Medicine, Lausanne\n",
      "      University Hospital, Lausanne, Switzerland.\n",
      "FAU - Do, K Q\n",
      "AU  - Do KQ\n",
      "AD  - Department of Psychiatry, Center for Psychiatric Neuroscience, Centre Hospitalier\n",
      "      Universitaire Vaudois, University of Lausanne, Prilly, Lausanne, Switzerland.\n",
      "FAU - Monnet-Tschudi, F\n",
      "AU  - Monnet-Tschudi F\n",
      "AD  - Department of Physiology, University of Lausanne, Lausanne, Switzerland.\n",
      "FAU - Tenenbaum, L\n",
      "AU  - Tenenbaum L\n",
      "AD  - Laboratory of Cellular and Molecular Neurotherapies, Center for Neuroscience\n",
      "      Research, Department of Clinical Neuroscience, Lausanne University Hospital,\n",
      "      Lausanne, Switzerland.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161207\n",
      "PL  - England\n",
      "TA  - J Neuroinflammation\n",
      "JT  - Journal of neuroinflammation\n",
      "JID - 101222974\n",
      "PMC - PMC5142340\n",
      "OTO - NOTNLM\n",
      "OT  - Adeno-associated viral vector\n",
      "OT  - Cytokine\n",
      "OT  - Lipopolysaccharide\n",
      "OT  - Neuroinflammation\n",
      "OT  - Oxidative stress\n",
      "OT  - Parkinson's disease\n",
      "OT  - Substantia nigra\n",
      "OT  - Toll-like interacting protein\n",
      "OT  - Tollip\n",
      "OT  - iNOS\n",
      "EDAT- 2016/12/09 06:00\n",
      "MHDA- 2016/12/09 06:00\n",
      "CRDT- 2016/12/09 06:00\n",
      "PHST- 2016/08/26 [received]\n",
      "PHST- 2016/11/18 [accepted]\n",
      "AID - 10.1186/s12974-016-0766-5 [doi]\n",
      "AID - 10.1186/s12974-016-0766-5 [pii]\n",
      "PST - epublish\n",
      "SO  - J Neuroinflammation. 2016 Dec 7;13(1):303.\n",
      "\n",
      "PMID- 27924068\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "DA  - 20161207\n",
      "LR  - 20161217\n",
      "IS  - 1674-8301 (Print)\n",
      "IS  - 1674-8301 (Linking)\n",
      "VI  - 30\n",
      "IP  - 6\n",
      "DP  - 2016 Nov\n",
      "TI  - Protein inhibitor of activated STAT 4 (PIAS4) regulates liver fibrosis through\n",
      "      modulating SMAD3 activity.\n",
      "PG  - 496-501\n",
      "LID - 10.7555/JBR.30.20160049 [doi]\n",
      "AB  - Excessive fibrogenesis disrupts normal liver structure, impairs liver function,\n",
      "      and precipitates the development of cirrhosis, an irreversible end-stage liver\n",
      "      disease. A host of factors including nutrition surplus contribute to liver\n",
      "      fibrosis but the underlying mechanism is not fully understood. In the present\n",
      "      study, we investigated the involvement of protein inhibitor for activated stat 4 \n",
      "      (PIAS4) in liver fibrosis in a mouse model of non-alcoholic steatohepatitis\n",
      "      (NASH). We report that PIAS4 silencing using short hairpin RNA (shRNA) attenuated\n",
      "      high-fat high-carbohydrate (HFHC) diet induced liver fibrosis in mice.\n",
      "      Quantitative PCR and Western blotting analyses confirmed that PIAS4 knockdown\n",
      "      downregulated a panel of pro-fibrogenic genes including type I and type III\n",
      "      collagens, smooth muscle actin, and tissue inhibitors of metalloproteinase.\n",
      "      Mechanistically, PIAS4 silencing blocked the recruitment of SMAD3, a potent\n",
      "      pro-fibrogenic transcription factor, to the promoter regions of pro-fibrogenic\n",
      "      genes and dampened SMAD3 acetylation likely by upregulating SIRT1 expression. In \n",
      "      conclusion, PIAS4 may contribute to liver fibrosis by modulating SIRT1-dependent \n",
      "      SMAD3 acetylation.\n",
      "CI  - (c) 2016 by the Journal of Biomedical Research. All rights reserved.\n",
      "FAU - Xu, Huihui\n",
      "AU  - Xu H\n",
      "AD  - Key Laboratory of Cardiovascular Disease and Molecular Intervention, Department\n",
      "      of Pathophysiology, Nanjing Medical University, Nanjing, Jiangsu 211166, China.\n",
      "FAU - Fan, Zhiwen\n",
      "AU  - Fan Z\n",
      "AD  - Key Laboratory of Cardiovascular Disease and Molecular Intervention, Department\n",
      "      of Pathophysiology, Nanjing Medical University, Nanjing, Jiangsu 211166, China.\n",
      "FAU - Tian, Wenfang\n",
      "AU  - Tian W\n",
      "AD  - Key Laboratory of Cardiovascular Disease and Molecular Intervention, Department\n",
      "      of Pathophysiology, Nanjing Medical University, Nanjing, Jiangsu 211166, China.\n",
      "FAU - Xu, Yong\n",
      "AU  - Xu Y\n",
      "AD  - Key Laboratory of Cardiovascular Disease and Molecular Intervention, Department\n",
      "      of Pathophysiology, Nanjing Medical University, Nanjing, Jiangsu 211166, China;\n",
      "      yjxu@njmu.edu.cn.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20160801\n",
      "PL  - China\n",
      "TA  - J Biomed Res\n",
      "JT  - Journal of biomedical research\n",
      "JID - 101551157\n",
      "PMC - PMC5138582\n",
      "OTO - NOTNLM\n",
      "OT  - PIAS4\n",
      "OT  - SMAD3\n",
      "OT  - acetylation\n",
      "OT  - liver fibrosis\n",
      "OT  - transcriptional regulation\n",
      "EDAT- 2016/12/08 06:00\n",
      "MHDA- 2016/12/08 06:00\n",
      "CRDT- 2016/12/08 06:00\n",
      "PHST- 2016/04/09 [received]\n",
      "PHST- 2016/06/29 [revised]\n",
      "PHST- 2016/07/11 [accepted]\n",
      "AID - 10.7555/JBR.30.20160049 [doi]\n",
      "PST - ppublish\n",
      "SO  - J Biomed Res. 2016 Nov;30(6):496-501. doi: 10.7555/JBR.30.20160049. Epub 2016 Aug\n",
      "      1.\n",
      "\n",
      "PMID- 27920257\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161206\n",
      "LR  - 20161207\n",
      "IS  - 1098-5549 (Electronic)\n",
      "IS  - 0270-7306 (Linking)\n",
      "DP  - 2016 Dec 05\n",
      "TI  - PKR mediates inflammatory response induced by hyperosmotic stress.\n",
      "LID - MCB.00521-16 [pii]\n",
      "AB  - High extracellular osmolarity results in a switch from an adaptive to an\n",
      "      inflammatory gene expression program. We show that hyperosmotic stress activates \n",
      "      the protein kinase R (PKR) independently of its RNA binding domain. In turn, PKR \n",
      "      stimulated nuclear accumulation of nuclear factor-kappaB (NF-kappaB) p65 species \n",
      "      phosphorylated at serine-536, which was paralleled by the induction of a subset\n",
      "      of inflammatory NF-kappaB p65- responsive genes including inducible nitric oxide \n",
      "      synthase (iNOS), IL-6, and IL-1beta. The PKR mediated-hyperinduction of iNOS\n",
      "      decreased cell survival in mouse embryonic fibroblasts via mechanisms involving\n",
      "      nitric oxide (NO) synthesis and post-translational modification of proteins.\n",
      "      Moreover, we demonstrate that the PKR inhibitor C16 ameliorates both iNOS\n",
      "      amplification and disease-induced phenotypic breakdown of the intestinal\n",
      "      epithelial barrier caused by an increase in extracellular osmolarity induced by\n",
      "      dextran sodium sulfate (DSS) in vivo Collectively, these findings indicate that\n",
      "      PKR activation is an essential part of the molecular switch from adaptation to\n",
      "      inflammation in response to hyperosmotic stress.\n",
      "CI  - Copyright (c) 2016, American Society for Microbiology. All Rights Reserved.\n",
      "FAU - Farabaugh, Kenneth T\n",
      "AU  - Farabaugh KT\n",
      "AD  - Department of Pharmacology, Case Western Reserve University, Cleveland, OH, USA.\n",
      "FAU - Majumder, Mithu\n",
      "AU  - Majumder M\n",
      "AD  - Center for Regenerative Medicine, Case Western Reserve University, Cleveland, OH,\n",
      "      USA.\n",
      "FAU - Guan, Bo-Jhih\n",
      "AU  - Guan BJ\n",
      "AD  - Department of Genetics, Case Western Reserve University, Cleveland, OH, USA.\n",
      "FAU - Jobava, Raul\n",
      "AU  - Jobava R\n",
      "AD  - Department of Biochemistry, Case Western Reserve University, Cleveland, OH, USA.\n",
      "FAU - Wu, Jing\n",
      "AU  - Wu J\n",
      "AD  - Department of Genetics, Case Western Reserve University, Cleveland, OH, USA.\n",
      "FAU - Krokowski, Dawid\n",
      "AU  - Krokowski D\n",
      "AD  - Department of Genetics, Case Western Reserve University, Cleveland, OH, USA.\n",
      "FAU - Gao, Xing-Huang\n",
      "AU  - Gao XH\n",
      "AD  - Department of Genetics, Case Western Reserve University, Cleveland, OH, USA.\n",
      "FAU - Schuster, Andrew\n",
      "AU  - Schuster A\n",
      "AD  - Department of Cellular and Molecular Medicine, Cleveland Clinic, Cleveland, OH,\n",
      "      USA.\n",
      "FAU - Longworth, Michelle\n",
      "AU  - Longworth M\n",
      "AD  - Department of Cellular and Molecular Medicine, Cleveland Clinic, Cleveland, OH,\n",
      "      USA.\n",
      "FAU - Chan, Edward D\n",
      "AU  - Chan ED\n",
      "AD  - Department of Medicine, University of Colorado Anschutz Medical Campus, Aurora,\n",
      "      CO, USA.\n",
      "FAU - Bianchi, Massimiliano\n",
      "AU  - Bianchi M\n",
      "AD  - Department of Clinical and Experimental Medicine, Universita degli Studi di\n",
      "      Parma, Parma PR, Italy.\n",
      "FAU - Dey, Madhusudan\n",
      "AU  - Dey M\n",
      "AD  - Department of Biological Sciences, University of Wisconsin-Milwaukee, Milwaukee, \n",
      "      WI, USA.\n",
      "FAU - Koromilas, Antonis E\n",
      "AU  - Koromilas AE\n",
      "AD  - Department of Microbiology and Immunology, McGill University, Montreal QC,\n",
      "      Canada.\n",
      "FAU - Ramakrishnan, Parameswaran\n",
      "AU  - Ramakrishnan P\n",
      "AD  - Department of Pathology, Case Western Reserve University, Cleveland, OH\n",
      "      mxh8@case.edu pxr150@case.edu.\n",
      "FAU - Hatzoglou, Maria\n",
      "AU  - Hatzoglou M\n",
      "AD  - Department of Genetics, Case Western Reserve University, Cleveland, OH, USA\n",
      "      mxh8@case.edu pxr150@case.edu.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161205\n",
      "PL  - United States\n",
      "TA  - Mol Cell Biol\n",
      "JT  - Molecular and cellular biology\n",
      "JID - 8109087\n",
      "EDAT- 2016/12/07 06:00\n",
      "MHDA- 2016/12/07 06:00\n",
      "CRDT- 2016/12/07 06:00\n",
      "AID - MCB.00521-16 [pii]\n",
      "AID - 10.1128/MCB.00521-16 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Mol Cell Biol. 2016 Dec 5. pii: MCB.00521-16.\n",
      "\n",
      "PMID- 27920150\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161206\n",
      "LR  - 20161207\n",
      "IS  - 1530-6860 (Electronic)\n",
      "IS  - 0892-6638 (Linking)\n",
      "DP  - 2016 Dec 05\n",
      "TI  - Heterotrimeric kinesin-2, together with kinesin-1, steers vesicular\n",
      "      acetylcholinesterase movements toward the synapse.\n",
      "LID - fj.201600759RRR [pii]\n",
      "AB  - Acetylcholinesterase (AChE), which is implicated in the pathophysiology of\n",
      "      neurological disorders, is distributed along the axon and enriched at the\n",
      "      presynaptic basal lamina. It hydrolyses the neurotransmitter acetylcholine, which\n",
      "      inhibits synaptic transmission. Aberrant AChE activity and ectopic axonal\n",
      "      accumulation of the enzyme are associated with neurodegenerative disorders, such \n",
      "      as Alzheimer's disease. The molecular mechanism that underlies AChE transport is \n",
      "      still unclear. Here, we show that expression of Drosophila AChE tagged with\n",
      "      photoactivable green fluorescent protein and m-Cherry (GPAC) in cholinergic\n",
      "      neurons compensates for the RNA interference-mediated knockdown of endogenous\n",
      "      AChE activity. GPAC-AChE, which is enriched in the neuropil region of the brain, \n",
      "      moves in the apparently vesicular form in axons with an anterograde bias in\n",
      "      Drosophila larvae. Two anterograde motors, kinesin-1 and -2, propel distinct\n",
      "      aspects of GPAC-AChE movements. Total loss of kinesin-2 reduces the density of\n",
      "      anterograde traffic and increases bidirectional movements of GPAC-AChE vesicles\n",
      "      without altering their speed. A partial loss of kinesin-1 reduces both the\n",
      "      density and speed of anterograde GPAC-AChE traffic and enhances the pool of\n",
      "      stationary vesicles. Together, these results suggest that combining activity of a\n",
      "      relatively weak kinesin-2 with that of a stronger kinesin-1 motor could steer\n",
      "      AChE-containing vesicles toward synapse, and provides a molecular basis for the\n",
      "      observed subcellular distribution of the enzyme.-Kulkarni, A., Khan, Y., Ray, K. \n",
      "      Heterotrimeric kinesin-2, together with kinesin-1, steers vesicular\n",
      "      acetylcholinesterase movements toward the synapse.\n",
      "CI  - (c) FASEB.\n",
      "FAU - Kulkarni, Anuttama\n",
      "AU  - Kulkarni A\n",
      "AD  - Sophia College, Mumbai, India.\n",
      "AD  - Tata Institute of Fundamental Research, Mumbai, India.\n",
      "FAU - Khan, Yasmin\n",
      "AU  - Khan Y\n",
      "AD  - Sophia College, Mumbai, India.\n",
      "FAU - Ray, Krishanu\n",
      "AU  - Ray K\n",
      "AD  - Tata Institute of Fundamental Research, Mumbai, India krishanu@tifr.res.in.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161205\n",
      "PL  - United States\n",
      "TA  - FASEB J\n",
      "JT  - FASEB journal : official publication of the Federation of American Societies for \n",
      "      Experimental Biology\n",
      "JID - 8804484\n",
      "OTO - NOTNLM\n",
      "OT  - Drosophila\n",
      "OT  - intracellular transport\n",
      "OT  - lateral-chordotonal-neuron\n",
      "OT  - mouse brain\n",
      "EDAT- 2016/12/07 06:00\n",
      "MHDA- 2016/12/07 06:00\n",
      "CRDT- 2016/12/07 06:00\n",
      "PHST- 2016/07/07 [received]\n",
      "PHST- 2016/11/14 [accepted]\n",
      "AID - fj.201600759RRR [pii]\n",
      "AID - 10.1096/fj.201600759RRR [doi]\n",
      "PST - aheadofprint\n",
      "SO  - FASEB J. 2016 Dec 5. pii: fj.201600759RRR.\n",
      "\n",
      "PMID- 27919828\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161206\n",
      "LR  - 20161212\n",
      "IS  - 1873-6300 (Electronic)\n",
      "IS  - 0891-0618 (Linking)\n",
      "VI  - 80\n",
      "DP  - 2016 Dec 02\n",
      "TI  - Mucuna pruriens reduces inducible nitric oxide synthase expression in\n",
      "      Parkinsonian mice model.\n",
      "PG  - 1-10\n",
      "LID - S0891-0618(16)30133-8 [pii]\n",
      "LID - 10.1016/j.jchemneu.2016.11.009 [doi]\n",
      "AB  - Parkinson's disease is one of the most common neurodegenerative disease found in \n",
      "      aged peoples. Plentiful studies are being conducted to find a suitable and\n",
      "      effective cure for this disease giving special impetus on use of herbal plants.\n",
      "      The study aimed at investigating the effect of ethanolic extract of Mucuna\n",
      "      pruriens (Mp) on level of nitric oxide (NO) in paraquat (PQ) induced Parkinson's \n",
      "      disease (PD) mouse model and its subsequent contribution to lipid peroxidation.\n",
      "      Twenty four Swiss albino mice were divided into three groups; Control, PQ and\n",
      "      PQ+Mp. PQ doses were given intraperitoneally, twice in a week and oral dose of\n",
      "      ethanolic extract of Mp seed was given for 9 weeks. Nitrite content and lipid\n",
      "      peroxidation was measured in all treated groups along with respective controls.\n",
      "      RNA was isolated from the nigrostriatal tissue of control and the treated mice\n",
      "      and was reverse transcribed into cDNA. PCR was performed to amplify iNOS mRNA and\n",
      "      western blot analysis was performed to check its protein level. We had also\n",
      "      perfused the mice in all treated group and performed Tyrosine hydroxylase (TH)\n",
      "      and iNOS immunoreactivity in substantia nigra region of mice brain. PQ-treatment \n",
      "      increased nitrite content, expression of iNOS and lipid peroxidation compared to \n",
      "      respective controls. Mp treatment resulted in a significant attenuation of iNOS\n",
      "      expression, nitrite content and lipid peroxidation demonstrating that it reduces \n",
      "      nitric oxide in PQ-induced Parkinson's disease. Interestingly; we also observed\n",
      "      that mRNA, protein expression and immunoreactivity of iNOS was significantly\n",
      "      decreased after Mp treatment and TH immunoreactivity was significantly improved\n",
      "      after the treatment of Mp. Our results demonstrated that Mp protects the\n",
      "      dopaminergic neurons from the NO injury in substantia nigra.\n",
      "CI  - Copyright (c) 2016 Elsevier B.V. All rights reserved.\n",
      "FAU - Yadav, Satyndra Kumar\n",
      "AU  - Yadav SK\n",
      "AD  - Department of Biochemistry, Institute of Science, Banaras Hindu University,\n",
      "      Varanasi 221005, India. Electronic address: satyndra_yadav@yahoo.co.in.\n",
      "FAU - Rai, Sachchida Nand\n",
      "AU  - Rai SN\n",
      "AD  - Department of Biochemistry, Institute of Science, Banaras Hindu University,\n",
      "      Varanasi 221005, India. Electronic address: raibiochem@gmail.com.\n",
      "FAU - Singh, Surya Pratap\n",
      "AU  - Singh SP\n",
      "AD  - Department of Biochemistry, Institute of Science, Banaras Hindu University,\n",
      "      Varanasi 221005, India. Electronic address: suryasinghbhu16@gmail.com.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161202\n",
      "PL  - Netherlands\n",
      "TA  - J Chem Neuroanat\n",
      "JT  - Journal of chemical neuroanatomy\n",
      "JID - 8902615\n",
      "OTO - NOTNLM\n",
      "OT  - Dopaminergic neurons\n",
      "OT  - Immunoreactivity\n",
      "OT  - Intraperitoneally\n",
      "OT  - Lipid peroxidation\n",
      "OT  - Nitric oxide\n",
      "EDAT- 2016/12/07 06:00\n",
      "MHDA- 2016/12/07 06:00\n",
      "CRDT- 2016/12/07 06:00\n",
      "PHST- 2016/07/12 [received]\n",
      "PHST- 2016/10/29 [revised]\n",
      "PHST- 2016/11/30 [accepted]\n",
      "AID - S0891-0618(16)30133-8 [pii]\n",
      "AID - 10.1016/j.jchemneu.2016.11.009 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - J Chem Neuroanat. 2016 Dec 2;80:1-10. doi: 10.1016/j.jchemneu.2016.11.009.\n",
      "\n",
      "PMID- 27918553\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "DA  - 20161205\n",
      "LR  - 20161224\n",
      "IS  - 2157-9024 (Linking)\n",
      "VI  - 5\n",
      "IP  - 12\n",
      "DP  - 2016 Dec 05\n",
      "TI  - Jagged 1 is a major Notch ligand along cholangiocarcinoma development in mice and\n",
      "      humans.\n",
      "PG  - e274\n",
      "LID - 10.1038/oncsis.2016.73 [doi]\n",
      "AB  - Intrahepatic cholangiocarcinoma (ICC) is a rare yet deadly malignancy with\n",
      "      limited treatment options. Activation of the Notch signalling cascade has been\n",
      "      implicated in cholangiocarcinogenesis. However, while several studies focused on \n",
      "      the Notch receptors required for ICC development, little is known about the\n",
      "      upstream inducers responsible for their activation. Here, we show that the Jagged\n",
      "      1 (Jag1) ligand is almost ubiquitously upregulated in human ICC samples when\n",
      "      compared with corresponding non-tumorous counterparts. Furthermore, we found that\n",
      "      while overexpression of Jag1 alone does not lead to liver tumour development,\n",
      "      overexpression of Jag1 synergizes with activated AKT signalling to promote liver \n",
      "      carcinogenesis in AKT/Jag1 mice. Histologically, tumours consisted exclusively of\n",
      "      ICC, with hepatocellular tumours not occurring in AKT/Jag1 mice. Furthermore,\n",
      "      tumours from AKT/Jag1 mice exhibited extensive desmoplastic reaction, an\n",
      "      important feature of human ICC. At the molecular level, we found that both\n",
      "      AKT/mTOR and Notch cascades are activated in AKT/Jag1 ICC tissues, and that the\n",
      "      Notch signalling is necessary for ICC development in AKT/Jag1 mice. In human ICC \n",
      "      cell lines, silencing of Jag1 via specific small interfering RNA reduces\n",
      "      proliferation and increases apoptosis. Finally, combined inhibition of AKT and\n",
      "      Notch pathways is highly detrimental for the in vitro growth of ICC cell lines.\n",
      "      In summary, our study demonstrates that Jag1 is an important upstream inducer of \n",
      "      the Notch signalling in human and mouse ICC. Targeting Jag1 might represent a\n",
      "      novel therapeutic strategy for the treatment of this deadly disease.\n",
      "FAU - Che, L\n",
      "AU  - Che L\n",
      "AD  - Key Laboratory of Carcinogenesis and Translational Research (Ministry of\n",
      "      Education), Peking University Cancer Hospital & Institute, Beijing, China.\n",
      "AD  - Department of Bioengineering and Therapeutic Sciences and Liver Center,\n",
      "      University of California, San Francisco, CA, USA.\n",
      "FAU - Fan, B\n",
      "AU  - Fan B\n",
      "AD  - Key Laboratory of Carcinogenesis and Translational Research (Ministry of\n",
      "      Education), Peking University Cancer Hospital & Institute, Beijing, China.\n",
      "AD  - Department of Bioengineering and Therapeutic Sciences and Liver Center,\n",
      "      University of California, San Francisco, CA, USA.\n",
      "FAU - Pilo, M G\n",
      "AU  - Pilo MG\n",
      "AD  - Institute of Pathology, University of Greifswald, Greifswald, Germany.\n",
      "FAU - Xu, Z\n",
      "AU  - Xu Z\n",
      "AD  - Department of Bioengineering and Therapeutic Sciences and Liver Center,\n",
      "      University of California, San Francisco, CA, USA.\n",
      "AD  - Department of Gastroenterology, Guizhou Provincial People's Hospital, The\n",
      "      Affiliated People's Hospital of Guizhou Medical University, Guiyang, China.\n",
      "FAU - Liu, Y\n",
      "AU  - Liu Y\n",
      "AD  - Department of Gastroenterology, 307 Hospital of PLA, Beijing, China.\n",
      "FAU - Cigliano, A\n",
      "AU  - Cigliano A\n",
      "AD  - Institute of Pathology, University of Greifswald, Greifswald, Germany.\n",
      "FAU - Cossu, A\n",
      "AU  - Cossu A\n",
      "AD  - Unit of Pathology, Azienda Ospedaliero Universitaria Sassari, Sassari, Italy.\n",
      "FAU - Palmieri, G\n",
      "AU  - Palmieri G\n",
      "AD  - Institute of Biomolecular Chemistry, National Research Council, Sassari, Italy.\n",
      "FAU - Pascale, R M\n",
      "AU  - Pascale RM\n",
      "AD  - Department of Clinical and Experimental Medicine, University of Sassari, Sassari,\n",
      "      Italy.\n",
      "FAU - Porcu, A\n",
      "AU  - Porcu A\n",
      "AD  - Department of Clinical and Experimental Medicine, University of Sassari, Sassari,\n",
      "      Italy.\n",
      "FAU - Vidili, G\n",
      "AU  - Vidili G\n",
      "AD  - Department of Clinical and Experimental Medicine, University of Sassari, Sassari,\n",
      "      Italy.\n",
      "FAU - Serra, M\n",
      "AU  - Serra M\n",
      "AD  - Institute of Pathology, University of Greifswald, Greifswald, Germany.\n",
      "FAU - Dombrowski, F\n",
      "AU  - Dombrowski F\n",
      "AD  - Institute of Pathology, University of Greifswald, Greifswald, Germany.\n",
      "FAU - Ribback, S\n",
      "AU  - Ribback S\n",
      "AD  - Institute of Pathology, University of Greifswald, Greifswald, Germany.\n",
      "FAU - Calvisi, D F\n",
      "AU  - Calvisi DF\n",
      "AD  - Institute of Pathology, University of Greifswald, Greifswald, Germany.\n",
      "AD  - Department of Clinical and Experimental Medicine, University of Sassari, Sassari,\n",
      "      Italy.\n",
      "FAU - Chen, X\n",
      "AU  - Chen X\n",
      "AD  - Department of Bioengineering and Therapeutic Sciences and Liver Center,\n",
      "      University of California, San Francisco, CA, USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161205\n",
      "PL  - United States\n",
      "TA  - Oncogenesis\n",
      "JT  - Oncogenesis\n",
      "JID - 101580004\n",
      "PMC - PMC5177771\n",
      "EDAT- 2016/12/06 06:00\n",
      "MHDA- 2016/12/06 06:00\n",
      "CRDT- 2016/12/06 06:00\n",
      "PHST- 2016/07/25 [received]\n",
      "PHST- 2016/09/26 [revised]\n",
      "PHST- 2016/10/10 [accepted]\n",
      "AID - oncsis201673 [pii]\n",
      "AID - 10.1038/oncsis.2016.73 [doi]\n",
      "PST - epublish\n",
      "SO  - Oncogenesis. 2016 Dec 5;5(12):e274. doi: 10.1038/oncsis.2016.73.\n",
      "\n",
      "PMID- 27918527\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "DA  - 20161205\n",
      "LR  - 20161206\n",
      "IS  - 2058-5276 (Electronic)\n",
      "IS  - 2058-5276 (Linking)\n",
      "VI  - 2\n",
      "DP  - 2016 Dec 05\n",
      "TI  - Induction and suppression of antiviral RNA interference by influenza A virus in\n",
      "      mammalian cells.\n",
      "PG  - 16250\n",
      "LID - 10.1038/nmicrobiol.2016.250 [doi]\n",
      "AB  - Influenza A virus (IAV) causes annual epidemics and occasional pandemics, and is \n",
      "      one of the best-characterized human RNA viral pathogens1. However, a\n",
      "      physiologically relevant role for the RNA interference (RNAi) suppressor activity\n",
      "      of the IAV non-structural protein 1 (NS1), reported over a decade ago2, remains\n",
      "      unknown3. Plant and insect viruses have evolved diverse virulence proteins to\n",
      "      suppress RNAi as their hosts produce virus-derived small interfering RNAs\n",
      "      (siRNAs) that direct specific antiviral defence4-7 by an RNAi mechanism dependent\n",
      "      on the slicing activity of Argonaute proteins (AGOs)8,9. Recent studies have\n",
      "      documented induction and suppression of antiviral RNAi in mouse embryonic stem\n",
      "      cells and suckling mice10,11. However, it is still under debate whether infection\n",
      "      by IAV or any other RNA virus that infects humans induces and/or suppresses\n",
      "      antiviral RNAi in mature mammalian somatic cells12-21. Here, we demonstrate that \n",
      "      mature human somatic cells produce abundant virus-derived siRNAs\n",
      "      co-immunoprecipitated with AGOs in response to IAV infection. We show that the\n",
      "      biogenesis of viral siRNAs from IAV double-stranded RNA (dsRNA) precursors in\n",
      "      infected cells is mediated by wild-type human Dicer and potently suppressed by\n",
      "      both NS1 of IAV as well as virion protein 35 (VP35) of Ebola and Marburg\n",
      "      filoviruses. We further demonstrate that the slicing catalytic activity of AGO2\n",
      "      inhibits IAV and other RNA viruses in mature mammalian cells, in an\n",
      "      interferon-independent fashion. Altogether, our work shows that IAV infection\n",
      "      induces and suppresses antiviral RNAi in differentiated mammalian somatic cells.\n",
      "FAU - Li, Yang\n",
      "AU  - Li Y\n",
      "AD  - Department of Plant Pathology &Microbiology, and Institute for Integrative Genome\n",
      "      Biology, University of California, Riverside, California 92521, USA.\n",
      "AD  - State Key Laboratory of Genetic Engineering, Collaborative Innovation Centre of\n",
      "      Genetics and Development, School of Life Sciences, Fudan University, Shanghai\n",
      "      200438, China.\n",
      "FAU - Basavappa, Megha\n",
      "AU  - Basavappa M\n",
      "AD  - Gastrointestinal Unit and Center for the Study of Inflammatory Bowel Disease,\n",
      "      Massachusetts General Hospital, Harvard Medical School, Boston, Massachusetts\n",
      "      02114, USA.\n",
      "FAU - Lu, Jinfeng\n",
      "AU  - Lu J\n",
      "AD  - Department of Plant Pathology &Microbiology, and Institute for Integrative Genome\n",
      "      Biology, University of California, Riverside, California 92521, USA.\n",
      "AD  - Graduate Program in Genetics, Genomics, and Bioinformatics, University of\n",
      "      California, Riverside, California 92521, USA.\n",
      "FAU - Dong, Shuwei\n",
      "AU  - Dong S\n",
      "AD  - Department of Plant Pathology &Microbiology, and Institute for Integrative Genome\n",
      "      Biology, University of California, Riverside, California 92521, USA.\n",
      "FAU - Cronkite, D Alexander\n",
      "AU  - Cronkite DA\n",
      "AD  - Gastrointestinal Unit and Center for the Study of Inflammatory Bowel Disease,\n",
      "      Massachusetts General Hospital, Harvard Medical School, Boston, Massachusetts\n",
      "      02114, USA.\n",
      "FAU - Prior, John T\n",
      "AU  - Prior JT\n",
      "AD  - Gastrointestinal Unit and Center for the Study of Inflammatory Bowel Disease,\n",
      "      Massachusetts General Hospital, Harvard Medical School, Boston, Massachusetts\n",
      "      02114, USA.\n",
      "FAU - Reinecker, Hans-Christian\n",
      "AU  - Reinecker HC\n",
      "AD  - Gastrointestinal Unit and Center for the Study of Inflammatory Bowel Disease,\n",
      "      Massachusetts General Hospital, Harvard Medical School, Boston, Massachusetts\n",
      "      02114, USA.\n",
      "FAU - Hertzog, Paul\n",
      "AU  - Hertzog P\n",
      "AD  - Centre for Innate Immunity and Infectious Diseases, Hudson Institute of Medical\n",
      "      Research, 27-31 Wright Street, Clayton, Victoria 3168, Australia.\n",
      "FAU - Han, Yanhong\n",
      "AU  - Han Y\n",
      "AD  - Department of Plant Pathology &Microbiology, and Institute for Integrative Genome\n",
      "      Biology, University of California, Riverside, California 92521, USA.\n",
      "FAU - Li, Wan-Xiang\n",
      "AU  - Li WX\n",
      "AD  - Department of Plant Pathology &Microbiology, and Institute for Integrative Genome\n",
      "      Biology, University of California, Riverside, California 92521, USA.\n",
      "FAU - Cheloufi, Sihem\n",
      "AU  - Cheloufi S\n",
      "AD  - Massachusetts General Hospital, Cancer Center and Center for Regenerative\n",
      "      Medicine and Harvard Stem Cell Institute, 185 Cambridge Street, Boston,\n",
      "      Massachusetts 02114, USA.\n",
      "FAU - Karginov, Fedor V\n",
      "AU  - Karginov FV\n",
      "AD  - Department of Cell Biology and Neuroscience, University of California, Riverside,\n",
      "      California 92521, USA.\n",
      "FAU - Ding, Shou-Wei\n",
      "AU  - Ding SW\n",
      "AD  - Department of Plant Pathology &Microbiology, and Institute for Integrative Genome\n",
      "      Biology, University of California, Riverside, California 92521, USA.\n",
      "AD  - Graduate Program in Genetics, Genomics, and Bioinformatics, University of\n",
      "      California, Riverside, California 92521, USA.\n",
      "FAU - Jeffrey, Kate L\n",
      "AU  - Jeffrey KL\n",
      "AD  - Gastrointestinal Unit and Center for the Study of Inflammatory Bowel Disease,\n",
      "      Massachusetts General Hospital, Harvard Medical School, Boston, Massachusetts\n",
      "      02114, USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161205\n",
      "PL  - England\n",
      "TA  - Nat Microbiol\n",
      "JT  - Nature microbiology\n",
      "JID - 101674869\n",
      "EDAT- 2016/12/06 06:00\n",
      "MHDA- 2016/12/06 06:00\n",
      "CRDT- 2016/12/06 06:00\n",
      "PHST- 2016/10/28 [received]\n",
      "PHST- 2016/11/08 [accepted]\n",
      "AID - nmicrobiol2016250 [pii]\n",
      "AID - 10.1038/nmicrobiol.2016.250 [doi]\n",
      "PST - epublish\n",
      "SO  - Nat Microbiol. 2016 Dec 5;2:16250. doi: 10.1038/nmicrobiol.2016.250.\n",
      "\n",
      "PMID- 27913791\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161203\n",
      "LR  - 20161204\n",
      "IS  - 1753-4267 (Electronic)\n",
      "IS  - 1753-4259 (Linking)\n",
      "DP  - 2016 Dec 01\n",
      "TI  - Circadian transcription factor BMAL1 regulates innate immunity against select RNA\n",
      "      viruses.\n",
      "LID - 1753425916681075 [pii]\n",
      "AB  - BMAL1 (brain and muscle ARNT-like protein 1, also known as MOP3 or ARNT3) belongs\n",
      "      to the family of the basic helix-loop-helix (bHLH)-PAS domain-containing\n",
      "      transcription factors, and is a key component of the molecular oscillator that\n",
      "      generates circadian rhythms. Here, we report that BMAL1-deficient cells are\n",
      "      significantly more susceptible to infection by two major respiratory viruses of\n",
      "      the Paramyxoviridae family, namely RSV and PIV3. Embryonic fibroblasts from\n",
      "      Bmal1-/- mice produced nearly 10-fold more progeny virus than their wild type\n",
      "      controls. These results were supported by animal studies whereby pulmonary\n",
      "      infection of RSV produced a more severe disease and morbidity in Bmal1-/-mice.\n",
      "      These results show that BMAL1 can regulate cellular innate immunity against\n",
      "      specific RNA viruses.\n",
      "CI  - (c) The Author(s) 2016.\n",
      "FAU - Majumdar, Tanmay\n",
      "AU  - Majumdar T\n",
      "AD  - Department of Biological, Geological and Environmental Sciences, and Center for\n",
      "      Gene Regulation in Health and Disease, Cleveland State University, Cleveland, OH,\n",
      "      USA.\n",
      "FAU - Dhar, Jayeeta\n",
      "AU  - Dhar J\n",
      "AD  - Department of Biological, Geological and Environmental Sciences, and Center for\n",
      "      Gene Regulation in Health and Disease, Cleveland State University, Cleveland, OH,\n",
      "      USA.\n",
      "FAU - Patel, Sonal\n",
      "AU  - Patel S\n",
      "AD  - Department of Biological, Geological and Environmental Sciences, and Center for\n",
      "      Gene Regulation in Health and Disease, Cleveland State University, Cleveland, OH,\n",
      "      USA.\n",
      "FAU - Kondratov, Roman\n",
      "AU  - Kondratov R\n",
      "AD  - Department of Biological, Geological and Environmental Sciences, and Center for\n",
      "      Gene Regulation in Health and Disease, Cleveland State University, Cleveland, OH,\n",
      "      USA.\n",
      "FAU - Barik, Sailen\n",
      "AU  - Barik S\n",
      "AD  - Department of Biological, Geological and Environmental Sciences, and Center for\n",
      "      Gene Regulation in Health and Disease, Cleveland State University, Cleveland, OH,\n",
      "      USA s.barik@csuohio.edu.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161201\n",
      "PL  - United States\n",
      "TA  - Innate Immun\n",
      "JT  - Innate immunity\n",
      "JID - 101469670\n",
      "OTO - NOTNLM\n",
      "OT  - BMAL1\n",
      "OT  - Paramyxovirus\n",
      "OT  - Toxoplasma gondii\n",
      "OT  - circadian clock\n",
      "OT  - innate immunity\n",
      "OT  - parainfluenza virus\n",
      "OT  - pneumovirus\n",
      "OT  - respiratory syncytial virus\n",
      "EDAT- 2016/12/04 06:00\n",
      "MHDA- 2016/12/04 06:00\n",
      "CRDT- 2016/12/04 06:00\n",
      "PHST- 2016/05/30 [received]\n",
      "PHST- 2016/09/11 [revised]\n",
      "PHST- 2016/10/13 [accepted]\n",
      "AID - 1753425916681075 [pii]\n",
      "AID - 10.1177/1753425916681075 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Innate Immun. 2016 Dec 1. pii: 1753425916681075.\n",
      "\n",
      "PMID- 27913212\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161203\n",
      "LR  - 20161218\n",
      "IS  - 0006-3002 (Print)\n",
      "IS  - 0006-3002 (Linking)\n",
      "VI  - 1863\n",
      "IP  - 2\n",
      "DP  - 2016 Nov 29\n",
      "TI  - VCP cooperates with UBXD1 to degrade mitochondrial outer membrane protein MCL1 in\n",
      "      model of Huntington's disease.\n",
      "PG  - 552-559\n",
      "LID - S0925-4439(16)30320-9 [pii]\n",
      "LID - 10.1016/j.bbadis.2016.11.026 [doi]\n",
      "AB  - Proteasome-dependent turnover of mitochondrial outer membrane (OMM)-associated\n",
      "      proteins is one of the mechanisms for maintaining proper mitochondrial quality\n",
      "      and function. However, the underlying pathways and their implications in human\n",
      "      disease are poorly understood. Huntington's disease (HD) is a fatal, inherited\n",
      "      neurodegenerative disorder caused by expanded CAG repeats in the N terminal of\n",
      "      the huntingtin gene (mutant Huntingtin, mtHtt). In this study, we show an\n",
      "      extensive degradation of the OMM protein MCL1 (Myeloid cell leukemia sequence 1) \n",
      "      in both HD mouse striatal cells and HD patient fibroblasts. The decrease in MCL1 \n",
      "      level is associated with mitochondrial and cellular damage.\n",
      "      Valosin-containing-protein (VCP) is an AAA-ATPase central to protein turnover via\n",
      "      the ubiquitin proteasome system (UPS). We found that VCP translocates to\n",
      "      mitochondria and promotes MCL1 degradation in HD cell cultures. Either\n",
      "      down-regulation of VCP by RNA interference or inhibition of VCP by a dominant\n",
      "      negative mutant abolishes MCL1 degradation in HD cell cultures. We further show\n",
      "      that UBX-domain containing protein 1 (UBXD1), a known co-factor of VCP assisting \n",
      "      in the recognition of substrates for protein degradation, selectively binds to\n",
      "      MCL1 and interacts with VCP to mediate MCL1 extraction from the mitochondria.\n",
      "      These results indicate that the OMM protein MCL1 is degraded by the VCP-UBXD1\n",
      "      complex and that the process is promoted by the presence of mtHtt. Therefore, our\n",
      "      finding provides a new insight into the mechanism of mitochondrial dysfunction in\n",
      "      HD.\n",
      "CI  - Copyright (c) 2016 Elsevier B.V. All rights reserved.\n",
      "FAU - Guo, Xing\n",
      "AU  - Guo X\n",
      "AD  - Department of Physiology & Biophysics, Case Western Reserve University School of \n",
      "      Medicine, Cleveland, OH 44106, USA.\n",
      "FAU - Qi, Xin\n",
      "AU  - Qi X\n",
      "AD  - Department of Physiology & Biophysics, Case Western Reserve University School of \n",
      "      Medicine, Cleveland, OH 44106, USA; Center for Mitochondrial Disease, Case\n",
      "      Western Reserve University School of Medicine, Cleveland, OH 44106, USA.\n",
      "      Electronic address: xxq38@case.edu.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161129\n",
      "PL  - Netherlands\n",
      "TA  - Biochim Biophys Acta\n",
      "JT  - Biochimica et biophysica acta\n",
      "JID - 0217513\n",
      "OTO - NOTNLM\n",
      "OT  - Huntington's disease\n",
      "OT  - Mitochondria\n",
      "OT  - Protein degradation\n",
      "OT  - Valosin-containing protein\n",
      "EDAT- 2016/12/04 06:00\n",
      "MHDA- 2016/12/04 06:00\n",
      "CRDT- 2016/12/04 06:00\n",
      "PHST- 2016/08/28 [received]\n",
      "PHST- 2016/11/07 [revised]\n",
      "PHST- 2016/11/28 [accepted]\n",
      "AID - S0925-4439(16)30320-9 [pii]\n",
      "AID - 10.1016/j.bbadis.2016.11.026 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Biochim Biophys Acta. 2016 Nov 29;1863(2):552-559. doi:\n",
      "      10.1016/j.bbadis.2016.11.026.\n",
      "\n",
      "PMID- 27911317\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161202\n",
      "LR  - 20161203\n",
      "IS  - 1875-8908 (Electronic)\n",
      "IS  - 1387-2877 (Linking)\n",
      "DP  - 2016 Nov 28\n",
      "TI  - Alzheimer's Disease Histological and Behavioral Manifestations in Transgenic Mice\n",
      "      Correlate with Specific Gut Microbiome State.\n",
      "AB  - Alzheimer's disease (AD) is a neurodegenerative brain disease and is the most\n",
      "      common form of dementia. In recent years, many studies indicated the association \n",
      "      of gut microbiota changes with metabolic diseases. However, the gut microbiota of\n",
      "      AD has not been investigated. The present study aims to compare the gut\n",
      "      microbiota in APP/PS1 transgenic mice of AD and C57/Bl6 wild-type (WT) mice by\n",
      "      pyrosequencing the V3 and V4 regions of the bacterial 16S ribosomal RNA genes.\n",
      "      The 3-, 6-, and 8-month-old APP/PS1 and WT mice were used to explore the effects \n",
      "      of age on the gut microbiota. First, the results indicated that impaired spatial \n",
      "      learning and memory appeared in 6-month-old APP/PS1 mice and was further\n",
      "      aggravated in the 8-month-old group, which was consistent with\n",
      "      immunohistochemical studies of amyloid plaque. Second, AD histological and\n",
      "      behavioral manifestations in the APP/PS1 mice were found to be correlated with a \n",
      "      specific gut microbiome state. Third, the microbiota diversity of APP/PS1 mice\n",
      "      decreased with increased age. Fourth, further inspection showed that the\n",
      "      abundance of Helicobacteraceae and Desulfovibrionaceae at the family level and\n",
      "      Odoribacter and Helicobacter at the genus level increased significantly in\n",
      "      APP/PS1 mice than in WT mice, while Prevotella abundance in WT mice was\n",
      "      significantly higher than in APP/PS1 mice. More human studies are warranted to\n",
      "      explore the potential of gut microbiota as diagnostic biomarkers or therapeutic\n",
      "      target for AD.\n",
      "FAU - Shen, Liang\n",
      "AU  - Shen L\n",
      "FAU - Liu, Lu\n",
      "AU  - Liu L\n",
      "FAU - Ji, Hong-Fang\n",
      "AU  - Ji HF\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161128\n",
      "PL  - Netherlands\n",
      "TA  - J Alzheimers Dis\n",
      "JT  - Journal of Alzheimer's disease : JAD\n",
      "JID - 9814863\n",
      "OTO - NOTNLM\n",
      "OT  - 16S ribosomal RNA\n",
      "OT  - APP/PS1 mice\n",
      "OT  - Alzheimer's disease\n",
      "OT  - gut microbiota\n",
      "EDAT- 2016/12/03 06:00\n",
      "MHDA- 2016/12/03 06:00\n",
      "CRDT- 2016/12/03 06:00\n",
      "AID - JAD160884 [pii]\n",
      "AID - 10.3233/JAD-160884 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - J Alzheimers Dis. 2016 Nov 28.\n",
      "\n",
      "PMID- 27910163\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161202\n",
      "LR  - 20161203\n",
      "IS  - 1365-2184 (Electronic)\n",
      "IS  - 0960-7722 (Linking)\n",
      "DP  - 2016 Dec 02\n",
      "TI  - Serum amyloid A, an acute phase protein, stimulates proliferative and\n",
      "      proinflammatory responses of keratinocytes.\n",
      "LID - 10.1111/cpr.12320 [doi]\n",
      "AB  - OBJECTIVES: Serum amyloid A (SAA), an acute phase protein, is highly expressed in\n",
      "      psoriatic lesions but its function is not fully understood. The aim of this study\n",
      "      was to explore its role in activation of keratinocytes. MATERIALS AND METHODS:\n",
      "      Real-time PCR and immunofluorescence were performed to examine SAA expression in \n",
      "      imiquimod (IMQ)-induced psoriasis-like mice. In vivo function of SAA was examined\n",
      "      by treating psoriasis-like mice with SAA neutralising antibody. Cell viability\n",
      "      was monitored using the CCK-8 assay. Real-time PCR was performed to determine\n",
      "      expression of genes associated with differentiation and inflammation. Ki67+\n",
      "      percentage and immunological markers were analysed by flow cytometry. Involvement\n",
      "      of formyl peptide receptor-like 1 (FPRL1) in SAA signal transduction was\n",
      "      determined by RNA interference. Binding of SAA and FPRL1 was examined by\n",
      "      co-immunoprecipitaion. Western blotting was conducted to assess phosphorylation\n",
      "      of downstream signalling molecules. RESULTS: SAA was highly expressed in skin\n",
      "      lesions of IMQ-treated psoriasis-like mice and neutralising SAA attenuated\n",
      "      epidermal hyperplasia and inflammation. SAA in vitro promoted keratinocyte\n",
      "      proliferation and expression of immunological mediators, while inhibiting\n",
      "      differentiation. Effects of SAA on keratinocyte proliferation and inflammation\n",
      "      were mediated by FPRL1, as well as activation of the PI3K/Akt pathway.\n",
      "      CONCLUSIONS: These observations indicate that SAA/FPRL1 contributed to\n",
      "      pathogenesis of psoriasis by promoting keratinocyte proliferation and\n",
      "      inflammation, thus providing a potential therapeutic target for disease therapy.\n",
      "CI  - (c) 2016 John Wiley & Sons Ltd.\n",
      "FAU - Yu, Ning\n",
      "AU  - Yu N\n",
      "AD  - Department of Dermatology, Shanghai Skin Disease Hospital, Shanghai, China.\n",
      "FAU - Zhang, Shujie\n",
      "AU  - Zhang S\n",
      "AD  - Experimental Research Center, Eye and ENT Hospital of Fudan University, Shanghai,\n",
      "      China.\n",
      "FAU - Lu, Jiajing\n",
      "AU  - Lu J\n",
      "AD  - Department of Dermatology, Shanghai Skin Disease Hospital, Shanghai, China.\n",
      "FAU - Li, Ying\n",
      "AU  - Li Y\n",
      "AD  - Department of Dermatology, Shanghai Skin Disease Hospital, Shanghai, China.\n",
      "FAU - Yi, Xuemei\n",
      "AU  - Yi X\n",
      "AD  - Department of Dermatology, Shanghai Skin Disease Hospital, Shanghai, China.\n",
      "FAU - Tang, Li\n",
      "AU  - Tang L\n",
      "AD  - Department of Dermatology, Shanghai Skin Disease Hospital, Shanghai, China.\n",
      "FAU - Su, Lina\n",
      "AU  - Su L\n",
      "AD  - Department of Dermatology, Shanghai Skin Disease Hospital, Shanghai, China.\n",
      "FAU - Ding, Yangfeng\n",
      "AU  - Ding Y\n",
      "AD  - Department of Dermatology, Shanghai Skin Disease Hospital, Shanghai, China.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161202\n",
      "PL  - England\n",
      "TA  - Cell Prolif\n",
      "JT  - Cell proliferation\n",
      "JID - 9105195\n",
      "OTO - NOTNLM\n",
      "OT  - imiquimod-induced psoriasis-like mouse\n",
      "OT  - keratinocytes\n",
      "OT  - psoriasis\n",
      "OT  - serum amyloid A\n",
      "EDAT- 2016/12/03 06:00\n",
      "MHDA- 2016/12/03 06:00\n",
      "CRDT- 2016/12/03 06:00\n",
      "PHST- 2016/08/30 [received]\n",
      "PHST- 2016/10/27 [accepted]\n",
      "AID - 10.1111/cpr.12320 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Cell Prolif. 2016 Dec 2. doi: 10.1111/cpr.12320.\n",
      "\n",
      "PMID- 27910050\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "DA  - 20161202\n",
      "LR  - 20161203\n",
      "IS  - 1940-6029 (Electronic)\n",
      "IS  - 1064-3745 (Linking)\n",
      "VI  - 1521\n",
      "DP  - 2017\n",
      "TI  - Gene Transfer to Rodent Hearts In Vivo.\n",
      "PG  - 195-204\n",
      "AB  - Altering expression of specific genes in rodent models allows the dissection of\n",
      "      various pathways involved in various disease states. Advances in gene transfer\n",
      "      technologies produced vectors with tissue specificity, various time courses,\n",
      "      increasing or knocking down genes in an externally controlled fashion. To achieve\n",
      "      this goal, appropriate vector delivery method is essential for successfully\n",
      "      conducting experiments. In this chapter, we focus on cardiac gene transfer in\n",
      "      rodents that can be employed for delivering both viral and nonviral vectors.\n",
      "FAU - Del Monte, Federica\n",
      "AU  - Del Monte F\n",
      "AD  - Cardiovascular Research Center, Beth Israel Deaconess Hospital, Boston, MA, USA.\n",
      "FAU - Ishikawa, Kiyotake\n",
      "AU  - Ishikawa K\n",
      "AD  - Cardiovascular Research Center, Icahn School of Medicine at Mount Sinai, One\n",
      "      Gustave L. Levy Place, Box 1030, New York, NY, 10029-6574, USA.\n",
      "FAU - Hajjar, Roger J\n",
      "AU  - Hajjar RJ\n",
      "AD  - Cardiovascular Research Center, Icahn School of Medicine at Mount Sinai, One\n",
      "      Gustave L. Levy Place, Box 1030, New York, NY, 10029-6574, USA.\n",
      "      roger.hajjar@mssm.edu.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PL  - United States\n",
      "TA  - Methods Mol Biol\n",
      "JT  - Methods in molecular biology (Clifton, N.J.)\n",
      "JID - 9214969\n",
      "OTO - NOTNLM\n",
      "OT  - Adeno virus\n",
      "OT  - Adeno-associated virus\n",
      "OT  - Gene delivery\n",
      "OT  - Modified RNA\n",
      "OT  - Mouse\n",
      "OT  - Plasmid\n",
      "OT  - Rat\n",
      "OT  - Small animals\n",
      "EDAT- 2016/12/03 06:00\n",
      "MHDA- 2016/12/03 06:00\n",
      "CRDT- 2016/12/03 06:00\n",
      "AID - 10.1007/978-1-4939-6588-5_13 [doi]\n",
      "PST - ppublish\n",
      "SO  - Methods Mol Biol. 2017;1521:195-204.\n",
      "\n",
      "PMID- 27909811\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161202\n",
      "LR  - 20161203\n",
      "IS  - 1432-1335 (Electronic)\n",
      "IS  - 0171-5216 (Linking)\n",
      "DP  - 2016 Dec 01\n",
      "TI  - Plasma microRNA-103, microRNA-107, and microRNA-194 levels are not biomarkers for\n",
      "      human diffuse gastric cancer.\n",
      "AB  - PURPOSE: Diffuse-type gastric cancer (DGC) carries a poor prognosis. Effective\n",
      "      screening is one measure that might improve the prognosis of this disease. An\n",
      "      E-cadherin/p53 double-conditional knockout (DCKO) mouse line recapitulates human \n",
      "      DGC morphologically and molecularly. Three circulating microRNAs (miRNA)\n",
      "      (miR-103, miR-107, miR-194) in DCKO mice have been identified as biomarkers for\n",
      "      DGC. We sought to evaluate whether these circulating miRNAs could be used for the\n",
      "      detection of human DGC. METHODS: Subjects were 50 patients with DGC. Controls\n",
      "      were first-time outpatients at Aichi Cancer Center Hospital, age- and\n",
      "      sex-matched, without a cancer diagnosis. Total RNA containing miRNA was extracted\n",
      "      from the plasma samples and then reverse-transcribed. The levels of miRNAs in\n",
      "      plasma samples were quantitatively determined by real-time RT-PCR. Spiked-in\n",
      "      cel-miR-39 was analyzed as a normalization control. RESULTS: Levels of the three \n",
      "      plasma microRNA levels in DGC cases with or without an intestinal component were \n",
      "      not significantly different from those in control subjects. The areas under the\n",
      "      receiver operating characteristic curve of miR-103, miR-107, and miR-194 were\n",
      "      0.548, 0.563, and 0.512, respectively. CONCLUSIONS: In contrast to the DCKO mouse\n",
      "      model, plasma miR-103, miR-107, and miR-194 levels are not altered in DGC and are\n",
      "      not suitable for human DGC screening.\n",
      "FAU - Oze, Isao\n",
      "AU  - Oze I\n",
      "AUID- ORCID: http://orcid.org/0000-0002-0762-1147\n",
      "AD  - Division of Epidemiology and Prevention, Aichi Cancer Center Research Institute, \n",
      "      1-1 Kanokoden, Chikusa-ku, Nagoya, Aichi, Japan. i_oze@aichi-cc.jp.\n",
      "FAU - Shimada, Shu\n",
      "AU  - Shimada S\n",
      "AD  - Department of Molecular Oncology, Graduate School of Medical and Dental Sciences,\n",
      "      Tokyo Medical and Dental University, Tokyo, Japan.\n",
      "FAU - Nagasaki, Hiromi\n",
      "AU  - Nagasaki H\n",
      "AD  - Department of Molecular Oncology, Graduate School of Medical and Dental Sciences,\n",
      "      Tokyo Medical and Dental University, Tokyo, Japan.\n",
      "FAU - Akiyama, Yoshimitsu\n",
      "AU  - Akiyama Y\n",
      "AD  - Department of Molecular Oncology, Graduate School of Medical and Dental Sciences,\n",
      "      Tokyo Medical and Dental University, Tokyo, Japan.\n",
      "FAU - Watanabe, Miki\n",
      "AU  - Watanabe M\n",
      "AD  - Division of Epidemiology and Prevention, Aichi Cancer Center Research Institute, \n",
      "      1-1 Kanokoden, Chikusa-ku, Nagoya, Aichi, Japan.\n",
      "FAU - Yatabe, Yasushi\n",
      "AU  - Yatabe Y\n",
      "AD  - Department of Pathology and Molecular Diagnostics, Aichi Cancer Center Hospital, \n",
      "      Nagoya, Japan.\n",
      "FAU - Matsuo, Keitaro\n",
      "AU  - Matsuo K\n",
      "AD  - Division of Molecular and Clinical Epidemiology, Aichi Cancer Center Research\n",
      "      Institute, Nagoya, Japan.\n",
      "FAU - Yuasa, Yasuhito\n",
      "AU  - Yuasa Y\n",
      "AD  - Department of Molecular Oncology, Graduate School of Medical and Dental Sciences,\n",
      "      Tokyo Medical and Dental University, Tokyo, Japan.\n",
      "LA  - eng\n",
      "PT  - Letter\n",
      "DEP - 20161201\n",
      "PL  - Germany\n",
      "TA  - J Cancer Res Clin Oncol\n",
      "JT  - Journal of cancer research and clinical oncology\n",
      "JID - 7902060\n",
      "OTO - NOTNLM\n",
      "OT  - E-cadherin\n",
      "OT  - Gastric cancer/PC\n",
      "OT  - MicroRNAs\n",
      "OT  - p53 genes\n",
      "EDAT- 2016/12/03 06:00\n",
      "MHDA- 2016/12/03 06:00\n",
      "CRDT- 2016/12/03 06:00\n",
      "PHST- 2016/11/04 [received]\n",
      "PHST- 2016/11/28 [accepted]\n",
      "AID - 10.1007/s00432-016-2316-z [doi]\n",
      "AID - 10.1007/s00432-016-2316-z [pii]\n",
      "PST - aheadofprint\n",
      "SO  - J Cancer Res Clin Oncol. 2016 Dec 1.\n",
      "\n",
      "PMID- 27909059\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161202\n",
      "LR  - 20161203\n",
      "IS  - 1083-351X (Electronic)\n",
      "IS  - 0021-9258 (Linking)\n",
      "DP  - 2016 Dec 01\n",
      "TI  - The Role of the Histone Methyltransferase Enhancer of Zeste Homolog 2 (EZH2) in\n",
      "      the Pathobiological Mechanisms Underlying Inflammatory Bowel Disease (IBD).\n",
      "LID - jbc.M116.749663 [pii]\n",
      "AB  - Regulatory T (Treg) cells expressing the transcription factor FOXP3 play a\n",
      "      pivotal role in maintaining immunologic self-tolerance. We and others have\n",
      "      previously shown that EZH2 is recruited to the FOXP3 promoter and its targets in \n",
      "      Treg cells. To further address the role for EZH2 in Treg cellular function, we\n",
      "      have now generated mice that lack EZH2 specifically in Treg cells\n",
      "      (EZH2DeltaFOXP3+). We find that EZH2 deficiency in FOXP3+ T cells results in\n",
      "      lethal multi-organ autoimmunity. We further demonstrate that EZH2DeltaFOXP3+ T\n",
      "      cells lack a regulatory phenotype in vitro and secrete pro-inflammatory\n",
      "      cytokines. Of special interest, EZH2DeltaFOXP3+ mice develop spontaneous\n",
      "      inflammatory bowel disease. Guided by these results, we assessed the FOXP3 and\n",
      "      EZH2 gene networks by RNA-Seq in isolated intestinal CD4+ T cells from patients\n",
      "      with Crohns disease. Gene network analysis demonstrates that these CD4+ T cells\n",
      "      display a Th1/Th17-like phenotype with an enrichment of gene targets shared by\n",
      "      FOXP3 and EZH2. Combined, these results suggest that the inflammatory milieu\n",
      "      found in Crohns disease lead to or result from deregulation of\n",
      "      FOXP3/EZH2-enforced T cell gene networks contributing to the underlying\n",
      "      intestinal inflammation.\n",
      "CI  - Copyright (c) 2016, The American Society for Biochemistry and Molecular Biology.\n",
      "FAU - Sarmento, Olga F\n",
      "AU  - Sarmento OF\n",
      "AD  - Mayo Clinic, United States.\n",
      "FAU - Svingen, Phyllis A\n",
      "AU  - Svingen PA\n",
      "AD  - Mayo Clinic, United States.\n",
      "FAU - Xiong, Yuning\n",
      "AU  - Xiong Y\n",
      "AD  - Mayo Clinic, United States.\n",
      "FAU - Sun, Zhifu\n",
      "AU  - Sun Z\n",
      "AD  - Mayo Clinic, United States.\n",
      "FAU - Bamidele, Adebowale O\n",
      "AU  - Bamidele AO\n",
      "AD  - Mayo Clinic, United States.\n",
      "FAU - Mathison, Angela J\n",
      "AU  - Mathison AJ\n",
      "AD  - Mayo Clinic, United States.\n",
      "FAU - Smyrk, Thomas C\n",
      "AU  - Smyrk TC\n",
      "AD  - Mayo Clinic, United States.\n",
      "FAU - Nair, Asha A\n",
      "AU  - Nair AA\n",
      "AD  - Mayo Clinic, United States.\n",
      "FAU - Gonzalez, Michelle M\n",
      "AU  - Gonzalez MM\n",
      "AD  - Mayo Clinic, United States.\n",
      "FAU - Sagstetter, Mary R\n",
      "AU  - Sagstetter MR\n",
      "AD  - Mayo Clinic, United States.\n",
      "FAU - Baheti, Saurabh\n",
      "AU  - Baheti S\n",
      "AD  - Mayo Clinic, United States.\n",
      "FAU - McGovern, Dermot P B\n",
      "AU  - McGovern DP\n",
      "AD  - Cedars Sinai Hospital, United States.\n",
      "FAU - Friton, Jessica J\n",
      "AU  - Friton JJ\n",
      "AD  - Mayo Clinic, United States.\n",
      "FAU - Papadakis, Konstantinos A\n",
      "AU  - Papadakis KA\n",
      "AD  - Mayo Clinic, United States.\n",
      "FAU - Gautam, Goel\n",
      "AU  - Gautam G\n",
      "AD  - Mass General Hospital, United States.\n",
      "FAU - Xavier, Ramnik J\n",
      "AU  - Xavier RJ\n",
      "AD  - Mass General Hospital, United States.\n",
      "FAU - Urrutia, Raul A\n",
      "AU  - Urrutia RA\n",
      "AD  - Mayo Clinic, United States.\n",
      "FAU - Faubion, William A\n",
      "AU  - Faubion WA\n",
      "AD  - Mayo Clinic, United States; faubion.william@mayo.edu.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161201\n",
      "PL  - United States\n",
      "TA  - J Biol Chem\n",
      "JT  - The Journal of biological chemistry\n",
      "JID - 2985121R\n",
      "OTO - NOTNLM\n",
      "OT  - Crohn?s disease\n",
      "OT  - EZH2\n",
      "OT  - T helper cells\n",
      "OT  - T regulatory cell\n",
      "OT  - epigenetics\n",
      "OT  - forkhead box P3 (FOXP3)\n",
      "OT  - histone methylation\n",
      "OT  - inflammatory bowel disease (IBD)\n",
      "OT  - transcriptional corepressor\n",
      "EDAT- 2016/12/03 06:00\n",
      "MHDA- 2016/12/03 06:00\n",
      "CRDT- 2016/12/03 06:00\n",
      "PHST- 2016/07/21 [received]\n",
      "PHST- 2016/12/01 [accepted]\n",
      "AID - M116.749663 [pii]\n",
      "AID - 10.1074/jbc.M116.749663 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - J Biol Chem. 2016 Dec 1. pii: jbc.M116.749663.\n",
      "\n",
      "PMID- 27909014\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161202\n",
      "LR  - 20161203\n",
      "IS  - 1940-4034 (Electronic)\n",
      "IS  - 1074-2484 (Linking)\n",
      "DP  - 2016 Dec 01\n",
      "TI  - Sodium 4-Phenylbutyrate Attenuates Myocardial Reperfusion Injury by Reducing the \n",
      "      Unfolded Protein Response.\n",
      "LID - 1074248416679308 [pii]\n",
      "AB  - BACKGROUND: The unfolded protein response (UPR) plays a pivotal role in\n",
      "      ischemia-reperfusion (I/R) injury in various organs such as heart, brain, and\n",
      "      liver. Sodium 4-phenylbutyrate (PBA) reportedly acts as a chemical chaperone that\n",
      "      reduces UPR. In the present study, we evaluated the effect of PBA on reducing the\n",
      "      UPR and protecting against myocardial I/R injury in mice. METHODS: Male C57BL/6\n",
      "      mice were subjected to 30-minute myocardial I/R, and were treated with\n",
      "      phosphate-buffered saline (as a vehicle) or PBA. RESULTS: At 4 hours after\n",
      "      reperfusion, mice treated with PBA had reduced serum cardiac troponin I levels\n",
      "      and numbers of apoptotic cells in left ventricles (LVs) in myocardial I/R.\n",
      "      Infarct size had also reduced in mice treated with PBA at 48 hours after\n",
      "      reperfusion. At 2 hours after reperfusion, UPR markers, including eukaryotic\n",
      "      initiation of the factor 2alpha-subunit, activating transcription factor-6,\n",
      "      inositol-requiring enzyme-1, glucose-regulated protein 78, CCAAT/enhancer-binding\n",
      "      protein (C/EBP) homologous protein, and caspase-12, were significantly increased \n",
      "      in mice treated with vehicle compared to sham-operated mice. Administration of\n",
      "      PBA significantly reduced the I/R-induced increases of these markers. Cardiac\n",
      "      function and dimensions were assessed at 21 days after I/R. Sodium\n",
      "      4-phenylbutyrate dedicated to the improvement of cardiac parameters deterioration\n",
      "      including LV end-diastolic diameter and LV fractional shortening. Consistently,\n",
      "      PBA reduced messenger RNA expression levels of cardiac remodeling markers such as\n",
      "      collagen type 1alpha1, brain natriuretic peptide, and alpha skeletal muscle actin\n",
      "      in LV at 21 days after I/R. CONCLUSION: Unfolded protein response mediates\n",
      "      myocardial I/R injury. Administration of PBA reduces the UPR, apoptosis, infarct \n",
      "      size, and preserved cardiac function. Hence, PBA may be a therapeutic option to\n",
      "      attenuate myocardial I/R injury in clinical practice.\n",
      "CI  - (c) The Author(s) 2016.\n",
      "FAU - Takatori, Osamu\n",
      "AU  - Takatori O\n",
      "AD  - Department of Disease Control and Homeostasis, Kanazawa University Graduate\n",
      "      School of Medical Science, Kanazawa, Japan.\n",
      "FAU - Usui, Soichiro\n",
      "AU  - Usui S\n",
      "AD  - Department of Disease Control and Homeostasis, Kanazawa University Graduate\n",
      "      School of Medical Science, Kanazawa, Japan.\n",
      "FAU - Okajima, Masaki\n",
      "AU  - Okajima M\n",
      "AD  - Department of Disease Control and Homeostasis, Kanazawa University Graduate\n",
      "      School of Medical Science, Kanazawa, Japan mmokaji@gmail.com.\n",
      "FAU - Kaneko, Shuichi\n",
      "AU  - Kaneko S\n",
      "AD  - Department of Disease Control and Homeostasis, Kanazawa University Graduate\n",
      "      School of Medical Science, Kanazawa, Japan.\n",
      "FAU - Ootsuji, Hiroshi\n",
      "AU  - Ootsuji H\n",
      "AD  - Department of Disease Control and Homeostasis, Kanazawa University Graduate\n",
      "      School of Medical Science, Kanazawa, Japan.\n",
      "FAU - Takashima, Shin-Ichiro\n",
      "AU  - Takashima SI\n",
      "AD  - Department of Disease Control and Homeostasis, Kanazawa University Graduate\n",
      "      School of Medical Science, Kanazawa, Japan.\n",
      "FAU - Kobayashi, Daisuke\n",
      "AU  - Kobayashi D\n",
      "AD  - Department of Disease Control and Homeostasis, Kanazawa University Graduate\n",
      "      School of Medical Science, Kanazawa, Japan.\n",
      "FAU - Murai, Hisayoshi\n",
      "AU  - Murai H\n",
      "AD  - Department of Disease Control and Homeostasis, Kanazawa University Graduate\n",
      "      School of Medical Science, Kanazawa, Japan.\n",
      "FAU - Furusho, Hiroshi\n",
      "AU  - Furusho H\n",
      "AD  - Department of Disease Control and Homeostasis, Kanazawa University Graduate\n",
      "      School of Medical Science, Kanazawa, Japan.\n",
      "FAU - Takamura, Masayuki\n",
      "AU  - Takamura M\n",
      "AD  - Department of Disease Control and Homeostasis, Kanazawa University Graduate\n",
      "      School of Medical Science, Kanazawa, Japan.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161201\n",
      "PL  - United States\n",
      "TA  - J Cardiovasc Pharmacol Ther\n",
      "JT  - Journal of cardiovascular pharmacology and therapeutics\n",
      "JID - 9602617\n",
      "OTO - NOTNLM\n",
      "OT  - apoptosis\n",
      "OT  - myocardial ischemia-reperfusion injury\n",
      "OT  - sodium 4-phenylbutyrate\n",
      "OT  - unfolded protein response\n",
      "EDAT- 2016/12/03 06:00\n",
      "MHDA- 2016/12/03 06:00\n",
      "CRDT- 2016/12/03 06:00\n",
      "PHST- 2014/02/10 [received]\n",
      "PHST- 2016/09/06 [accepted]\n",
      "AID - 1074248416679308 [pii]\n",
      "AID - 10.1177/1074248416679308 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - J Cardiovasc Pharmacol Ther. 2016 Dec 1. pii: 1074248416679308.\n",
      "\n",
      "PMID- 27906075\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "DA  - 20161201\n",
      "LR  - 20161217\n",
      "IS  - 2044-5040 (Electronic)\n",
      "IS  - 2044-5040 (Linking)\n",
      "VI  - 6\n",
      "IP  - 1\n",
      "DP  - 2016 Dec 01\n",
      "TI  - Nuclear bodies reorganize during myogenesis in vitro and are differentially\n",
      "      disrupted by expression of FSHD-associated DUX4.\n",
      "PG  - 42\n",
      "AB  - BACKGROUND: Nuclear bodies, such as nucleoli, PML bodies, and SC35 speckles, are \n",
      "      dynamic sub-nuclear structures that regulate multiple genetic and epigenetic\n",
      "      processes. Additional regulation is provided by RNA/DNA handling proteins,\n",
      "      notably TDP-43 and FUS, which have been linked to ALS pathology. Previous work\n",
      "      showed that mouse cell line myotubes have fewer but larger nucleoli than\n",
      "      myoblasts, and we had found that nuclear aggregation of TDP-43 in human myotubes \n",
      "      was induced by expression of DUX4-FL, a transcription factor that is aberrantly\n",
      "      expressed and causes pathology in facioscapulohumeral dystrophy (FSHD). However, \n",
      "      questions remained about nuclear bodies in human myogenesis and in muscle\n",
      "      disease. METHODS: We examined nucleoli, PML bodies, SC35 speckles, TDP-43, and\n",
      "      FUS in myoblasts and myotubes derived from healthy donors and from patients with \n",
      "      FSHD, laminin-alpha-2-deficiency (MDC1A), and alpha-sarcoglycan-deficiency\n",
      "      (LGMD2D). We further examined how these nuclear bodies and proteins were affected\n",
      "      by DUX4-FL expression. RESULTS: We found that nucleoli, PML bodies, and SC35\n",
      "      speckles reorganized during differentiation in vitro, with all three becoming\n",
      "      less abundant in myotube vs. myoblast nuclei. In addition, though PML bodies did \n",
      "      not change in size, both nucleoli and SC35 speckles were larger in myotube than\n",
      "      myoblast nuclei. Similar patterns of nuclear body reorganization occurred in\n",
      "      healthy control, MDC1A, and LGMD2D cultures, as well as in the large fraction of \n",
      "      nuclei that did not show DUX4-FL expression in FSHD cultures. In contrast, nuclei\n",
      "      that expressed endogenous or exogenous DUX4-FL, though retaining normal nucleoli,\n",
      "      showed disrupted morphology of some PML bodies and most SC35 speckles and also\n",
      "      co-aggregation of FUS with TDP-43. CONCLUSIONS: Nucleoli, PML bodies, and SC35\n",
      "      speckles reorganize during human myotube formation in vitro. These nuclear body\n",
      "      reorganizations are likely needed to carry out the distinct gene transcription\n",
      "      and splicing patterns that are induced upon myotube formation. DUX4-FL-induced\n",
      "      disruption of some PML bodies and most SC35 speckles, along with co-aggregation\n",
      "      of TDP-43 and FUS, could contribute to pathogenesis in FSHD, perhaps by locally\n",
      "      interfering with genetic and epigenetic regulation of gene expression in the\n",
      "      small subset of nuclei that express high levels of DUX4-FL at any one time.\n",
      "FAU - Homma, Sachiko\n",
      "AU  - Homma S\n",
      "AD  - Department of Neurology, Boston University School of Medicine, Boston, MA, 02118,\n",
      "      USA.\n",
      "FAU - Beermann, Mary Lou\n",
      "AU  - Beermann ML\n",
      "AD  - Department of Neurology, Boston University School of Medicine, Boston, MA, 02118,\n",
      "      USA.\n",
      "FAU - Yu, Bryant\n",
      "AU  - Yu B\n",
      "AD  - Department of Neurology, Boston University School of Medicine, Boston, MA, 02118,\n",
      "      USA.\n",
      "FAU - Boyce, Frederick M\n",
      "AU  - Boyce FM\n",
      "AD  - Department of Neurology, Massachusetts General Hospital, Boston, MA, 02114, USA.\n",
      "FAU - Miller, Jeffrey Boone\n",
      "AU  - Miller JB\n",
      "AD  - Department of Neurology, Boston University School of Medicine, Boston, MA, 02118,\n",
      "      USA. millerjb@bu.edu.\n",
      "AD  - Neuromuscular Biology & Disease Group, Department of Neurology, Boston University\n",
      "      School of Medicine, 700 Albany Street, Boston, MA, 02118, USA. millerjb@bu.edu.\n",
      "LA  - eng\n",
      "GR  - R01 AR060328/AR/NIAMS NIH HHS/United States\n",
      "GR  - R01 AR062587/AR/NIAMS NIH HHS/United States\n",
      "GR  - UL1 TR001430/TR/NCATS NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "DEP - 20161201\n",
      "PL  - England\n",
      "TA  - Skelet Muscle\n",
      "JT  - Skeletal muscle\n",
      "JID - 101561193\n",
      "PMC - PMC5134237\n",
      "OTO - NOTNLM\n",
      "OT  - DUX4\n",
      "OT  - FUS\n",
      "OT  - Facioscapulohumeral muscular dystrophy\n",
      "OT  - Myotube\n",
      "OT  - Nucleoli\n",
      "OT  - PML bodies\n",
      "OT  - SC35 speckles\n",
      "OT  - TDP-43\n",
      "EDAT- 2016/12/03 06:00\n",
      "MHDA- 2016/12/03 06:00\n",
      "CRDT- 2016/12/02 06:00\n",
      "PHST- 2016/08/13 [received]\n",
      "PHST- 2016/11/17 [accepted]\n",
      "AID - 10.1186/s13395-016-0113-7 [doi]\n",
      "AID - 10.1186/s13395-016-0113-7 [pii]\n",
      "PST - epublish\n",
      "SO  - Skelet Muscle. 2016 Dec 1;6(1):42.\n",
      "\n",
      "PMID- 27903277\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "DA  - 20161201\n",
      "LR  - 20161215\n",
      "IS  - 1743-422X (Electronic)\n",
      "IS  - 1743-422X (Linking)\n",
      "VI  - 13\n",
      "IP  - 1\n",
      "DP  - 2016 Nov 30\n",
      "TI  - Sequence diversity of dengue virus type 2 in brain and thymus of infected\n",
      "      interferon receptor ko mice: implications for dengue virulence.\n",
      "PG  - 199\n",
      "AB  - BACKGROUND: We previously reported that a clinical isolate of dengue virus (DENV)\n",
      "      is capable of causing acute-phase systemic infection in mice harboring knockouts \n",
      "      of the genes encoding type-I and -II interferon IFN receptors\n",
      "      (IFN-alpha/beta/gammaR KO mice); in contrast, other virulent DENV isolates\n",
      "      exhibited slow disease progression in this mice, yielding lethal infection around\n",
      "      20 days post-infection (p.i.). In the present study, we sought to clarify the\n",
      "      dynamics of slow disease progression by examining disease progression of a type-2\n",
      "      DENV clinical isolate (DV2P04/08) in mice. METHODS: The tissue distributions of\n",
      "      DV2P04/08 in several organs of infeted mice were examined at different time\n",
      "      points. Whole genome viral sequences from organs were determined. RESULTS: At day\n",
      "      6 p.i., high levels of viral RNA (vRNA) were detected in non-neuronal organs\n",
      "      (including peritoneal exudate cells (PECs), spleen, kidney, liver, lung, and bone\n",
      "      marrow) but not in brain. By day 14 p.i, vRNA levels subsequently decreased in\n",
      "      most organs, with the exception of thymus and brain. Sequence analysis of the\n",
      "      whole genome of the original P04/08 and those of viruses recovered from mouse\n",
      "      brain and thymus demonstrated the presence of both synonymous and non-synonymous \n",
      "      mutations. Individual mice showed different virus populations in the brain. The\n",
      "      vRNA sequence derived from brain of one mouse was nearly identical to the\n",
      "      original DV2P04/08 inoculum, suggesting that there was no need for adaptation of \n",
      "      DV2P04/08 for growth in the brain. However, quasispecies (that is, mixed\n",
      "      populations, detected as apparent nucleotide mixtures during sequencing) were\n",
      "      observed in the thymus of another mouse, and interestingly only mutant population\n",
      "      invaded the brain at a late stage of infection. CONCLUSIONS: These results\n",
      "      suggested that the mouse nearly succeeded in eliminating virus from non-neuronal \n",
      "      organs but failed to do so from brain. Although the cause of death by DV2P04/08\n",
      "      infection is likely to be the result of virus invasion to brain, its processes to\n",
      "      the death are different in individual mice. This study will provide a new insight\n",
      "      into disease progression of DENV in mice.\n",
      "FAU - Dhole, Priya\n",
      "AU  - Dhole P\n",
      "AD  - Research Institute for Microbial Diseases, Osaka University, Osaka, Japan.\n",
      "FAU - Nakayama, Emi E\n",
      "AU  - Nakayama EE\n",
      "AD  - Research Institute for Microbial Diseases, Osaka University, Osaka, Japan.\n",
      "FAU - Saito, Akatsuki\n",
      "AU  - Saito A\n",
      "AD  - Research Institute for Microbial Diseases, Osaka University, Osaka, Japan.\n",
      "FAU - Limkittikul, Kriengsak\n",
      "AU  - Limkittikul K\n",
      "AD  - Faculty of Tropical Medicine, Mahidol University, Bangkok, Thailand.\n",
      "FAU - Phanthanawiboon, Supranee\n",
      "AU  - Phanthanawiboon S\n",
      "AD  - Center for Emerging Infectious Diseases, Kohn Kaen University, Kohn Kaen,\n",
      "      Thailand.\n",
      "FAU - Shioda, Tatsuo\n",
      "AU  - Shioda T\n",
      "AD  - Research Institute for Microbial Diseases, Osaka University, Osaka, Japan.\n",
      "      shioda@biken.osaka-u.ac.jp.\n",
      "FAU - Kurosu, Takeshi\n",
      "AU  - Kurosu T\n",
      "AD  - Department of Virology I, National Institute for Infectious Diseases, Tokyo,\n",
      "      Japan. kurosu@niid.go.jp.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161130\n",
      "PL  - England\n",
      "TA  - Virol J\n",
      "JT  - Virology journal\n",
      "JID - 101231645\n",
      "PMC - PMC5129197\n",
      "OTO - NOTNLM\n",
      "OT  - Dengue virus\n",
      "OT  - Genome\n",
      "OT  - Mouse model\n",
      "OT  - Mutation\n",
      "OT  - Virulence specificity\n",
      "EDAT- 2016/12/03 06:00\n",
      "MHDA- 2016/12/03 06:00\n",
      "CRDT- 2016/12/02 06:00\n",
      "PHST- 2016/08/03 [received]\n",
      "PHST- 2016/11/25 [accepted]\n",
      "AID - 10.1186/s12985-016-0658-4 [doi]\n",
      "AID - 10.1186/s12985-016-0658-4 [pii]\n",
      "PST - epublish\n",
      "SO  - Virol J. 2016 Nov 30;13(1):199.\n",
      "\n",
      "PMID- 27902457\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161130\n",
      "LR  - 20161203\n",
      "IS  - 1949-2553 (Electronic)\n",
      "IS  - 1949-2553 (Linking)\n",
      "DP  - 2016 Nov 26\n",
      "TI  - TFDP3 confers chemoresistance in minimal residual disease within childhood T-cell\n",
      "      acute lymphoblastic leukemia.\n",
      "LID - 10.18632/oncotarget.13630 [doi]\n",
      "AB  - Acquired drug resistance in childhood T-cell acute lymphoblastic leukemia (T-ALL)\n",
      "      remains a significant clinical problem. In this study, a novel gene therapy\n",
      "      target for childhood T-ALL to overcome chemoresistance was discovered: TFDP3\n",
      "      increased in the minimal residual disease (MRD) positive childhood T-ALL\n",
      "      patients. Then, we established a preclinical model of resistance to induction\n",
      "      therapy to examine the functional relevance of TFDP3 to chemoresistance in MRD\n",
      "      derived from Jurkat/E6-1. Jurkat xenografts in NOD/SCID mice were exposed to a\n",
      "      four drug combination (VXLD) of vincristine (VCR), dexamethasone (DEX),\n",
      "      L-asparaginase (L-asp) and daunorubicin (DNR). During the 4-week VXLD treatment, \n",
      "      the level of TFDP3 increased 4-fold. High expression of TFDP3 was identified in\n",
      "      the re-emerging lines (Jurkat/MRD) with increased chemoresistance, which is\n",
      "      correlated with partially promoter demethylation of TFDP3. Downregulation of\n",
      "      TFDP3 by RNA interference reversed chemoresistance in Jurkat/MRD accompanied by\n",
      "      reinstated E2F1 activity that coincided with increased levels of p53, p73, and\n",
      "      associated proapoptotic target genes. Importantly, TFDP3 silencing in vivo\n",
      "      induced apparent benefit to overcome chemoresistance in combination with VXLD\n",
      "      treatment. Collectively, TFDP3 confers chemoresistance in MRD within childhood\n",
      "      T-ALL, indicating that TFDP3 is a potential gene therapy target for residual\n",
      "      cancer.\n",
      "FAU - Chu, Ming\n",
      "AU  - Chu M\n",
      "AD  - Department of Immunology, School of Basic Medical Sciences, Peking University,\n",
      "      Beijing, China.\n",
      "AD  - Key Laboratory of Medical Immunology, Ministry of Health, Beijing, China.\n",
      "FAU - Yin, Kailin\n",
      "AU  - Yin K\n",
      "AD  - Department of Immunology, School of Basic Medical Sciences, Peking University,\n",
      "      Beijing, China.\n",
      "AD  - Key Laboratory of Medical Immunology, Ministry of Health, Beijing, China.\n",
      "FAU - Dong, Yujun\n",
      "AU  - Dong Y\n",
      "AD  - Department of Hematology, Peking University First Hospital, Beijing, China.\n",
      "FAU - Wang, Pingzhang\n",
      "AU  - Wang P\n",
      "AD  - Department of Immunology, School of Basic Medical Sciences, Peking University,\n",
      "      Beijing, China.\n",
      "AD  - Key Laboratory of Medical Immunology, Ministry of Health, Beijing, China.\n",
      "FAU - Xue, Yun\n",
      "AU  - Xue Y\n",
      "AD  - Department of Anesthesiology, Fuling Center Hospital of Chongqing City,\n",
      "      Chongqing, China.\n",
      "FAU - Zhou, Peng\n",
      "AU  - Zhou P\n",
      "AD  - Department of Immunology, School of Basic Medical Sciences, Peking University,\n",
      "      Beijing, China.\n",
      "AD  - Key Laboratory of Medical Immunology, Ministry of Health, Beijing, China.\n",
      "FAU - Wang, Yuqi\n",
      "AU  - Wang Y\n",
      "AD  - Department of Immunology, School of Basic Medical Sciences, Peking University,\n",
      "      Beijing, China.\n",
      "AD  - Key Laboratory of Medical Immunology, Ministry of Health, Beijing, China.\n",
      "FAU - Wang, Yuedan\n",
      "AU  - Wang Y\n",
      "AD  - Department of Immunology, School of Basic Medical Sciences, Peking University,\n",
      "      Beijing, China.\n",
      "AD  - Key Laboratory of Medical Immunology, Ministry of Health, Beijing, China.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161126\n",
      "PL  - United States\n",
      "TA  - Oncotarget\n",
      "JT  - Oncotarget\n",
      "JID - 101532965\n",
      "OTO - NOTNLM\n",
      "OT  - E2F1\n",
      "OT  - TFDP3\n",
      "OT  - chemoresistance\n",
      "OT  - childhood T-ALL\n",
      "OT  - minimal residual disease\n",
      "EDAT- 2016/12/03 06:00\n",
      "MHDA- 2016/12/03 06:00\n",
      "CRDT- 2016/12/01 06:00\n",
      "PHST- 2016/09/06 [received]\n",
      "PHST- 2016/11/08 [accepted]\n",
      "AID - 13630 [pii]\n",
      "AID - 10.18632/oncotarget.13630 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Oncotarget. 2016 Nov 26. doi: 10.18632/oncotarget.13630.\n",
      "\n",
      "PMID- 27901018\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161130\n",
      "LR  - 20161203\n",
      "IS  - 1935-3456 (Electronic)\n",
      "IS  - 1933-0219 (Linking)\n",
      "DP  - 2016 Nov 30\n",
      "TI  - The signaling axis of microRNA-31/interleukin-25 regulates Th1/Th17-mediated\n",
      "      inflammation response in colitis.\n",
      "LID - 10.1038/mi.2016.102 [doi]\n",
      "AB  - Interleukin-25 (IL-25) is an important regulatory cytokine that has a key role on\n",
      "      mucosal immune tolerance during inflammation response. However, the molecular\n",
      "      mechanism that regulates the colonic IL-25 expression in Crohn's disease (CD)\n",
      "      remains unclear. In this study, IL-25 level was proved to decrease in\n",
      "      2,4,6-trinitrobenzene sulfonic acid (TNBS)-induced colitis mice and IL-10\n",
      "      knockout (KO) spontaneous colitis mice. An inverse correlation between IL-25 and \n",
      "      miR-31 was discovered in the colons from model mice and CD patients. Furthermore,\n",
      "      target validation analysis demonstrated that miR-31 directly regulated IL-25\n",
      "      expression by binding to its messenger RNA 3'-untranslated region. Changing\n",
      "      colonic miR-31 level in the colitis mice could affect the mucosal\n",
      "      IL-12/23-mediated Th1/Th17 pathway and lead to either amelioration or aggravation\n",
      "      of colonic inflammation. In addition, the therapeutic effects of anti-miR-31 in\n",
      "      TNBS-induced colitis were abolished by colonic treatment with IL-25 antibody or\n",
      "      colonic down-expression of IL-25. Our findings demonstrated that IL-25 could be a\n",
      "      crucial anti-inflammatory cytokine in TNBS-induced colitis and the signaling of\n",
      "      miR-31 targeting IL-25 might be a possible mechanism that regulates\n",
      "      IL-12/23-mediated Th1/Th17 inflammatory responses during colonic inflammation\n",
      "      process. Restoring colonic IL-25 expression and blocking Th1/Th17 responses via\n",
      "      intracolonic administration of miR-31 inhibitor may represent a promising\n",
      "      approach for CD treatment.Mucosal Immunology advance online publication 30\n",
      "      November 2016; doi:10.1038/mi.2016.102.\n",
      "FAU - Shi, T\n",
      "AU  - Shi T\n",
      "AD  - State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences,\n",
      "      Nanjing University, Nanjing, Jiangsu, China.\n",
      "FAU - Xie, Y\n",
      "AU  - Xie Y\n",
      "AD  - State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences,\n",
      "      Nanjing University, Nanjing, Jiangsu, China.\n",
      "FAU - Fu, Y\n",
      "AU  - Fu Y\n",
      "AD  - State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences,\n",
      "      Nanjing University, Nanjing, Jiangsu, China.\n",
      "FAU - Zhou, Q\n",
      "AU  - Zhou Q\n",
      "AD  - State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences,\n",
      "      Nanjing University, Nanjing, Jiangsu, China.\n",
      "FAU - Ma, Z\n",
      "AU  - Ma Z\n",
      "AD  - State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences,\n",
      "      Nanjing University, Nanjing, Jiangsu, China.\n",
      "FAU - Ma, J\n",
      "AU  - Ma J\n",
      "AD  - State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences,\n",
      "      Nanjing University, Nanjing, Jiangsu, China.\n",
      "FAU - Huang, Z\n",
      "AU  - Huang Z\n",
      "AD  - State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences,\n",
      "      Nanjing University, Nanjing, Jiangsu, China.\n",
      "FAU - Zhang, J\n",
      "AU  - Zhang J\n",
      "AD  - State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences,\n",
      "      Nanjing University, Nanjing, Jiangsu, China.\n",
      "AD  - State Key Laboratory of Analytical Chemistry for Life Sciences and Collaborative \n",
      "      Innovation Center of Chemistry for Life Sciences, Nanjing University, Nanjing,\n",
      "      Jiangsu, China.\n",
      "FAU - Chen, J\n",
      "AU  - Chen J\n",
      "AD  - State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences,\n",
      "      Nanjing University, Nanjing, Jiangsu, China.\n",
      "AD  - State Key Laboratory of Analytical Chemistry for Life Sciences and Collaborative \n",
      "      Innovation Center of Chemistry for Life Sciences, Nanjing University, Nanjing,\n",
      "      Jiangsu, China.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161130\n",
      "PL  - United States\n",
      "TA  - Mucosal Immunol\n",
      "JT  - Mucosal immunology\n",
      "JID - 101299742\n",
      "EDAT- 2016/12/03 06:00\n",
      "MHDA- 2016/12/03 06:00\n",
      "CRDT- 2016/12/01 06:00\n",
      "PHST- 2015/12/31 [received]\n",
      "PHST- 2016/10/01 [accepted]\n",
      "AID - mi2016102 [pii]\n",
      "AID - 10.1038/mi.2016.102 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Mucosal Immunol. 2016 Nov 30. doi: 10.1038/mi.2016.102.\n",
      "\n",
      "PMID- 27900797\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161130\n",
      "LR  - 20161215\n",
      "IS  - 1097-0045 (Electronic)\n",
      "IS  - 0270-4137 (Linking)\n",
      "DP  - 2016 Nov 30\n",
      "TI  - Evidence for Feedback Regulation Following Cholesterol Lowering Therapy in a\n",
      "      Prostate Cancer Xenograft Model.\n",
      "LID - 10.1002/pros.23282 [doi]\n",
      "AB  - BACKGROUND: Epidemiologic data suggest cholesterol-lowering drugs may prevent the\n",
      "      progression of prostate cancer, but not the incidence of the disease. However,\n",
      "      the association of combination therapy in cholesterol reduction on prostate or\n",
      "      any cancer is unclear. In this study, we compared the effects of the cholesterol \n",
      "      lowering drugs simvastatin and ezetimibe alone or in combination on the growth of\n",
      "      LAPC-4 prostate cancer in vivo xenografts. METHODS: Proliferation assays were\n",
      "      conducted by MTS solution and assessed by Student's t-test. 90 male nude mice\n",
      "      were placed on a high-cholesterol Western-diet for 7 days then injected\n",
      "      subcutaneously with 1 x 105 LAPC-4 cells. Two weeks post-injection, mice were\n",
      "      randomized to control, 11 mg/kg/day simvastatin, 30 mg/kg ezetimibe, or the\n",
      "      combination and sacrificed 42 days post-randomization. We used a generalized\n",
      "      linear model with the predictor variables of treatment, time, and treatment by\n",
      "      time (i.e., interaction term) with tumor volume as the outcome variable. Total\n",
      "      serum and tumor cholesterol were measured. Tumoral RNA was extracted and cDNA\n",
      "      synthesized from 1 ug of total RNA for quantitative real-time PCR. RESULTS:\n",
      "      Simvastatin directly reduced in vitro prostate cell proliferation in a\n",
      "      dose-dependent, cell line-specific manner, but ezetimibe had no effect. In vivo, \n",
      "      low continuous dosing of ezetimibe, delivered by food, or simvastatin, delivered \n",
      "      via an osmotic pump had no effect on tumor growth compared to control mice. In\n",
      "      contrast, dual treatment of simvastatin and ezetimibe accelerated tumor growth.\n",
      "      Ezetimibe significantly lowered serum cholesterol by 15%, while simvastatin had\n",
      "      no effect. Ezetimibe treatment resulted in higher tumor cholesterol. A sixfold\n",
      "      induction of low density lipoprotein receptor mRNA was observed in ezetimibe and \n",
      "      the combination with simvastatin versus control tumors. CONCLUSIONS: Systemic\n",
      "      cholesterol lowering by ezetimibe did not slow tumor growth, nor did the\n",
      "      cholesterol independent effects of simvastatin and the combined treatment\n",
      "      increased tumor growth. Despite lower serum cholesterol, tumors from ezetimibe\n",
      "      treated mice had higher levels of cholesterol. This study suggests that induction\n",
      "      of low density lipoprotein receptor is a possible mechanism of resistance that\n",
      "      prostate tumors use to counteract the therapeutic effects of lowering serum\n",
      "      cholesterol. Prostate (c) 2016 Wiley Periodicals, Inc.\n",
      "CI  - (c) 2016 Wiley Periodicals, Inc.\n",
      "FAU - Masko, Elizabeth M\n",
      "AU  - Masko EM\n",
      "AD  - Division of Urologic Surgery, Department of Surgery, Duke University Medical\n",
      "      Center, Durham, North Carolina.\n",
      "FAU - Alfaqih, Mahmoud A\n",
      "AU  - Alfaqih MA\n",
      "AD  - Division of Urologic Surgery, Department of Surgery, Duke University Medical\n",
      "      Center, Durham, North Carolina.\n",
      "FAU - Solomon, Keith R\n",
      "AU  - Solomon KR\n",
      "AD  - Department of Orthopedic Surgery, Harvard Medical School, Boston, Massachusetts.\n",
      "FAU - Barry, William T\n",
      "AU  - Barry WT\n",
      "AD  - Department of Biostatistics and Bioinformatics, Duke University, Durham, North\n",
      "      Carolina.\n",
      "FAU - Newgard, Christopher B\n",
      "AU  - Newgard CB\n",
      "AD  - Sarah W. Stedman Nutrition and Metabolism Center, Duke University, Durham, North \n",
      "      Carolina.\n",
      "FAU - Muehlbauer, Michael J\n",
      "AU  - Muehlbauer MJ\n",
      "AD  - Sarah W. Stedman Nutrition and Metabolism Center, Duke University, Durham, North \n",
      "      Carolina.\n",
      "FAU - Valilis, Nikolaos A\n",
      "AU  - Valilis NA\n",
      "AD  - Division of Urologic Surgery, Department of Surgery, Duke University Medical\n",
      "      Center, Durham, North Carolina.\n",
      "FAU - Phillips, Tameika E\n",
      "AU  - Phillips TE\n",
      "AD  - Division of Urologic Surgery, Department of Surgery, Duke University Medical\n",
      "      Center, Durham, North Carolina.\n",
      "FAU - Poulton, Susan H\n",
      "AU  - Poulton SH\n",
      "AD  - Division of Urologic Surgery, Department of Surgery, Duke University Medical\n",
      "      Center, Durham, North Carolina.\n",
      "FAU - Freedland, Alexis R\n",
      "AU  - Freedland AR\n",
      "AD  - Division of Urology, Department of Surgery, Cedars-Sinai Medical Center, Los\n",
      "      Angeles, California.\n",
      "FAU - Sun, Stephanie\n",
      "AU  - Sun S\n",
      "AD  - Department of Surgery, Durham Veterans Administration Hospital, Durham, North\n",
      "      Carolina.\n",
      "FAU - Dambal, Shweta K\n",
      "AU  - Dambal SK\n",
      "AD  - Division of Urology, Department of Surgery, Cedars-Sinai Medical Center, Los\n",
      "      Angeles, California.\n",
      "FAU - Sanders, Sergio E\n",
      "AU  - Sanders SE\n",
      "AD  - Division of Urology, Department of Surgery, Cedars-Sinai Medical Center, Los\n",
      "      Angeles, California.\n",
      "FAU - Macias, Everardo\n",
      "AU  - Macias E\n",
      "AD  - Division of Urologic Surgery, Department of Surgery, Duke University Medical\n",
      "      Center, Durham, North Carolina.\n",
      "AD  - Division of Urology, Department of Surgery, Cedars-Sinai Medical Center, Los\n",
      "      Angeles, California.\n",
      "FAU - Freeman, Michael R\n",
      "AU  - Freeman MR\n",
      "AD  - Division of Urology, Department of Surgery, Cedars-Sinai Medical Center, Los\n",
      "      Angeles, California.\n",
      "FAU - Dewhirst, Mark W\n",
      "AU  - Dewhirst MW\n",
      "AD  - Department of Radiation Oncology, Duke University Medical Center, Durham, North\n",
      "      Carolina.\n",
      "FAU - Pizzo, Salvatore V\n",
      "AU  - Pizzo SV\n",
      "AD  - Department of Pathology, Duke University Medical Center, Durham, North Carolina.\n",
      "FAU - Freedland, Stephen J\n",
      "AU  - Freedland SJ\n",
      "AD  - Division of Urology, Department of Surgery, Cedars-Sinai Medical Center, Los\n",
      "      Angeles, California.\n",
      "AD  - Department of Surgery, Durham Veterans Administration Hospital, Durham, North\n",
      "      Carolina.\n",
      "LA  - eng\n",
      "GR  - P01 CA098912/CA/NCI NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "DEP - 20161130\n",
      "PL  - United States\n",
      "TA  - Prostate\n",
      "JT  - The Prostate\n",
      "JID - 8101368\n",
      "OTO - NOTNLM\n",
      "OT  - cholesterol\n",
      "OT  - ezetimibe\n",
      "OT  - prostate cancer\n",
      "OT  - simvastatin\n",
      "EDAT- 2016/12/03 06:00\n",
      "MHDA- 2016/12/03 06:00\n",
      "CRDT- 2016/12/01 06:00\n",
      "PHST- 2016/04/07 [received]\n",
      "PHST- 2016/11/04 [accepted]\n",
      "AID - 10.1002/pros.23282 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Prostate. 2016 Nov 30. doi: 10.1002/pros.23282.\n",
      "\n",
      "PMID- 27900389\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161130\n",
      "LR  - 20161203\n",
      "IS  - 1528-3658 (Electronic)\n",
      "IS  - 1076-1551 (Linking)\n",
      "VI  - 22\n",
      "DP  - 2016 Nov 23\n",
      "TI  - The clearance of cell remnants and the regeneration of the injured muscle depend \n",
      "      on soluble pattern recognition receptor PTX3.\n",
      "LID - 10.2119/molmed.2016.00002 [doi]\n",
      "AB  - OBJECTIVE: The signals causing the resolution of muscle inflammation are only\n",
      "      partially characterized. The long pentraxin PTX3, which modulates leukocyte\n",
      "      recruitment and activation, could contribute. METHODS: We analysed the expression\n",
      "      of ptx3 after muscle injury and verified whether hematopoietic precursors are a\n",
      "      source of the protein. The kinetics of regeneration and leukocytes infiltration, \n",
      "      the accumulation of cell remnants and anti-histidyl-t-RNA synthetase\n",
      "      autoantibodies were compared in wild-type and ptx3-deficient mice. RESULTS: Ptx3 \n",
      "      expression was up-regulated three-five days after injury and restricted to the\n",
      "      extracellular matrix. Cellular debris and leukocytes persisted in the muscle of\n",
      "      ptx3-deficient mice for a long time after wild-type animals had healed.\n",
      "      ptx3-deficient macrophages expressed receptors involved in apoptotic cell\n",
      "      clearance and engulfed dead cells in vitro. Accumulation of cell debris in a\n",
      "      pro-inflammatory microenvironment was not sufficient to elicit autoantibodies.\n",
      "      CONCLUSION: PTX3 generated in response to muscle injury prompts the clearance of \n",
      "      debris and the termination of the inflammatory response.\n",
      "FAU - Vezzoli, Michela\n",
      "AU  - Vezzoli M\n",
      "AD  - Division of Immunology, Transplantation and Infectious Disease, San Raffaele\n",
      "      Scientific Institute, Milano, Italy.\n",
      "FAU - Sciorati, Clara\n",
      "AU  - Sciorati C\n",
      "AD  - Division of Immunology, Transplantation and Infectious Disease, San Raffaele\n",
      "      Scientific Institute, Milano, Italy.\n",
      "FAU - Campana, Lara\n",
      "AU  - Campana L\n",
      "AD  - Division of Immunology, Transplantation and Infectious Disease, San Raffaele\n",
      "      Scientific Institute, Milano, Italy.\n",
      "FAU - Monno, Antonella\n",
      "AU  - Monno A\n",
      "AD  - Division of Immunology, Transplantation and Infectious Disease, San Raffaele\n",
      "      Scientific Institute, Milano, Italy.\n",
      "FAU - Doglio, Maria Giulia\n",
      "AU  - Doglio MG\n",
      "AD  - Division of Immunology, Transplantation and Infectious Disease, San Raffaele\n",
      "      Scientific Institute, Milano, Italy.\n",
      "FAU - Rigamonti, Elena\n",
      "AU  - Rigamonti E\n",
      "AD  - Division of Immunology, Transplantation and Infectious Disease, San Raffaele\n",
      "      Scientific Institute, Milano, Italy.\n",
      "FAU - Corna, Gianfranca\n",
      "AU  - Corna G\n",
      "AD  - Division of Immunology, Transplantation and Infectious Disease, San Raffaele\n",
      "      Scientific Institute, Milano, Italy.\n",
      "FAU - Touvier, Thierry\n",
      "AU  - Touvier T\n",
      "AD  - Division of Immunology, Transplantation and Infectious Disease, San Raffaele\n",
      "      Scientific Institute, Milano, Italy.\n",
      "FAU - Castiglioni, Alessandra\n",
      "AU  - Castiglioni A\n",
      "AD  - Division of Immunology, Transplantation and Infectious Disease, San Raffaele\n",
      "      Scientific Institute, Milano, Italy.\n",
      "FAU - Capobianco, Annalisa\n",
      "AU  - Capobianco A\n",
      "AD  - Division of Immunology, Transplantation and Infectious Disease, San Raffaele\n",
      "      Scientific Institute, Milano, Italy.\n",
      "FAU - Mantovani, Alberto\n",
      "AU  - Mantovani A\n",
      "AD  - Humanitas Clinical and Research Center, Rozzano, Milano, Italy.\n",
      "AD  - Humanitas University, Rozzano, Milan, Italy.\n",
      "FAU - Manfredi, Angelo A\n",
      "AU  - Manfredi AA\n",
      "AD  - Division of Immunology, Transplantation and Infectious Disease, San Raffaele\n",
      "      Scientific Institute, Milano, Italy.\n",
      "AD  - Vita-Salute San Raffaele University, School of Medicine, Milano, Italy.\n",
      "FAU - Garlanda, Cecilia\n",
      "AU  - Garlanda C\n",
      "AD  - Humanitas Clinical and Research Center, Rozzano, Milano, Italy.\n",
      "FAU - Rovere-Querini, Patrizia\n",
      "AU  - Rovere-Querini P\n",
      "AD  - Division of Immunology, Transplantation and Infectious Disease, San Raffaele\n",
      "      Scientific Institute, Milano, Italy.\n",
      "AD  - Vita-Salute San Raffaele University, School of Medicine, Milano, Italy.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161123\n",
      "PL  - United States\n",
      "TA  - Mol Med\n",
      "JT  - Molecular medicine (Cambridge, Mass.)\n",
      "JID - 9501023\n",
      "OTO - NOTNLM\n",
      "OT  - Apoptosis\n",
      "OT  - cell death\n",
      "OT  - immunology\n",
      "OT  - inflammation\n",
      "OT  - rheumatology\n",
      "EDAT- 2016/12/03 06:00\n",
      "MHDA- 2016/12/03 06:00\n",
      "CRDT- 2016/12/01 06:00\n",
      "PHST- 2016/01/02 [received]\n",
      "PHST- 2016/09/29 [accepted]\n",
      "AID - molmed.2016.00002 [pii]\n",
      "AID - 10.2119/molmed.2016.00002 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Mol Med. 2016 Nov 23;22. doi: 10.2119/molmed.2016.00002.\n",
      "\n",
      "PMID- 27900262\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "DA  - 20161130\n",
      "LR  - 20161215\n",
      "IS  - 2212-8778 (Electronic)\n",
      "IS  - 2212-8778 (Linking)\n",
      "VI  - 5\n",
      "IP  - 12\n",
      "DP  - 2016 Dec\n",
      "TI  - Impaired histone deacetylases 5 and 6 expression mimics the effects of obesity\n",
      "      and hypoxia on adipocyte function.\n",
      "PG  - 1200-1207\n",
      "AB  - OBJECTIVE: The goal of the study was to investigate the role of histone\n",
      "      deacetylases (HDACs) in adipocyte function associated with obesity and hypoxia.\n",
      "      METHODS: Total proteins and RNA were prepared from human visceral adipose tissues\n",
      "      (VAT) of human obese and normal weight subjects and from white adipose tissue\n",
      "      (WAT) of C57Bl6-Rj mice fed a normal or high fat diet (HFD) for 16 weeks. HDAC\n",
      "      activity was measured by colorimetric assay whereas the gene and protein\n",
      "      expression were monitored by real-time PCR and by western blotting, respectively.\n",
      "      RNA interference (RNAi) was used to silence the expression of genes in 3T3-L1\n",
      "      adipocytes. RESULTS: Total HDAC activity was decreased in VAT and WAT from obese \n",
      "      individuals and from mice fed a HFD, respectively. The HDAC activity reduction\n",
      "      was associated with decreased HDAC5/Hdac5 and HDAC6/Hdac6 expression in human and\n",
      "      mice adipocyte fraction. Similarly, hypoxia hampered total Hdac activity and\n",
      "      reduced the expression of Hdac5 and Hdac6 in 3T3-L1 adipocytes. The decrease of\n",
      "      both Hdac5 and Hdac6 by hypoxia was associated with altered expression of\n",
      "      adipokines and of the inducible cAMP early repressor (Icer), a key repressor that\n",
      "      is defective in human and mice obesity. Silencing of Icer in adipocytes\n",
      "      reproduced the changes in adipokine levels under hypoxia and obesity, suggesting \n",
      "      a causative effect. Finally, modeling the defect of the two Hdacs in adipocytes\n",
      "      by RNAi or selective inhibitors mimicked the effects of hypoxia on the expression\n",
      "      of Icer, leading to impairment of insulin-induced glucose uptake. CONCLUSION:\n",
      "      Hdac5 and Hdac6 expression are required for the adequate expression of Icer and\n",
      "      adipocyte function. Altered adipose expression of the two Hdacs in obesity by\n",
      "      hypoxia may contribute to the development of metabolic abnormalities.\n",
      "FAU - Bricambert, Julien\n",
      "AU  - Bricambert J\n",
      "AD  - Univ. Lille, CNRS, Institut Pasteur de Lille, UMR 8199 - EGID, F-59000 Lille,\n",
      "      France.\n",
      "FAU - Favre, Dimitri\n",
      "AU  - Favre D\n",
      "AD  - Service of Internal Medicine, Centre Hospitalier Universitaire Vaudois and\n",
      "      University of Lausanne, CH-1011 Lausanne, Switzerland.\n",
      "FAU - Brajkovic, Saska\n",
      "AU  - Brajkovic S\n",
      "AD  - Univ. Lille, CNRS, Institut Pasteur de Lille, UMR 8199 - EGID, F-59000 Lille,\n",
      "      France; Service of Internal Medicine, Centre Hospitalier Universitaire Vaudois\n",
      "      and University of Lausanne, CH-1011 Lausanne, Switzerland.\n",
      "FAU - Bonnefond, Amelie\n",
      "AU  - Bonnefond A\n",
      "AD  - Univ. Lille, CNRS, Institut Pasteur de Lille, UMR 8199 - EGID, F-59000 Lille,\n",
      "      France.\n",
      "FAU - Boutry, Raphael\n",
      "AU  - Boutry R\n",
      "AD  - Univ. Lille, CNRS, Institut Pasteur de Lille, UMR 8199 - EGID, F-59000 Lille,\n",
      "      France.\n",
      "FAU - Salvi, Roberto\n",
      "AU  - Salvi R\n",
      "AD  - Univ. Lille, CNRS, Institut Pasteur de Lille, UMR 8199 - EGID, F-59000 Lille,\n",
      "      France.\n",
      "FAU - Plaisance, Valerie\n",
      "AU  - Plaisance V\n",
      "AD  - Univ. Lille, CNRS, Institut Pasteur de Lille, UMR 8199 - EGID, F-59000 Lille,\n",
      "      France.\n",
      "FAU - Chikri, Mohamed\n",
      "AU  - Chikri M\n",
      "AD  - Qatar Biomedical Research Institute, Hamad Bin Khalifa University, Qatar\n",
      "      Foundation, P.O. Box 5825, Doha, Qatar; Univ. Sidi Mohammed Ben Abdellah, FMPF,\n",
      "      Fes, Morocco.\n",
      "FAU - Chinetti-Gbaguidi, Giulia\n",
      "AU  - Chinetti-Gbaguidi G\n",
      "AD  - Univ. Lille, Inserm, CHU Lille, Institut Pasteur de Lille, U1011 - EGID, F-59000 \n",
      "      Lille, France; Inserm, U 1081, Institute for Research on Cancer and Aging of Nice\n",
      "      (IRCAN), University of Nice-Sophia Antipolis, Nice and Clinical Chemistry\n",
      "      Laboratory, University Hospital, Nice, France.\n",
      "FAU - Staels, Bart\n",
      "AU  - Staels B\n",
      "AD  - Univ. Lille, Inserm, CHU Lille, Institut Pasteur de Lille, U1011 - EGID, F-59000 \n",
      "      Lille, France.\n",
      "FAU - Giusti, Vittorio\n",
      "AU  - Giusti V\n",
      "AD  - Metabolic Center, Fribourg Hospital HFR, Fribourg, Switzerland.\n",
      "FAU - Caiazzo, Robert\n",
      "AU  - Caiazzo R\n",
      "AD  - Univ. Lille, Inserm, CHU Lille, U1190 - EGID, F-59000 Lille, France.\n",
      "FAU - Pattou, Francois\n",
      "AU  - Pattou F\n",
      "AD  - Univ. Lille, Inserm, CHU Lille, U1190 - EGID, F-59000 Lille, France.\n",
      "FAU - Waeber, Gerard\n",
      "AU  - Waeber G\n",
      "AD  - Service of Internal Medicine, Centre Hospitalier Universitaire Vaudois and\n",
      "      University of Lausanne, CH-1011 Lausanne, Switzerland.\n",
      "FAU - Froguel, Philippe\n",
      "AU  - Froguel P\n",
      "AD  - Univ. Lille, CNRS, Institut Pasteur de Lille, UMR 8199 - EGID, F-59000 Lille,\n",
      "      France; Department of Genomic of Common Disease, Imperial College London, UK.\n",
      "FAU - Abderrahmani, Amar\n",
      "AU  - Abderrahmani A\n",
      "AD  - Univ. Lille, CNRS, Institut Pasteur de Lille, UMR 8199 - EGID, F-59000 Lille,\n",
      "      France; Department of Genomic of Common Disease, Imperial College London, UK.\n",
      "LA  - eng\n",
      "GR  - 294785/European Research Council/International\n",
      "PT  - Journal Article\n",
      "DEP - 20161005\n",
      "PL  - Germany\n",
      "TA  - Mol Metab\n",
      "JT  - Molecular metabolism\n",
      "JID - 101605730\n",
      "PMC - PMC5123204\n",
      "OTO - NOTNLM\n",
      "OT  - Adipocytes\n",
      "OT  - Adipokines\n",
      "OT  - Histone deacetylases\n",
      "OT  - Insulin resistance\n",
      "OT  - Obesity\n",
      "EDAT- 2016/12/03 06:00\n",
      "MHDA- 2016/12/03 06:00\n",
      "CRDT- 2016/12/01 06:00\n",
      "PHST- 2016/09/06 [received]\n",
      "PHST- 2016/09/23 [revised]\n",
      "PHST- 2016/09/27 [accepted]\n",
      "AID - 10.1016/j.molmet.2016.09.011 [doi]\n",
      "AID - S2212-8778(16)30188-0 [pii]\n",
      "PST - epublish\n",
      "SO  - Mol Metab. 2016 Oct 5;5(12):1200-1207. eCollection 2016 Dec.\n",
      "\n",
      "PMID- 27899663\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161130\n",
      "LR  - 20161203\n",
      "IS  - 1362-4962 (Electronic)\n",
      "IS  - 0305-1048 (Linking)\n",
      "DP  - 2016 Nov 28\n",
      "TI  - Large-scale analysis of microRNA expression, epi-transcriptomic features and\n",
      "      biogenesis.\n",
      "LID - gkw1031 [pii]\n",
      "AB  - MicroRNAs are important genetic regulators in both animals and plants. They have \n",
      "      a range of functions spanning development, differentiation, growth, metabolism\n",
      "      and disease. The advent of next-generation sequencing technologies has made it a \n",
      "      relatively straightforward task to detect these molecules and their relative\n",
      "      expression via sequencing. There are a large number of published studies with\n",
      "      deposited datasets. However, there are currently few resources that capitalize on\n",
      "      these data to better understand the features, distribution and biogenesis of\n",
      "      miRNAs. Herein, we focus on Human and Mouse for which the majority of data are\n",
      "      available. We reanalyse sequencing data from 461 samples into a coordinated\n",
      "      catalog of microRNA expression. We use this to perform large-scale analyses of\n",
      "      miRNA function and biogenesis. These analyses include global expression\n",
      "      comparison, co-expression of miRNA clusters and the prediction of miRNA\n",
      "      strand-specificity and underlying constraints. Additionally, we report for the\n",
      "      first time a global analysis of miRNA epi-transcriptomic modifications and assess\n",
      "      their prevalence across tissues, samples and families. Finally, we report a list \n",
      "      of potentially mis-annotated miRNAs in miRBase based on their aggregated\n",
      "      modification profiles. The results have been collated into a comprehensive online\n",
      "      repository of miRNA expression and features such as modifications and RNA editing\n",
      "      events, which is available at: http://wwwdev.ebi.ac.uk/enright-dev/miratlas We\n",
      "      believe these findings will further contribute to our understanding of miRNA\n",
      "      function in animals and benefit the miRNA community in general.\n",
      "CI  - (c) The Author(s) 2016. Published by Oxford University Press on behalf of Nucleic\n",
      "      Acids Research.\n",
      "FAU - Vitsios, Dimitrios M\n",
      "AU  - Vitsios DM\n",
      "AD  - EMBL-European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton,\n",
      "      Cambridge CB10 1SD, UK.\n",
      "FAU - Davis, Matthew P\n",
      "AU  - Davis MP\n",
      "AD  - EMBL-European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton,\n",
      "      Cambridge CB10 1SD, UK.\n",
      "FAU - van Dongen, Stijn\n",
      "AU  - van Dongen S\n",
      "AD  - EMBL-European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton,\n",
      "      Cambridge CB10 1SD, UK.\n",
      "FAU - Enright, Anton J\n",
      "AU  - Enright AJ\n",
      "AD  - EMBL-European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton,\n",
      "      Cambridge CB10 1SD, UK aje@ebi.ac.uk.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161128\n",
      "PL  - England\n",
      "TA  - Nucleic Acids Res\n",
      "JT  - Nucleic acids research\n",
      "JID - 0411011\n",
      "EDAT- 2016/12/03 06:00\n",
      "MHDA- 2016/12/03 06:00\n",
      "CRDT- 2016/12/01 06:00\n",
      "PHST- 2016/10/18 [accepted]\n",
      "PHST- 2016/10/17 [revised]\n",
      "PHST- 2016/09/22 [received]\n",
      "AID - gkw1031 [pii]\n",
      "AID - 10.1093/nar/gkw1031 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Nucleic Acids Res. 2016 Nov 28. pii: gkw1031.\n",
      "\n",
      "PMID- 27899633\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "DA  - 20161130\n",
      "LR  - 20161203\n",
      "IS  - 1362-4962 (Electronic)\n",
      "IS  - 0305-1048 (Linking)\n",
      "DP  - 2016 Nov 29\n",
      "TI  - JMJD6 and U2AF65 co-regulate alternative splicing in both JMJD6 enzymatic\n",
      "      activity dependent and independent manner.\n",
      "LID - gkw1144 [pii]\n",
      "AB  - JMJD6, a jumonji C (Jmj C) domain-containing protein demethylase and hydroxylase,\n",
      "      has been implicated in an array of biological processes. It has been shown that\n",
      "      JMJD6 interacts with and hydroxylates multiple serine/arginine-rich (SR) proteins\n",
      "      and SR related proteins, including U2AF65, all of which are known to function in \n",
      "      alternative splicing regulation. However, whether JMJD6 is widely involved in\n",
      "      alternative splicing and the molecular mechanism underlying JMJD6-regulated\n",
      "      alternative splicing have remained incompletely understood. Here, by using\n",
      "      RASL-Seq, we investigated the functional impact of RNA-dependent interaction\n",
      "      between JMJD6 and U2AF65, revealing that JMJD6 and U2AF65 co-regulated a large\n",
      "      number of alternative splicing events. We further demonstrated the JMJD6 function\n",
      "      in alternative splicing in jmjd6 knockout mice. Mechanistically, we showed that\n",
      "      the enzymatic activity of JMJD6 was required for a subset of JMJD6-regulated\n",
      "      splicing, and JMJD6-mediated lysine hydroxylation of U2AF65 could account for, at\n",
      "      least partially, their co-regulated alternative splicing events, suggesting both \n",
      "      JMJD6 enzymatic activity-dependent and independent control of alternative\n",
      "      splicing. These findings reveal an intimate link between JMJD6 and U2AF65 in\n",
      "      alternative splicing regulation, which has important implications in development \n",
      "      and disease processes.\n",
      "CI  - (c) The Author(s) 2016. Published by Oxford University Press on behalf of Nucleic\n",
      "      Acids Research.\n",
      "FAU - Yi, Jia\n",
      "AU  - Yi J\n",
      "AD  - Fujian Provincial Key Laboratory of Innovative Drug Target Research, School of\n",
      "      Pharmaceutical Sciences, Xiamen University, Xiang'an South Road, Xiamen, Fujian\n",
      "      361102, China.\n",
      "FAU - Shen, Hai-Feng\n",
      "AU  - Shen HF\n",
      "AD  - Fujian Provincial Key Laboratory of Innovative Drug Target Research, School of\n",
      "      Pharmaceutical Sciences, Xiamen University, Xiang'an South Road, Xiamen, Fujian\n",
      "      361102, China.\n",
      "FAU - Qiu, Jin-Song\n",
      "AU  - Qiu JS\n",
      "AD  - Department of Cellular and Molecular Medicine, University of California, San\n",
      "      Diego, La Jolla, CA 92093-0651, USA.\n",
      "FAU - Huang, Ming-Feng\n",
      "AU  - Huang MF\n",
      "AD  - Fujian Provincial Key Laboratory of Innovative Drug Target Research, School of\n",
      "      Pharmaceutical Sciences, Xiamen University, Xiang'an South Road, Xiamen, Fujian\n",
      "      361102, China.\n",
      "FAU - Zhang, Wen-Juan\n",
      "AU  - Zhang WJ\n",
      "AD  - Fujian Provincial Key Laboratory of Innovative Drug Target Research, School of\n",
      "      Pharmaceutical Sciences, Xiamen University, Xiang'an South Road, Xiamen, Fujian\n",
      "      361102, China.\n",
      "FAU - Ding, Jian-Cheng\n",
      "AU  - Ding JC\n",
      "AD  - Fujian Provincial Key Laboratory of Innovative Drug Target Research, School of\n",
      "      Pharmaceutical Sciences, Xiamen University, Xiang'an South Road, Xiamen, Fujian\n",
      "      361102, China.\n",
      "FAU - Zhu, Xiao-Yan\n",
      "AU  - Zhu XY\n",
      "AD  - Department of Medicine, School of Medicine, University of California, San Diego, \n",
      "      La Jolla, CA 92093-0648, USA.\n",
      "FAU - Zhou, Yu\n",
      "AU  - Zhou Y\n",
      "AD  - Department of Cellular and Molecular Medicine, University of California, San\n",
      "      Diego, La Jolla, CA 92093-0651, USA.\n",
      "FAU - Fu, Xiang-Dong\n",
      "AU  - Fu XD\n",
      "AD  - Department of Cellular and Molecular Medicine, University of California, San\n",
      "      Diego, La Jolla, CA 92093-0651, USA xdfu@ucsd.edu.\n",
      "FAU - Liu, Wen\n",
      "AU  - Liu W\n",
      "AD  - Fujian Provincial Key Laboratory of Innovative Drug Target Research, School of\n",
      "      Pharmaceutical Sciences, Xiamen University, Xiang'an South Road, Xiamen, Fujian\n",
      "      361102, China w2liu@xmu.edu.cn.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161129\n",
      "PL  - England\n",
      "TA  - Nucleic Acids Res\n",
      "JT  - Nucleic acids research\n",
      "JID - 0411011\n",
      "EDAT- 2016/12/03 06:00\n",
      "MHDA- 2016/12/03 06:00\n",
      "CRDT- 2016/12/01 06:00\n",
      "PHST- 2016/11/02 [accepted]\n",
      "PHST- 2016/09/24 [received]\n",
      "AID - gkw1144 [pii]\n",
      "AID - 10.1093/nar/gkw1144 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Nucleic Acids Res. 2016 Nov 29. pii: gkw1144.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "fetchHandle = Entrez.efetch(db = \"pubmed\", id = searchRecord[\"IdList\"], rettype = \"medline\", retmode = \"text\")\n",
    "fetchRecords = fetchHandle.read()\n",
    "print(fetchRecords)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'AB': 'AMP-activated protein kinase (AMPK) is a metabolic enzyme that can be activated by nutrient stress or genetic mutations. Missense mutations in the regulatory subunit, PRKAG2, activate AMPK and cause left ventricular hypertrophy, glycogen accumulation, and ventricular pre-excitation. Using human iPS cell models combined with three-dimensional cardiac microtissues, we show that activating PRKAG2 mutations increase microtissue twitch force by enhancing myocyte survival. Integrating RNA sequencing with metabolomics, PRKAG2 mutations that activate AMPK remodeled global metabolism by regulating RNA transcripts to favor glycogen storage and oxidative metabolism instead of glycolysis. As in patients with PRKAG2 cardiomyopathy, iPS cell and mouse models are protected from cardiac fibrosis, and we define a crosstalk between AMPK and post-transcriptional regulation of TGFbeta isoform signaling that has implications in fibrotic forms of cardiomyopathy. Our results establish critical connections among metabolic sensing, myocyte survival, and TGFbeta signaling.',\n",
       " 'AD': \"The Jackson Laboratory for Genomic Medicine, Farmington, CT 06032, USA; Cardiology Center, University of Connecticut Health, Farmington, CT 06030, USA. Electronic address: travis.hinson@jax.org. Department of Biomedical Engineering, Boston University, Boston, MA 02215, USA; The Wyss Institute for Biologically Inspired Engineering at Harvard University, Boston, MA 02115, USA. The Jackson Laboratory for Genomic Medicine, Farmington, CT 06032, USA. Department of Genetics, Harvard Medical School, Boston, MA 02115, USA. Division of Cardiovascular Medicine, Brigham and Women's Hospital, Boston, MA 02115, USA. Division of Cardiovascular Medicine, Cardiovascular Institute, Stanford University School of Medicine, Stanford, CA 94305, USA. Division of Cardiovascular Medicine, Massachusetts General Hospital, Boston, MA 02114, USA. Division of Cardiovascular Disease, University of Alabama at Birmingham, Birmingham, AL 35294, USA. Department of Genetics, Harvard Medical School, Boston, MA 02115, USA. Department of Genetics, Harvard Medical School, Boston, MA 02115, USA. Department of Biomedical Engineering, Boston University, Boston, MA 02215, USA; The Wyss Institute for Biologically Inspired Engineering at Harvard University, Boston, MA 02115, USA. Penn Cardiovascular Institute, University of Pennsylvania, Philadelphia, PA 19104, USA. Division of Cardiovascular Medicine, Massachusetts General Hospital, Boston, MA 02114, USA; Division of Cardiovascular Medicine, Beth Israel Deaconess Hospital, Boston, MA 02115, USA. Division of Cardiovascular Medicine, Cardiovascular Institute, Stanford University School of Medicine, Stanford, CA 94305, USA. Department of Biomedical Engineering, Boston University, Boston, MA 02215, USA; The Wyss Institute for Biologically Inspired Engineering at Harvard University, Boston, MA 02115, USA. Department of Genetics, Harvard Medical School, Boston, MA 02115, USA. Department of Genetics, Harvard Medical School, Boston, MA 02115, USA; Division of Cardiovascular Medicine, Brigham and Women's Hospital, Boston, MA 02115, USA; Howard Hughes Medical Institute, Chevy Chase, MD 20815, USA. Electronic address: cseidman@genetics.med.harvard.edu.\",\n",
       " 'AID': ['S2211-1247(16)31640-0 [pii]', '10.1016/j.celrep.2016.11.066 [doi]'],\n",
       " 'AU': ['Hinson JT',\n",
       "  'Chopra A',\n",
       "  'Lowe A',\n",
       "  'Sheng CC',\n",
       "  'Gupta RM',\n",
       "  'Kuppusamy R',\n",
       "  \"O'Sullivan J\",\n",
       "  'Rowe G',\n",
       "  'Wakimoto H',\n",
       "  'Gorham J',\n",
       "  'Zhang K',\n",
       "  'Musunuru K',\n",
       "  'Gerszten RE',\n",
       "  'Wu SM',\n",
       "  'Chen CS',\n",
       "  'Seidman JG',\n",
       "  'Seidman CE'],\n",
       " 'CI': ['Copyright (c) 2016 The Author(s). Published by Elsevier Inc. All rights reserved.'],\n",
       " 'CRDT': ['2016/12/24 06:00'],\n",
       " 'DA': '20161223',\n",
       " 'DP': '2016 Dec 20',\n",
       " 'EDAT': '2016/12/24 06:00',\n",
       " 'FAU': ['Hinson, J Travis',\n",
       "  'Chopra, Anant',\n",
       "  'Lowe, Andre',\n",
       "  'Sheng, Calvin C',\n",
       "  'Gupta, Rajat M',\n",
       "  'Kuppusamy, Rajarajan',\n",
       "  \"O'Sullivan, John\",\n",
       "  'Rowe, Glenn',\n",
       "  'Wakimoto, Hiroko',\n",
       "  'Gorham, Joshua',\n",
       "  'Zhang, Kehan',\n",
       "  'Musunuru, Kiran',\n",
       "  'Gerszten, Robert E',\n",
       "  'Wu, Sean M',\n",
       "  'Chen, Christopher S',\n",
       "  'Seidman, Jonathan G',\n",
       "  'Seidman, Christine E'],\n",
       " 'IP': '12',\n",
       " 'IS': '2211-1247 (Electronic)',\n",
       " 'JID': '101573691',\n",
       " 'JT': 'Cell reports',\n",
       " 'LA': ['eng'],\n",
       " 'LID': 'S2211-1247(16)31640-0 [pii] 10.1016/j.celrep.2016.11.066 [doi]',\n",
       " 'LR': '20161224',\n",
       " 'MHDA': '2016/12/24 06:00',\n",
       " 'OT': ['AMP-activated protein kinase',\n",
       "  'AMPK',\n",
       "  'PRKAG2',\n",
       "  'TGF-beta',\n",
       "  'Wolff-Parkinson-White syndrome',\n",
       "  'fibrosis',\n",
       "  'glycogen storage disease',\n",
       "  'hypertrophic cardiomyopathy'],\n",
       " 'OTO': ['NOTNLM'],\n",
       " 'OWN': 'NLM',\n",
       " 'PG': '3292-3304',\n",
       " 'PHST': ['2016/07/01 [received]',\n",
       "  '2016/09/19 [revised]',\n",
       "  '2016/11/21 [accepted]'],\n",
       " 'PL': 'United States',\n",
       " 'PMID': '28009297',\n",
       " 'PST': 'ppublish',\n",
       " 'PT': ['Journal Article'],\n",
       " 'SO': 'Cell Rep. 2016 Dec 20;17(12):3292-3304. doi: 10.1016/j.celrep.2016.11.066.',\n",
       " 'STAT': 'In-Data-Review',\n",
       " 'TA': 'Cell Rep',\n",
       " 'TI': 'Integrative Analysis of PRKAG2 Cardiomyopathy iPS and Microtissue Models Identifies AMPK as a Regulator of Metabolism, Survival, and Fibrosis.',\n",
       " 'VI': '17'}"
      ]
     },
     "execution_count": 44,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from StringIO import StringIO\n",
    "\n",
    "rec_file = StringIO(fetchRecord)\n",
    "medline_rec = Medline.read(rec_file)\n",
    "\n",
    "medline_rec"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'AMP-activated protein kinase (AMPK) is a metabolic enzyme that can be activated by nutrient stress or genetic mutations. Missense mutations in the regulatory subunit, PRKAG2, activate AMPK and cause left ventricular hypertrophy, glycogen accumulation, and ventricular pre-excitation. Using human iPS cell models combined with three-dimensional cardiac microtissues, we show that activating PRKAG2 mutations increase microtissue twitch force by enhancing myocyte survival. Integrating RNA sequencing with metabolomics, PRKAG2 mutations that activate AMPK remodeled global metabolism by regulating RNA transcripts to favor glycogen storage and oxidative metabolism instead of glycolysis. As in patients with PRKAG2 cardiomyopathy, iPS cell and mouse models are protected from cardiac fibrosis, and we define a crosstalk between AMPK and post-transcriptional regulation of TGFbeta isoform signaling that has implications in fibrotic forms of cardiomyopathy. Our results establish critical connections among metabolic sensing, myocyte survival, and TGFbeta signaling.'"
      ]
     },
     "execution_count": 45,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "medline_rec['AB']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# write to file\n",
    "txtFile = open(\"dt/bioinfAbstract.search.txt\", \"w\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "import StringIO\n",
    "from Bio import Medline\n",
    "for id in searchRecord['IdList']:\n",
    "    fetchHandle = Entrez.efetch(db = \"pubmed\", id = id, rettype = \"medline\", retmode = \"text\")\n",
    "    fetchRecords = fetchHandle.read()\n",
    "    recordString = StringIO.StringIO(fetchRecords)\n",
    "    medlineRecord = Medline.read(recordString)\n",
    "    txtFile.write(medlineRecord['AB'] + '\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "txtFile.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# http://stackoverflow.com/questions/36087715/how-can-i-extract-the-abstract-from-efetch-biopython-entrez\n",
    "from StringIO import StringIO\n",
    "from Bio import Entrez, Medline\n",
    "\n",
    "def search_medline(query, email):\n",
    "    Entrez.email = email\n",
    "    search = Entrez.esearch(db='pubmed', term=query, usehistory='y')\n",
    "    handle = Entrez.read(search)\n",
    "    try:\n",
    "        return handle\n",
    "    except Exception as e:\n",
    "        raise IOError(str(e))\n",
    "    finally:\n",
    "        search.close()\n",
    "\n",
    "def fetch_rec(rec_id, entrez_handle):\n",
    "    fetch_handle = Entrez.efetch(db='pubmed', id=rec_id,\n",
    "                                 rettype='Medline', retmode='text',\n",
    "                                 webenv=entrez_handle['WebEnv'],\n",
    "                                 query_key=entrez_handle['QueryKey'])\n",
    "    rec = fetch_handle.read()\n",
    "    return rec\n",
    "\n",
    "def main(query, email):\n",
    "    rec_handler = search_medline(query, email)\n",
    "\n",
    "    for rec_id in rec_handler['IdList']:\n",
    "        rec = fetch_rec(rec_id, rec_handler)\n",
    "        rec_file = StringIO(rec)\n",
    "        medline_rec = Medline.read(rec_file)\n",
    "        if 'AB' in medline_rec:\n",
    "            print(medline_rec['AB'])\n",
    "\n",
    "if __name__ == '__main__':\n",
    "    email = \"my-email@provider.sth\"\n",
    "    query = \"Tischner[AU] Cortex-specific down-regulation\"\n",
    "    main(query, email)"
   ]
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python [default]",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
